
<!doctype html>
<html lang="en">
<head>
<meta charset="utf-8" />
<title>JABTEZ000000000.1 - 27 region(s) - antiSMASH results</title>
<style>img {width: 20px; height: 20px;}</style>
<link rel="stylesheet" type="text/css" href="css/bacteria.css">
<meta property="og:title" content="JABTEZ000000000.1 - 8 record(s) - 27 region(s)" />
<meta property="og:description" content="">
<meta property="og:image" content="https://antismash.secondarymetabolites.org/images/bacteria_logo.png" />
<meta property="og:image:alt" content="antiSMASH logo" />
<meta property="og:image:width" content="600" />
<meta property="og:image:height" content="600" />
<meta property="og:url" content="https://antismash.secondarymetabolites.org/upload/bacteria-6297c27d-b3ea-459a-ac6f-b08751881fc8/index.html" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta name="twitter:site" content="@antismash_dev" />
</head>

<body>
<div id="header">
<nav>
<div class="main-link">
<div>
<a href="https://antismash.secondarymetabolites.org/">
<img src="images/bacteria_antismash_logo.svg" alt="antiSMASH logo" style="width:40px;height:unset;">
</a>
</div>
<div>
<a href="https://antismash.secondarymetabolites.org/">
antiSMASH version 7.1.0
</a>
</div>
</div>
<div class="custom-description"></div>
<div class="ancillary-links">
<div class="ancillary-link dropdown-menu" id="download-dropdown"><a href="#" id="download-dropdown-link"><img src="images/download.svg" alt="download"> &nbsp; Download</a>
<ul class="dropdown-options">

<li><a href="JABTEZ000000000.1.zip">Download all results</a></li>

<li><a href="JABTEZ000000000.1.gbk">Download GenBank summary file</a></li>
<li><a href="JABTEZ000000000.1.json">Download JSON results file</a></li>

<li><a href="bacteria-6297c27d-b3ea-459a-ac6f-b08751881fc8.log">Download log file</a></li>

</ul>
</div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/about"><img src="images/about.svg" alt="about"> &nbsp; About</a></div>
<div class="ancillary-link"><a href="https://docs.antismash.secondarymetabolites.org/"><img src="images/help.svg" alt="help"> &nbsp; Help</a></div>
<div class="ancillary-link"><a href="https://antismash.secondarymetabolites.org/#!/contact"><img src="images/contact.svg" alt="contact"> &nbsp; Contact</a></div>
</div>
</nav>
<div id="buttons">
<span id="region-type">Select genomic region:</span>
<div id="regionbuttons">
<div class="arrow-left" id="prev-region"></div>
<div class="regbutton" style="min-width: inherit;"><a href="#">Overview</a></div>
<div style="display: flex; flex-wrap: wrap">


<div class="regbutton RiPP RiPP-like r1c1">
<a href="#r1c1">1.1</a>
</div>

<div class="regbutton PKS T1PKS r1c2">
<a href="#r1c2">1.2</a>
</div>

<div class="regbutton RiPP lanthipeptide-class-iv r1c3">
<a href="#r1c3">1.3</a>
</div>

<div class="regbutton RiPP r1c4">
<a href="#r1c4">1.4</a>
</div>

<div class="regbutton other other r1c5">
<a href="#r1c5">1.5</a>
</div>



<div class="regbutton hybrid r2c1">
<a href="#r2c1">2.1</a>
</div>

<div class="regbutton NRPS NRPS r2c2">
<a href="#r2c2">2.2</a>
</div>

<div class="regbutton hybrid r2c3">
<a href="#r2c3">2.3</a>
</div>

<div class="regbutton hybrid r2c4">
<a href="#r2c4">2.4</a>
</div>

<div class="regbutton terpene terpene r2c5">
<a href="#r2c5">2.5</a>
</div>

<div class="regbutton hybrid r2c6">
<a href="#r2c6">2.6</a>
</div>

<div class="regbutton hybrid r2c7">
<a href="#r2c7">2.7</a>
</div>



<div class="regbutton terpene terpene r3c1">
<a href="#r3c1">3.1</a>
</div>

<div class="regbutton NRPS NRPS r3c2">
<a href="#r3c2">3.2</a>
</div>

<div class="regbutton RiPP lanthipeptide-class-iii r3c3">
<a href="#r3c3">3.3</a>
</div>

<div class="regbutton NRPS NAPAA r3c4">
<a href="#r3c4">3.4</a>
</div>

<div class="regbutton hybrid r3c5">
<a href="#r3c5">3.5</a>
</div>



<div class="regbutton NRPS r4c1">
<a href="#r4c1">4.1</a>
</div>

<div class="regbutton hybrid r4c2">
<a href="#r4c2">4.2</a>
</div>



<div class="regbutton NRPS NRPS r5c1">
<a href="#r5c1">5.1</a>
</div>

<div class="regbutton hybrid r5c2">
<a href="#r5c2">5.2</a>
</div>

<div class="regbutton hybrid r5c3">
<a href="#r5c3">5.3</a>
</div>



<div class="regbutton other hydrogen-cyanide r6c1">
<a href="#r6c1">6.1</a>
</div>

<div class="regbutton PKS T3PKS r6c2">
<a href="#r6c2">6.2</a>
</div>

<div class="regbutton hybrid r6c3">
<a href="#r6c3">6.3</a>
</div>



<div class="regbutton other NI-siderophore r8c1">
<a href="#r8c1">8.1</a>
</div>



<div class="regbutton NRPS NRPS r9c1">
<a href="#r9c1">9.1</a>
</div>


</div>
<div id="last-regbutton"><div class="arrow-right" id="next-region"></div></div>
</div>
</div>
</div>

<!-- overview page -->

<div class="page" id="overview">


<h3>Identified secondary metabolite regions using strictness &#39;relaxed&#39;<span id="truncated"></span></h3>

<div class="overview-layout">
<div id="record-tables">
<div id="single-record-tables">
<div class="record-overview-header">
<strong>JABTEZ010000001.1</strong> (Planomonospora sp. ID67723)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-1" data-filename="JABTEZ010000001.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-1" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton RiPP RiPP-like r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">62,410</td>
<td class="digits table-split-left">73,276</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c2">
<td class="regbutton PKS T1PKS r1c2">
<a href="#r1c2">Region&nbsp1.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">241,518</td>
<td class="digits table-split-left">287,829</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000813/1" target="_blank">K-252a</a></td>
<td>Alkaloid</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 14%, #ffffff00 14%)">14%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c3">
<td class="regbutton RiPP lanthipeptide-class-iv r1c3">
<a href="#r1c3">Region&nbsp1.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iv" target="_blank">lanthipeptide-class-iv</a>
</td>
<td class="digits">1,133,085</td>
<td class="digits table-split-left">1,155,793</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c4">
<td class="regbutton RiPP r1c4">
<a href="#r1c4">Region&nbsp1.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thiopeptide" target="_blank">thiopeptide</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lap" target="_blank">LAP</a>
</td>
<td class="digits">1,742,150</td>
<td class="digits table-split-left">1,772,558</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c5">
<td class="regbutton other other r1c5">
<a href="#r1c5">Region&nbsp1.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>
</td>
<td class="digits">2,270,695</td>
<td class="digits table-split-left">2,311,132</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>JABTEZ010000002.1</strong> (Planomonospora sp. ID67723)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-2" data-filename="JABTEZ010000002.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-2" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r2c1">
<td class="regbutton hybrid r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">68,150</td>
<td class="digits table-split-left">118,622</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002001/1" target="_blank">crochelin A</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c2">
<td class="regbutton NRPS NRPS r2c2">
<a href="#r2c2">Region&nbsp2.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">215,400</td>
<td class="digits table-split-left">270,759</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000163/1" target="_blank">tetronasin</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 3%, #ffffff00 3%)">3%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c3">
<td class="regbutton hybrid r2c3">
<a href="#r2c3">Region&nbsp2.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">479,865</td>
<td class="digits table-split-left">573,731</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001008/1" target="_blank">maduropeptin</a></td>
<td>Polyketide:Iterative type I polyketide+Polyketide:Enediyne type I polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 41%, #ffffff00 41%)">41%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c4">
<td class="regbutton hybrid r2c4">
<a href="#r2c4">Region&nbsp2.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lap" target="_blank">LAP</a>
</td>
<td class="digits">1,285,856</td>
<td class="digits table-split-left">1,347,562</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001752/1" target="_blank">qinichelins</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 16%, #ffffff00 16%)">16%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c5">
<td class="regbutton terpene terpene r2c5">
<a href="#r2c5">Region&nbsp2.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">1,372,493</td>
<td class="digits table-split-left">1,393,338</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002671/1" target="_blank">kendomycin B</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c6">
<td class="regbutton hybrid r2c6">
<a href="#r2c6">Region&nbsp2.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-ii" target="_blank">lanthipeptide-class-ii</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,729,536</td>
<td class="digits table-split-left">1,806,044</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002466/1" target="_blank">peucechelin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c7">
<td class="regbutton hybrid r2c7">
<a href="#r2c7">Region&nbsp2.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1,936,660</td>
<td class="digits table-split-left">1,998,654</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002001/1" target="_blank">crochelin A</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>JABTEZ010000003.1</strong> (Planomonospora sp. ID67723)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-3" data-filename="JABTEZ010000003.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-3" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r3c1">
<td class="regbutton terpene terpene r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">373,105</td>
<td class="digits table-split-left">394,103</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c2">
<td class="regbutton NRPS NRPS r3c2">
<a href="#r3c2">Region&nbsp3.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">622,779</td>
<td class="digits table-split-left">687,796</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000354/1" target="_blank">friulimicin A/friulimicin B/friulimicin C/friulimicin D</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 15%, #ffffff00 15%)">15%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c3">
<td class="regbutton RiPP lanthipeptide-class-iii r3c3">
<a href="#r3c3">Region&nbsp3.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
</td>
<td class="digits">750,481</td>
<td class="digits table-split-left">773,255</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000501/1" target="_blank">catenulipeptin</a></td>
<td>RiPP:Lanthipeptide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>

</tr>
<tr class="linked-row even" data-anchor="#r3c4">
<td class="regbutton NRPS NAPAA r3c4">
<a href="#r3c4">Region&nbsp3.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
</td>
<td class="digits">916,608</td>
<td class="digits table-split-left">950,510</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002174/1" target="_blank">ε-Poly-L-lysine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c5">
<td class="regbutton hybrid r3c5">
<a href="#r3c5">Region&nbsp3.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iv" target="_blank">lanthipeptide-class-iv</a>
</td>
<td class="digits">972,257</td>
<td class="digits table-split-left">1,007,606</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000519/1" target="_blank">labyrinthopeptin A2/labyrinthopeptin A1/labyrinthopeptin A3</a></td>
<td>RiPP:Lanthipeptide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>JABTEZ010000004.1</strong> (Planomonospora sp. ID67723)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-4" data-filename="JABTEZ010000004.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-4" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r4c1">
<td class="regbutton NRPS r4c1">
<a href="#r4c1">Region&nbsp4.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">86,408</td>
<td class="digits table-split-left">134,411</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002051/1" target="_blank">antipain</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r4c2">
<td class="regbutton hybrid r4c2">
<a href="#r4c2">Region&nbsp4.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#pks-like" target="_blank">PKS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
</td>
<td class="digits">244,100</td>
<td class="digits table-split-left">285,164</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002098/1" target="_blank">ECO-0501</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>JABTEZ010000005.1</strong> (Planomonospora sp. ID67723)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-5" data-filename="JABTEZ010000005.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-5" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r5c1">
<td class="regbutton NRPS NRPS r5c1">
<a href="#r5c1">Region&nbsp5.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">377,573</td>
<td class="digits table-split-left">426,460</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002001/1" target="_blank">crochelin A</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>

</tr>
<tr class="linked-row even" data-anchor="#r5c2">
<td class="regbutton hybrid r5c2">
<a href="#r5c2">Region&nbsp5.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">590,054</td>
<td class="digits table-split-left">681,562</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000661/1" target="_blank">geosmin</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c3">
<td class="regbutton hybrid r5c3">
<a href="#r5c3">Region&nbsp5.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">706,575</td>
<td class="digits table-split-left">757,169</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001818/1" target="_blank">vazabitide A</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>JABTEZ010000006.1</strong> (Planomonospora sp. ID67723)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-6" data-filename="JABTEZ010000006.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-6" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r6c1">
<td class="regbutton other hydrogen-cyanide r6c1">
<a href="#r6c1">Region&nbsp6.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hydrogen-cyanide" target="_blank">hydrogen-cyanide</a>
</td>
<td class="digits">67,500</td>
<td class="digits table-split-left">80,656</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002285/1" target="_blank">aborycin</a></td>
<td>RiPP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 21%, #ffffff00 21%)">21%</td>

</tr>
<tr class="linked-row even" data-anchor="#r6c2">
<td class="regbutton PKS T3PKS r6c2">
<a href="#r6c2">Region&nbsp6.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
</td>
<td class="digits">119,067</td>
<td class="digits table-split-left">160,176</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000282/1" target="_blank">alkylresorcinol</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c3">
<td class="regbutton hybrid r6c3">
<a href="#r6c3">Region&nbsp6.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">417,280</td>
<td class="digits table-split-left">449,361</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001332/1" target="_blank">microtermolide A</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>JABTEZ010000008.1</strong> (Planomonospora sp. ID67723)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-8" data-filename="JABTEZ010000008.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-8" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r8c1">
<td class="regbutton other NI-siderophore r8c1">
<a href="#r8c1">Region&nbsp8.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ni-siderophore" target="_blank">NI-siderophore</a>
</td>
<td class="digits">10,930</td>
<td class="digits table-split-left">42,204</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002466/1" target="_blank">peucechelin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 10%, #ffffff00 10%)">10%</td>

</tr>

</tbody>
</table>
</div><div class="record-overview-header">
<strong>JABTEZ010000009.1</strong> (Planomonospora sp. ID67723)
<div class="download-container">
<div class="download-icon-inverse download-svg" data-tag="record-minimap-9" data-filename="JABTEZ010000009.1_overview.svg">
</div>
</div>
</div>
<div class="record-overview-details">
<svg id="record-minimap-9" class="record-overview-svg" height="1" width="1"></svg>
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>
<tr class="linked-row odd" data-anchor="#r9c1">
<td class="regbutton NRPS NRPS r9c1">
<a href="#r9c1">Region&nbsp9.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">2,803</td>

<td colspan="3"></td>

</tr>

</tbody>
</table>
</div>
</div>

<div id="compact-record-table" style="display: none;">
<div class="record-table">
<table class="region-table">
<thead>
<tr>
<th>Region</th>
<th>Type</th>
<th>From</th>
<th>To</th>
<th colspan="2">Most similar known cluster</th>
<th>Similarity</th>
</tr>
</thead>
<tbody>


<tr class="linked-row odd" data-anchor="#r1c1">
<td class="regbutton RiPP RiPP-like r1c1">
<a href="#r1c1">Region&nbsp1.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ripp-like" target="_blank">RiPP-like</a>
</td>
<td class="digits">62,410</td>
<td class="digits table-split-left">73,276</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c2">
<td class="regbutton PKS T1PKS r1c2">
<a href="#r1c2">Region&nbsp1.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">241,518</td>
<td class="digits table-split-left">287,829</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000813/1" target="_blank">K-252a</a></td>
<td>Alkaloid</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 14%, #ffffff00 14%)">14%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c3">
<td class="regbutton RiPP lanthipeptide-class-iv r1c3">
<a href="#r1c3">Region&nbsp1.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iv" target="_blank">lanthipeptide-class-iv</a>
</td>
<td class="digits">1,133,085</td>
<td class="digits table-split-left">1,155,793</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r1c4">
<td class="regbutton RiPP r1c4">
<a href="#r1c4">Region&nbsp1.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#thiopeptide" target="_blank">thiopeptide</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lap" target="_blank">LAP</a>
</td>
<td class="digits">1,742,150</td>
<td class="digits table-split-left">1,772,558</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row odd" data-anchor="#r1c5">
<td class="regbutton other other r1c5">
<a href="#r1c5">Region&nbsp1.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#other" target="_blank">other</a>
</td>
<td class="digits">2,270,695</td>
<td class="digits table-split-left">2,311,132</td>

<td colspan="3"></td>

</tr>


<tr class="linked-row even" data-anchor="#r2c1">
<td class="regbutton hybrid r2c1">
<a href="#r2c1">Region&nbsp2.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">68,150</td>
<td class="digits table-split-left">118,622</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002001/1" target="_blank">crochelin A</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c2">
<td class="regbutton NRPS NRPS r2c2">
<a href="#r2c2">Region&nbsp2.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">215,400</td>
<td class="digits table-split-left">270,759</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000163/1" target="_blank">tetronasin</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 3%, #ffffff00 3%)">3%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c3">
<td class="regbutton hybrid r2c3">
<a href="#r2c3">Region&nbsp2.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">479,865</td>
<td class="digits table-split-left">573,731</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001008/1" target="_blank">maduropeptin</a></td>
<td>Polyketide:Iterative type I polyketide+Polyketide:Enediyne type I polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 41%, #ffffff00 41%)">41%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c4">
<td class="regbutton hybrid r2c4">
<a href="#r2c4">Region&nbsp2.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lap" target="_blank">LAP</a>
</td>
<td class="digits">1,285,856</td>
<td class="digits table-split-left">1,347,562</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001752/1" target="_blank">qinichelins</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 16%, #ffffff00 16%)">16%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c5">
<td class="regbutton terpene terpene r2c5">
<a href="#r2c5">Region&nbsp2.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">1,372,493</td>
<td class="digits table-split-left">1,393,338</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002671/1" target="_blank">kendomycin B</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 9%, #ffffff00 9%)">9%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r2c6">
<td class="regbutton hybrid r2c6">
<a href="#r2c6">Region&nbsp2.6</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-ii" target="_blank">lanthipeptide-class-ii</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrp-metallophore" target="_blank">NRP-metallophore</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1,729,536</td>
<td class="digits table-split-left">1,806,044</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002466/1" target="_blank">peucechelin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>
<tr class="linked-row even" data-anchor="#r2c7">
<td class="regbutton hybrid r2c7">
<a href="#r2c7">Region&nbsp2.7</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">1,936,660</td>
<td class="digits table-split-left">1,998,654</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002001/1" target="_blank">crochelin A</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r3c1">
<td class="regbutton terpene terpene r3c1">
<a href="#r3c1">Region&nbsp3.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>
</td>
<td class="digits">373,105</td>
<td class="digits table-split-left">394,103</td>

<td colspan="3"></td>

</tr>
<tr class="linked-row even" data-anchor="#r3c2">
<td class="regbutton NRPS NRPS r3c2">
<a href="#r3c2">Region&nbsp3.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">622,779</td>
<td class="digits table-split-left">687,796</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000354/1" target="_blank">friulimicin A/friulimicin B/friulimicin C/friulimicin D</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 15%, #ffffff00 15%)">15%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c3">
<td class="regbutton RiPP lanthipeptide-class-iii r3c3">
<a href="#r3c3">Region&nbsp3.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iii" target="_blank">lanthipeptide-class-iii</a>
</td>
<td class="digits">750,481</td>
<td class="digits table-split-left">773,255</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000501/1" target="_blank">catenulipeptin</a></td>
<td>RiPP:Lanthipeptide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(210, 105, 30, 0.3), rgba(210, 105, 30, 0.3) 60%, #ffffff00 60%)">60%</td>

</tr>
<tr class="linked-row even" data-anchor="#r3c4">
<td class="regbutton NRPS NAPAA r3c4">
<a href="#r3c4">Region&nbsp3.4</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#napaa" target="_blank">NAPAA</a>
</td>
<td class="digits">916,608</td>
<td class="digits table-split-left">950,510</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002174/1" target="_blank">ε-Poly-L-lysine</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r3c5">
<td class="regbutton hybrid r3c5">
<a href="#r3c5">Region&nbsp3.5</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#lanthipeptide-class-iv" target="_blank">lanthipeptide-class-iv</a>
</td>
<td class="digits">972,257</td>
<td class="digits table-split-left">1,007,606</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000519/1" target="_blank">labyrinthopeptin A2/labyrinthopeptin A1/labyrinthopeptin A3</a></td>
<td>RiPP:Lanthipeptide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 40%, #ffffff00 40%)">40%</td>

</tr>


<tr class="linked-row even" data-anchor="#r4c1">
<td class="regbutton NRPS r4c1">
<a href="#r4c1">Region&nbsp4.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps-like" target="_blank">NRPS-like</a>
</td>
<td class="digits">86,408</td>
<td class="digits table-split-left">134,411</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002051/1" target="_blank">antipain</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r4c2">
<td class="regbutton hybrid r4c2">
<a href="#r4c2">Region&nbsp4.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#pks-like" target="_blank">PKS-like</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#butyrolactone" target="_blank">butyrolactone</a>
</td>
<td class="digits">244,100</td>
<td class="digits table-split-left">285,164</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002098/1" target="_blank">ECO-0501</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 4%, #ffffff00 4%)">4%</td>

</tr>


<tr class="linked-row even" data-anchor="#r5c1">
<td class="regbutton NRPS NRPS r5c1">
<a href="#r5c1">Region&nbsp5.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">377,573</td>
<td class="digits table-split-left">426,460</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002001/1" target="_blank">crochelin A</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 12%, #ffffff00 12%)">12%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r5c2">
<td class="regbutton hybrid r5c2">
<a href="#r5c2">Region&nbsp5.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#terpene" target="_blank">terpene</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">590,054</td>
<td class="digits table-split-left">681,562</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000661/1" target="_blank">geosmin</a></td>
<td>Terpene</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row even" data-anchor="#r5c3">
<td class="regbutton hybrid r5c3">
<a href="#r5c3">Region&nbsp5.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">706,575</td>
<td class="digits table-split-left">757,169</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001818/1" target="_blank">vazabitide A</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r6c1">
<td class="regbutton other hydrogen-cyanide r6c1">
<a href="#r6c1">Region&nbsp6.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#hydrogen-cyanide" target="_blank">hydrogen-cyanide</a>
</td>
<td class="digits">67,500</td>
<td class="digits table-split-left">80,656</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002285/1" target="_blank">aborycin</a></td>
<td>RiPP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 21%, #ffffff00 21%)">21%</td>

</tr>
<tr class="linked-row even" data-anchor="#r6c2">
<td class="regbutton PKS T3PKS r6c2">
<a href="#r6c2">Region&nbsp6.2</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t3pks" target="_blank">T3PKS</a>
</td>
<td class="digits">119,067</td>
<td class="digits table-split-left">160,176</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0000282/1" target="_blank">alkylresorcinol</a></td>
<td>Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(0, 100, 0, 0.3), rgba(0, 100, 0, 0.3) 100%, #ffffff00 100%)">100%</td>

</tr>
<tr class="linked-row odd" data-anchor="#r6c3">
<td class="regbutton hybrid r6c3">
<a href="#r6c3">Region&nbsp6.3</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>,<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#t1pks" target="_blank">T1PKS</a>
</td>
<td class="digits">417,280</td>
<td class="digits table-split-left">449,361</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0001332/1" target="_blank">microtermolide A</a></td>
<td>NRP+Polyketide</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 13%, #ffffff00 13%)">13%</td>

</tr>


<tr class="linked-row even" data-anchor="#r8c1">
<td class="regbutton other NI-siderophore r8c1">
<a href="#r8c1">Region&nbsp8.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#ni-siderophore" target="_blank">NI-siderophore</a>
</td>
<td class="digits">10,930</td>
<td class="digits table-split-left">42,204</td>




<td><a class="external-link" href="https://mibig.secondarymetabolites.org/go/BGC0002466/1" target="_blank">peucechelin</a></td>
<td>NRP</td>
<td class="digits similarity-text" style="background-image: linear-gradient(to left, rgba(205, 92, 92, 0.3), rgba(205, 92, 92, 0.3) 10%, #ffffff00 10%)">10%</td>

</tr>


<tr class="linked-row odd" data-anchor="#r9c1">
<td class="regbutton NRPS NRPS r9c1">
<a href="#r9c1">Region&nbsp9.1</a>
</td>
<td>

<a class="external-link" href="https://docs.antismash.secondarymetabolites.org/glossary/#nrps" target="_blank">NRPS</a>
</td>
<td class="digits">1</td>
<td class="digits table-split-left">2,803</td>

<td colspan="3"></td>

</tr>


</tbody>
</table>
</div>
</div>


<dl id="regionless-records">
<dt class="record-overview-header">No secondary metabolite regions were found in these records:</dt>

<dd><strong>JABTEZ010000007.1</strong> (Planomonospora sp. ID67723)</dd>

</dl>

</div>

<div class="overview-switches">
<div class="overview-switch-compact switch-container"><span class="switch-desc">Compact view</span> <label class="switch">  <input class="overview-switch-compact" type="checkbox">  <span class="slider"></span> </label></div>
</div>

</div>
</div>








<div class="page" id='r1c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000001.1 - Region 1 - RiPP-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-234"></div> <span class="help-tooltip" id="region_svg-help-234">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000001.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c1-svg" data-filename="JABTEZ010000001.1_r1c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 62,410 - 73,276 nt. (total: 10,867 nt)

<a class="cluster-rules-header" id="r1c1-rules-header" href="#r1c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c1-rules">RiPP-like: (strepbact or Antimicrobial14 or Bacteriocin_IId or BacteriocIIc_cy or Bacteriocin_II or Bacteriocin_IIi or Lactococcin or Antimicrobial17 or Lactococcin_972 or Bacteriocin_IIc or LcnG-beta or Cloacin or Linocin_M18 or TIGR03651 or TIGR03693 or TIGR03601 or TIGR03795 or TIGR03975 or DUF692 or TIGR01193 or (YcaO or TIGR03604))</div>

<div class="region-svg-container">
<div id='r1c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-235"></div> <span class="help-tooltip" id="focus-panel-help-235">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c1-gene-table-details" data-name="r1c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c1-MIBiG-cluster-compare" data-name="r1c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c1-knownclusterblast" data-name="r1c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c1-subclusterblast" data-name="r1c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c1-tfbs-finder" data-name="r1c1-tfbs-finder">
TFBS Finder
</div>


</div>


<div class="body-details-section r1c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-6"></div> <span class="help-tooltip" id="gt-help-tooltip-help-6">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-1"></div> <span class="help-tooltip" id="MIBiG-help-1">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c1"></div>
<div id="comparison-MIBiG-r1c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">RiPP-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002152: 347-13298">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002152/index.html#r1c1" target="_new">BGC0002152</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Other</td>

<td>toxoflavin</td>


<td>Pseudomonas protegens Pf-5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001323: 6416-74792">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001323/index.html#r1c1" target="_new">BGC0001323</a></td>





<td class="cc-heat-cell" title="0.27 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>


<td>0.07</td>
<td>Terpene</td>

<td>monoterpenes-diterpenes</td>


<td>Solanum pennellii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001321: 3449-149366">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001321/index.html#r1c1" target="_new">BGC0001321</a></td>





<td class="cc-heat-cell" title="0.24 (id:0.45, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>


<td>0.05</td>
<td>Terpene</td>

<td>monoterpenes-diterpenes</td>


<td>Solanum habrochaites</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000275: 58-12211">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000275/index.html#r1c1" target="_new">BGC0000275</a></td>





<td class="cc-heat-cell" title="0.21 (id:0.45, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>tetracenomycin C</td>


<td>Streptomyces glaucescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001301: 139-53189">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001301/index.html#r1c1" target="_new">BGC0001301</a></td>





<td class="cc-heat-cell" title="0.21 (id:0.46, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>


<td>0.05</td>
<td>Polyketide</td>

<td>anthracimycin</td>


<td>Streptomyces sp. T676</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001078: 694-27170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001078/index.html#r1c1" target="_new">BGC0001078</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.50, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>Terpene, Polyketide</td>

<td>furaquinocin B</td>


<td>Streptomyces sp. KO-3988</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001331: 0-34976">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001331/index.html#r1c1" target="_new">BGC0001331</a></td>





<td class="cc-heat-cell" title="0.19 (id:0.42, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>


<td>0.04</td>
<td>NRP, Polyketide</td>

<td>BE-43547A1, BE-43547A2, BE-43547B1, BE-43547B2, BE-43547B3, BE-43547C1, BE-43547C2</td>


<td>Salinispora arenicola CNR107</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001322: 5529-122963">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001322/index.html#r1c1" target="_new">BGC0001322</a></td>





<td class="cc-heat-cell" title="0.18 (id:0.48, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>


<td>0.04</td>
<td>Terpene</td>

<td>lycosantalonol</td>


<td>Solanum lycopersicum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001324: 0-104987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001324/index.html#r1c1" target="_new">BGC0001324</a></td>





<td class="cc-heat-cell" title="0.18 (id:0.45, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>


<td>0.04</td>
<td>Terpene</td>

<td>monoterpenes-diterpenes</td>


<td>Solanum pimpinellifolium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000652: 0-35797">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000652/index.html#r1c1" target="_new">BGC0000652</a></td>





<td class="cc-heat-cell" title="0.17 (id:0.47, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>


<td>0.04</td>
<td>Terpene</td>

<td>napyradiomycin</td>


<td>Streptomyces sp. CNQ525</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002152: 347-13298">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002152/index.html#r1c1" target="_new">BGC0002152</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.43, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Other</td>
<td>toxoflavin</td>
<td>Pseudomonas protegens Pf-5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001323: 6416-74792">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001323/index.html#r1c1" target="_new">BGC0001323</a></td>


<td class="cc-heat-cell" title="0.27 (id:0.45, order:N/A, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.27), rgb(0, 0, 0, 0.27) 27%, white 27%)"></td>
<td>0.27</td>
<td>Terpene</td>
<td>monoterpenes-diterpenes</td>
<td>Solanum pennellii</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001321: 3449-149366">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001321/index.html#r1c1" target="_new">BGC0001321</a></td>


<td class="cc-heat-cell" title="0.24 (id:0.45, order:N/A, components:0.12)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.24), rgb(0, 0, 0, 0.24) 24%, white 24%)"></td>
<td>0.24</td>
<td>Terpene</td>
<td>monoterpenes-diterpenes</td>
<td>Solanum habrochaites</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000275: 58-12211">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000275/index.html#r1c1" target="_new">BGC0000275</a></td>


<td class="cc-heat-cell" title="0.21 (id:0.45, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
<td>0.21</td>
<td>Polyketide</td>
<td>tetracenomycin C</td>
<td>Streptomyces glaucescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001301: 139-53189">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001301/index.html#r1c1" target="_new">BGC0001301</a></td>


<td class="cc-heat-cell" title="0.21 (id:0.46, order:N/A, components:0.10)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
<td>0.21</td>
<td>Polyketide</td>
<td>anthracimycin</td>
<td>Streptomyces sp. T676</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001078: 694-27170">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001078/index.html#r1c1" target="_new">BGC0001078</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.50, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>Terpene, Polyketide</td>
<td>furaquinocin B</td>
<td>Streptomyces sp. KO-3988</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001331: 0-34976">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001331/index.html#r1c1" target="_new">BGC0001331</a></td>


<td class="cc-heat-cell" title="0.19 (id:0.42, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.19), rgb(0, 0, 0, 0.19) 19%, white 19%)"></td>
<td>0.19</td>
<td>NRP, Polyketide</td>
<td>BE-43547A1, BE-43547A2, BE-43547B1, BE-43547B2, BE-43547B3, BE-43547C1, BE-43547C2</td>
<td>Salinispora arenicola CNR107</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001322: 5529-122963">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001322/index.html#r1c1" target="_new">BGC0001322</a></td>


<td class="cc-heat-cell" title="0.18 (id:0.48, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
<td>0.18</td>
<td>Terpene</td>
<td>lycosantalonol</td>
<td>Solanum lycopersicum</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001324: 0-104987">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001324/index.html#r1c1" target="_new">BGC0001324</a></td>


<td class="cc-heat-cell" title="0.18 (id:0.45, order:N/A, components:0.07)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
<td>0.18</td>
<td>Terpene</td>
<td>monoterpenes-diterpenes</td>
<td>Solanum pimpinellifolium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000652: 0-35797">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000652/index.html#r1c1" target="_new">BGC0000652</a></td>


<td class="cc-heat-cell" title="0.17 (id:0.47, order:N/A, components:0.06)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.17), rgb(0, 0, 0, 0.17) 17%, white 17%)"></td>
<td>0.17</td>
<td>Terpene</td>
<td>napyradiomycin</td>
<td>Streptomyces sp. CNQ525</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-2"></div> <span class="help-tooltip" id="cb-known-help-2">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-3"></div> <span class="help-tooltip" id="cb-sub-help-3">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-5"></div> <span class="help-tooltip" id="tfbs-body-help-5">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
CatR
<div class="collapser collapser-target-CatR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: H2O2-responsive repressor<br>
<span>Score</span>: 20.30 out of 25.31
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r1c1-66893-CatR"></div>

<div class="tfbs-hit-detail">
DasR
<div class="collapser collapser-target-DasR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: N-acetylglucosamine dependent repressor<br>
<span>Score</span>: 22.07 out of 27.04
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r1c1-71188-DasR"></div>

<div class="tfbs-hit-detail">
DasR
<div class="collapser collapser-target-DasR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: N-acetylglucosamine dependent repressor<br>
<span>Score</span>: 22.63 out of 27.04
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r1c1-71352-DasR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 16.96 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c1-71302-ZuR"></div>

<div class="tfbs-hit-detail">
DasR
<div class="collapser collapser-target-DasR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: N-acetylglucosamine dependent repressor<br>
<span>Score</span>: 20.85 out of 27.04
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c1-71333-DasR"></div>

<div class="tfbs-hit-detail">
DasR
<div class="collapser collapser-target-DasR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: N-acetylglucosamine dependent repressor<br>
<span>Score</span>: 20.90 out of 27.04
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c1-71374-DasR"></div>

</div>
</div>
</div>

</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c1-tfbs-finder" data-name="r1c1-tfbs-finder">
TFBS Finder
</div>


</div>










<div class="sidepanel-details-section r1c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-4"></div> <span class="help-tooltip" id="tfbs-side-help-4">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CatR" target="_blank">CatR</a></td>
<td>H2O2-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DasR" target="_blank">DasR</a></td>
<td>N-acetylglucosamine dependent repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>


</div>

</div>

</div>

<div class="page" id='r1c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000001.1 - Region 2 - T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-236"></div> <span class="help-tooltip" id="region_svg-help-236">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000001.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c2-svg" data-filename="JABTEZ010000001.1_r1c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 241,518 - 287,829 nt. (total: 46,312 nt)

<a class="cluster-rules-header" id="r1c2-rules-header" href="#r1c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c2-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r1c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-237"></div> <span class="help-tooltip" id="focus-panel-help-237">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c2-gene-table-details" data-name="r1c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c2-nrps_pks" data-name="r1c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r1c2-MIBiG-cluster-compare" data-name="r1c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c2-knownclusterblast" data-name="r1c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c2-subclusterblast" data-name="r1c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c2-tfbs-finder" data-name="r1c2-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r1c2-nrps-pks-bubbles" data-name="r1c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r1c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-16"></div> <span class="help-tooltip" id="gt-help-tooltip-help-16">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-7"></div> <span class="help-tooltip" id="nrps-domain-help-7">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-details-svg" data-filename="JABTEZ010000001.1_r1c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r1c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r1c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-8"></div> <span class="help-tooltip" id="MIBiG-help-8">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c2"></div>
<div id="comparison-MIBiG-r1c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Polyketide</td>

<td>strobilurin A</td>


<td>Strobilurus tenacellus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001647: 0-18309">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001647/index.html#r1c1" target="_new">BGC0001647</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>lagunapyrone A, lagunapyrone B, lagunapyrone C</td>


<td>Streptomyces sp. MP131-18</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001895: 425-18341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001895/index.html#r1c1" target="_new">BGC0001895</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>huanglongmycin A, huanglongmycin B, huanglongmycin C</td>


<td>Streptomyces sp. CB09001</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002285: 918-14445">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002285/index.html#r1c1" target="_new">BGC0002285</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>RiPP</td>

<td>aborycin</td>


<td>Streptomyces sp. ZS0098</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001964: 2-41089">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001964/index.html#r1c1" target="_new">BGC0001964</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>alkylpyrone-407, alkylpyrone-393</td>


<td>Cystobacterineae bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000706: 350-14906">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000706/index.html#r1c1" target="_new">BGC0000706</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.51, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Saccharide</td>

<td>kanamycin</td>


<td>Streptomyces kanamyceticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831: 1-6152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Polyketide</td>

<td>alkylpyrone-407, alkylpyrone-393</td>


<td>Myxococcus xanthus DK 1622</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000019: 339-17245">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000019/index.html#r1c1" target="_new">BGC0000019</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.52, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>angolamycin</td>


<td>Streptomyces eurythermus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001441: 0-27983">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001441/index.html#r1c1" target="_new">BGC0001441</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Other</td>

<td>belactosin A, belactosin C</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001302: 0-25686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001302/index.html#r1c1" target="_new">BGC0001302</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Other</td>

<td>lomofungin</td>


<td>Streptomyces lomondensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001909: 2476-67696">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001909/index.html#r1c1" target="_new">BGC0001909</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>strobilurin A</td>
<td>Strobilurus tenacellus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001647: 0-18309">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001647/index.html#r1c1" target="_new">BGC0001647</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>lagunapyrone A, lagunapyrone B, lagunapyrone C</td>
<td>Streptomyces sp. MP131-18</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001895: 425-18341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001895/index.html#r1c1" target="_new">BGC0001895</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.46, order:N/A, components:0.82)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>huanglongmycin A, huanglongmycin B, huanglongmycin C</td>
<td>Streptomyces sp. CB09001</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002285: 918-14445">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002285/index.html#r1c1" target="_new">BGC0002285</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.43, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>RiPP</td>
<td>aborycin</td>
<td>Streptomyces sp. ZS0098</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001964: 2-41089">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001964/index.html#r1c1" target="_new">BGC0001964</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.77)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Polyketide</td>
<td>alkylpyrone-407, alkylpyrone-393</td>
<td>Cystobacterineae bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000706: 350-14906">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000706/index.html#r1c1" target="_new">BGC0000706</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.51, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Saccharide</td>
<td>kanamycin</td>
<td>Streptomyces kanamyceticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831: 1-6152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>alkylpyrone-407, alkylpyrone-393</td>
<td>Myxococcus xanthus DK 1622</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000019: 339-17245">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000019/index.html#r1c1" target="_new">BGC0000019</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.52, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Polyketide</td>
<td>angolamycin</td>
<td>Streptomyces eurythermus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001441: 0-27983">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001441/index.html#r1c1" target="_new">BGC0001441</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>belactosin A, belactosin C</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001302: 0-25686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001302/index.html#r1c1" target="_new">BGC0001302</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>lomofungin</td>
<td>Streptomyces lomondensis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-9"></div> <span class="help-tooltip" id="cb-known-help-9">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r1c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r1c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r1c2_1.svg">
K-252a
</option>

<option value = "svg/knownclusterblast_r1c2_2.svg">
K-252a
</option>

</select>
<button id="knownclusterblast-r1c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r1c2-svg"></div>

</div>
</div>

<div class="body-details-section r1c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-10"></div> <span class="help-tooltip" id="cb-sub-help-10">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c2-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-14"></div> <span class="help-tooltip" id="tfbs-body-help-14">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AfsQ1
<div class="collapser collapser-target-AfsQ1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Two-component system AfsQ1-Q2, activator of antibiotic production<br>
<span>Score</span>: 18.33 out of 19.52
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r1c2-285939-AfsQ1"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 16.93 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c2-243812-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r1c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-15"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-15">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r1c2-domain-bubble-svg" data-filename="JABTEZ010000001.1_r1c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r1c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (241517 - 287829): single T1PKS
</option>


</select>

<div class="details-svg" id="r1c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r1c2-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r1c2-nrps_pks_products" data-name="r1c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r1c2-nrps_pks_monomers" data-name="r1c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r1c2-tfbs-finder" data-name="r1c2-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r1c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-11"></div> <span class="help-tooltip" id="nrps-structures-help-11">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 241517 - 287829:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r1c2-2" class="smiles-canvas" data-smiles="C([*])C(-O)"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(pk)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r1c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C([*])C(-O)"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r1c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-12"></div> <span class="help-tooltip" id="nrps-monomers-help-12">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">HS041_01240</span></strong>: pk
<div class="collapser collapser-target-HS041_01240 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>







<dd><strong>PKS_AT (624..908)</strong>: pk
<div class="collapser collapser-target-HS041_01240 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-HS041_01240-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 66.7%</dd>
<dd>Methylmalonyl-CoA: 62.5%</dd>
<dd>Ethylmalonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Methylmalonyl-CoA
<div class="collapser collapser-target-HS041_01240-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Methylmalonyl-CoA: 114.1</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 99.2</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 98.8</dt>

<dd></dd><dt>Malonyl-CoA: 94.9</dt>

<dd></dd><dt>fatty_acid: 70.0</dt>

<dd></dd><dt>Propionyl-CoA: 64.2</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 54.9</dt>

<dd></dd><dt>Isobutyryl-CoA: 49.3</dt>

<dd></dd><dt>trans-1,2-CPDA: 46.2</dt>

<dd></dd><dt>Acetyl-CoA: 44.9</dt>

<dd></dd><dt>CHC-CoA: 44.5</dt>

<dd></dd><dt>inactive: 32.5</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 17.7</dt>

<dd></dd><dt>Benzoyl-CoA: 13.1</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

<strong><span class="serif">HS041_01270</span></strong>: pk
<div class="collapser collapser-target-HS041_01270 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>CAL_domain (158..480)</strong>: pk
<div class="collapser collapser-target-HS041_01270 collapser-level-candidate"> </div><div class="collapser-content">

minowa:  NH2
<div class="collapser collapser-target-HS041_01270-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>NH2: 106.0</dt>

<dd></dd><dt>Acetyl-CoA: 52.0</dt>

<dd></dd><dt>AHBA: 21.2</dt>

<dd></dd><dt>fatty_acid: 16.8</dt>

<dd></dd><dt>shikimic_acid: -3.1</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r1c2-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-13"></div> <span class="help-tooltip" id="tfbs-side-help-13">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsQ1" target="_blank">AfsQ1</a></td>
<td>Two-component system AfsQ1-Q2, activator of antibiotic production</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r1c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000001.1 - Region 3 - lanthipeptide-class-iv <div class="help-container"> <div class="help-icon" data-id="region_svg-help-238"></div> <span class="help-tooltip" id="region_svg-help-238">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000001.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c3-svg" data-filename="JABTEZ010000001.1_r1c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,133,085 - 1,155,793 nt. (total: 22,709 nt)

<a class="cluster-rules-header" id="r1c3-rules-header" href="#r1c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c3-rules">lanthipeptide-class-iv: (cds(LANC_like and Pkinase) and not micKC)</div>

<div class="region-svg-container">
<div id='r1c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-239"></div> <span class="help-tooltip" id="focus-panel-help-239">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c3-gene-table-details" data-name="r1c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c3-MIBiG-cluster-compare" data-name="r1c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c3-knownclusterblast" data-name="r1c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c3-subclusterblast" data-name="r1c3-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r1c3-lanthipeptides" data-name="r1c3-lanthipeptides">
Lanthipeptides
</div>





<div class="body-details-header r1c3-tfbs-finder" data-name="r1c3-tfbs-finder">
TFBS Finder
</div>


</div>


<div class="body-details-section r1c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-24"></div> <span class="help-tooltip" id="gt-help-tooltip-help-24">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-17"></div> <span class="help-tooltip" id="MIBiG-help-17">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c3"></div>
<div id="comparison-MIBiG-r1c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">lanthipeptide-class-iv</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001226: 0-6018">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001226/index.html#r1c1" target="_new">BGC0001226</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>RiPP</td>

<td>streptocollin</td>


<td>Streptomyces collinus Tu 365</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563: 185-5330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>RiPP</td>

<td>venezuelin</td>


<td>Streptomyces venezuelae ATCC 10712</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>RiPP</td>

<td>SapB</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501: 0-7208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>RiPP</td>

<td>catenulipeptin</td>


<td>Catenulispora acidiphila DSM 44928</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496: 0-7345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>RiPP</td>

<td>AmfS</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513: 0-6203">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>RiPP</td>

<td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>


<td>Saccharopolyspora erythraea NRRL 2338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.21</td>
<td>RiPP</td>

<td>SRO15-3108</td>


<td>Streptomyces filamentosus NRRL 15998</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002313: 161-9914">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002313/index.html#r1c1" target="_new">BGC0002313</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Other</td>

<td>streptozotocin</td>


<td>Streptomyces achromogenes subsp. streptozoticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001210: 0-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001210/index.html#r1c1" target="_new">BGC0001210</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>RiPP</td>

<td>pseudomycoicidin</td>


<td>Bacillus pseudomycoides DSM 12442</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500: 0-8863">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500</a></td>





<td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>


<td>0.19</td>
<td>RiPP</td>

<td>carnolysin A1, carnolysin A2</td>


<td>Carnobacterium maltaromaticum</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001226: 0-6018">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001226/index.html#r1c1" target="_new">BGC0001226</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>RiPP</td>
<td>streptocollin</td>
<td>Streptomyces collinus Tu 365</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563: 185-5330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>RiPP</td>
<td>venezuelin</td>
<td>Streptomyces venezuelae ATCC 10712</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.47, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>RiPP</td>
<td>SapB</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501: 0-7208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>RiPP</td>
<td>catenulipeptin</td>
<td>Catenulispora acidiphila DSM 44928</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496: 0-7345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>RiPP</td>
<td>AmfS</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513: 0-6203">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.44, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>RiPP</td>
<td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>
<td>Saccharopolyspora erythraea NRRL 2338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>RiPP</td>
<td>SRO15-3108</td>
<td>Streptomyces filamentosus NRRL 15998</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002313: 161-9914">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002313/index.html#r1c1" target="_new">BGC0002313</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.63)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Other</td>
<td>streptozotocin</td>
<td>Streptomyces achromogenes subsp. streptozoticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001210: 0-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001210/index.html#r1c1" target="_new">BGC0001210</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>RiPP</td>
<td>pseudomycoicidin</td>
<td>Bacillus pseudomycoides DSM 12442</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000500: 0-8863">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000500/index.html#r1c1" target="_new">BGC0000500</a></td>


<td class="cc-heat-cell" title="0.51 (id:0.41, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.51), rgb(0, 0, 0, 0.51) 51%, white 51%)"></td>
<td>0.51</td>
<td>RiPP</td>
<td>carnolysin A1, carnolysin A2</td>
<td>Carnobacterium maltaromaticum</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-18"></div> <span class="help-tooltip" id="cb-known-help-18">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-19"></div> <span class="help-tooltip" id="cb-sub-help-19">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r1c3-lanthipeptides">
<div class="details">
<div class="heading">
<span>Lanthipeptide predictions</span>
<div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-20"></div> <span class="help-tooltip" id="lanthi-body-help-20">Shows the possible core peptides for each biosynthetic enzyme.<br>Includes CompaRiPPson results for any available databases, with a detailed explanation <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/comparippson' target='_blank'>here</a>.</span></div>
</div>
<div class="details-text ripp-body">

Lanthipeptides - No core peptides found.

</div>
</div>
</div>





<div class="body-details-section r1c3-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-23"></div> <span class="help-tooltip" id="tfbs-body-help-23">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (5):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
CsoR
<div class="collapser collapser-target-CsoR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Copper-responsive repressor<br>
<span>Score</span>: 22.87 out of 32.38
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c3-1134440-CsoR"></div>

<div class="tfbs-hit-detail">
HypR
<div class="collapser collapser-target-HypR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: L-hydroxyproline utilization repressor<br>
<span>Score</span>: 18.59 out of 22.40
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c3-1135793-HypR"></div>

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 22.67 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c3-1139893-LexA"></div>

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 22.51 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c3-1145973-NuR"></div>

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 20.32 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c3-1155605-NuR"></div>

</div>
</div>
</div>

</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r1c3-lanthipeptides" data-name="r1c3-lanthipeptides">
Lanthipeptides
</div>





<div class="sidepanel-details-header r1c3-tfbs-finder" data-name="r1c3-tfbs-finder">
TFBS Finder
</div>


</div>








<div class="sidepanel-details-section r1c3-lanthipeptides">
<div class="more-details">
<div class="heading">
<span>Core peptide predictions</span>
<div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-21"></div> <span class="help-tooltip" id="lanthi-side-help-21">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
</div>

<div class="prediction-text">No core peptides found.</div>

</div>

</div>





<div class="sidepanel-details-section r1c3-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-22"></div> <span class="help-tooltip" id="tfbs-side-help-22">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CsoR" target="_blank">CsoR</a></td>
<td>Copper-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/HypR" target="_blank">HypR</a></td>
<td>L-hydroxyproline utilization repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>


</div>

</div>

</div>

<div class="page" id='r1c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000001.1 - Region 4 - LAP,thiopeptide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-240"></div> <span class="help-tooltip" id="region_svg-help-240">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000001.1.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c4-svg" data-filename="JABTEZ010000001.1_r1c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,742,150 - 1,772,558 nt. (total: 30,409 nt)

<a class="cluster-rules-header" id="r1c4-rules-header" href="#r1c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c4-rules">thiopeptide: ((YcaO or TIGR03604) and ((thio_amide and (PF06968 or PF04055 or PF07366)) or Lant_dehydr_C or Lant_dehydr_N or PF07366 or PF06968 or PF04055) or thiostrepton)<br>LAP: ((((goadsporin_like or PF00881 or TIGR03605) and (YcaO or TIGR03604)) or TIGR03882 or TIGR03603 or TIGR03602) and not cyanobactin_synth)</div>

<div class="region-svg-container">
<div id='r1c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-241"></div> <span class="help-tooltip" id="focus-panel-help-241">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c4-gene-table-details" data-name="r1c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c4-MIBiG-cluster-compare" data-name="r1c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c4-knownclusterblast" data-name="r1c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c4-subclusterblast" data-name="r1c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r1c4-tfbs-finder" data-name="r1c4-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r1c4-thiopeptides" data-name="r1c4-thiopeptides">
Thiopeptides
</div>


</div>


<div class="body-details-section r1c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-33"></div> <span class="help-tooltip" id="gt-help-tooltip-help-33">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-25"></div> <span class="help-tooltip" id="MIBiG-help-25">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c4"></div>
<div id="comparison-MIBiG-r1c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">thiopeptide</span></th>

<th><span class="cc-heat-product">LAP</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002311: 0-5945">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002311/index.html#r1c1" target="_new">BGC0002311</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.70</td>
<td>RiPP</td>

<td>cutimycin</td>


<td>Cutibacterium acnes KPA171202</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001472: 1-12862">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001472/index.html#r1c1" target="_new">BGC0001472</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>RiPP</td>

<td>berninamycin A</td>


<td>Streptomyces bernensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000605: 0-17179">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000605/index.html#r1c1" target="_new">BGC0000605</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.58</td>
<td>RiPP</td>

<td>GE37468</td>


<td>Streptomyces sp. ATCC 55365</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000568: 0-5145">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000568/index.html#r1c1" target="_new">BGC0000568</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.55</td>
<td>RiPP</td>

<td>microcin B17</td>


<td>Escherichia coli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002144: 0-5515">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002144/index.html#r1c1" target="_new">BGC0002144</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>RiPP</td>

<td>phazolicin</td>


<td>Rhizobium sp. Pop5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000628: 0-13800">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000628/index.html#r1c1" target="_new">BGC0000628</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.52</td>
<td>RiPP</td>

<td>lactocillin</td>


<td>Lactobacillus paragasseri JV-V03</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001173: 0-9945">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001173/index.html#r1c1" target="_new">BGC0001173</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>RiPP</td>

<td>plantazolicin</td>


<td>Bacillus pumilus ATCC 7061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000569: 0-9892">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000569/index.html#r1c1" target="_new">BGC0000569</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>RiPP</td>

<td>plantazolicin</td>


<td>Bacillus velezensis FZB42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000566: 0-8290">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000566/index.html#r1c1" target="_new">BGC0000566</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.47</td>
<td>RiPP</td>

<td>streptolysin S</td>


<td>Streptococcus pyogenes M1 GAS</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002115: 0-14439">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002115/index.html#r1c1" target="_new">BGC0002115</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.46</td>
<td>RiPP</td>

<td>streptamidine</td>


<td>Streptomyces albidoflavus J1074</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002311: 0-5945">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002311/index.html#r1c1" target="_new">BGC0002311</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>RiPP</td>
<td>cutimycin</td>
<td>Cutibacterium acnes KPA171202</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001472: 1-12862">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001472/index.html#r1c1" target="_new">BGC0001472</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>RiPP</td>
<td>berninamycin A</td>
<td>Streptomyces bernensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000605: 0-17179">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000605/index.html#r1c1" target="_new">BGC0000605</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>RiPP</td>
<td>GE37468</td>
<td>Streptomyces sp. ATCC 55365</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000568: 0-5145">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000568/index.html#r1c1" target="_new">BGC0000568</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>RiPP</td>
<td>microcin B17</td>
<td>Escherichia coli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002144: 0-5515">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002144/index.html#r1c1" target="_new">BGC0002144</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>RiPP</td>
<td>phazolicin</td>
<td>Rhizobium sp. Pop5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000628: 0-13800">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000628/index.html#r1c1" target="_new">BGC0000628</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>RiPP</td>
<td>lactocillin</td>
<td>Lactobacillus paragasseri JV-V03</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001173: 0-9945">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001173/index.html#r1c1" target="_new">BGC0001173</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.43, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>RiPP</td>
<td>plantazolicin</td>
<td>Bacillus pumilus ATCC 7061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000569: 0-9892">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000569/index.html#r1c1" target="_new">BGC0000569</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>RiPP</td>
<td>plantazolicin</td>
<td>Bacillus velezensis FZB42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000566: 0-8290">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000566/index.html#r1c1" target="_new">BGC0000566</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>RiPP</td>
<td>streptolysin S</td>
<td>Streptococcus pyogenes M1 GAS</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002115: 0-14439">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002115/index.html#r1c1" target="_new">BGC0002115</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>RiPP</td>
<td>streptamidine</td>
<td>Streptomyces albidoflavus J1074</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-26"></div> <span class="help-tooltip" id="cb-known-help-26">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-27"></div> <span class="help-tooltip" id="cb-sub-help-27">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r1c4-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-30"></div> <span class="help-tooltip" id="tfbs-body-help-30">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
DasR
<div class="collapser collapser-target-DasR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: N-acetylglucosamine dependent repressor<br>
<span>Score</span>: 21.19 out of 27.04
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r1c4-1748927-DasR"></div>

</div>


</div>
</div>



<div class="body-details-section r1c4-thiopeptides">
<div class="details">
<div class="heading">
<span>Thiopeptide predictions</span>
<div class="help-container"> <div class="help-icon" data-id="thio-body-help-31"></div> <span class="help-tooltip" id="thio-body-help-31">Lists the possible core peptides for each biosynthetic enzyme. Predicted tail sequences are also shown, if present. <br>Includes CompaRiPPson results for any available databases, with a detailed explanation <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/comparippson' target='_blank'>here</a>.</span></div>
</div>
<div class="details-text ripp-body">

No core peptides found.

</div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r1c4-RREfinder" data-name="r1c4-RREfinder">
RREFinder
</div>



<div class="sidepanel-details-header r1c4-tfbs-finder" data-name="r1c4-tfbs-finder">
TFBS Finder
</div>



<div class="sidepanel-details-header r1c4-thiopeptides" data-name="r1c4-thiopeptides">
Thiopeptides
</div>


</div>








<div class="sidepanel-details-section r1c4-RREfinder">
<div class="more-details">
<div class="heading">
<span>RRE predictions</span>
<div class="help-container"> <div class="help-icon" data-id="rre-side-help-28"></div> <span class="help-tooltip" id="rre-side-help-28">RREfinder results sidepanel.</span></div>
</div>
<dl class = 'prediction-text'>

<span><strong>thiopeptide protocluster</strong> (1742149...1772558)</span>
<dt></dt>




<strong>HS041_08435</strong> <div class="collapser collapser-target-HS041_08435 collapser-level-cds"> </div><div class="collapser-content">

<dd>RRE type: Thiopeptide_F_RRE</dd>
<dd>Location: 24 - 104</dd>
<dd>Score: 29.8</dd>

</div><br>
<br>

<span><strong>LAP protocluster</strong> (1742149...1766545)</span>
<dt></dt>




<strong>HS041_08435</strong> <div class="collapser collapser-target-HS041_08435 collapser-level-cds"> </div><div class="collapser-content">

<dd>RRE type: Thiopeptide_F_RRE</dd>
<dd>Location: 24 - 104</dd>
<dd>Score: 29.8</dd>

</div><br>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r1c4-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-29"></div> <span class="help-tooltip" id="tfbs-side-help-29">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DasR" target="_blank">DasR</a></td>
<td>N-acetylglucosamine dependent repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>



<div class="sidepanel-details-section r1c4-thiopeptides">
<div class="more-details">
<div class="heading">
<span>Core peptide predictions</span>
<div class="help-container"> <div class="help-icon" data-id="thio-side-help-32"></div> <span class="help-tooltip" id="thio-side-help-32">Lists the possible core peptides in the region. Each core peptide lists its possible molecular weights and the scores for cleavage site prediction and RODEO. If relevant, other features, such as macrocycle and amidation, will also be listed.</span></div>
</div>

<div class="prediction-text">No core peptides found.</div>

</div>
</div>


</div>

</div>

</div>

<div class="page" id='r1c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000001.1 - Region 5 - other <div class="help-container"> <div class="help-icon" data-id="region_svg-help-242"></div> <span class="help-tooltip" id="region_svg-help-242">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000001.1.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r1c5-svg" data-filename="JABTEZ010000001.1_r1c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 2,270,695 - 2,311,132 nt. (total: 40,438 nt)

<a class="cluster-rules-header" id="r1c5-rules-header" href="#r1c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r1c5-rules">other: (LmbU or Neocarzinostat or bcpB or frbD or mitE or vlmB or prnB or CaiA or bacilysin or orf2_PTase)</div>

<div class="region-svg-container">
<div id='r1c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-243"></div> <span class="help-tooltip" id="focus-panel-help-243">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r1c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r1c5-gene-table-details" data-name="r1c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r1c5-MIBiG-cluster-compare" data-name="r1c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r1c5-knownclusterblast" data-name="r1c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r1c5-subclusterblast" data-name="r1c5-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r1c5-tfbs-finder" data-name="r1c5-tfbs-finder">
TFBS Finder
</div>


</div>


<div class="body-details-section r1c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-39"></div> <span class="help-tooltip" id="gt-help-tooltip-help-39">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r1c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r1c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r1c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r1c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r1c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-34"></div> <span class="help-tooltip" id="MIBiG-help-34">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r1c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r1c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r1c5"></div>
<div id="comparison-MIBiG-r1c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">other</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002571: 103-67235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002571/index.html#r1c1" target="_new">BGC0002571</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.47, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>NRP</td>

<td>pyoverdine DC3000</td>


<td>Pseudomonas syringae pv. tomato str. DC3000</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002621: 254-8993">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002621/index.html#r1c1" target="_new">BGC0002621</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Alkaloid</td>

<td>angustmycin A, angustmycin B, angustmycin C, aristeromycin, arabinofuranosyladenine</td>


<td>Streptomyces angustmyceticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002407: 0-6138">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002407/index.html#r1c1" target="_new">BGC0002407</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>Saccharide, Other</td>

<td>angustmycin A, angustmycin C</td>


<td>Streptomyces angustmyceticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001810: 122-16114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001810/index.html#r1c1" target="_new">BGC0001810</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.47, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Polyketide</td>

<td>trehangelin</td>


<td>Polymorphospora rubra</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001972: 267-32147">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001972/index.html#r1c1" target="_new">BGC0001972</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Other</td>

<td>toxoflavin, fervenulin</td>


<td>Streptomyces hiroshimensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001959: 1219-11580">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001959/index.html#r1c1" target="_new">BGC0001959</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Other</td>

<td>taylorflavin A, taylorflavin B</td>


<td>Streptomyces hiroshimensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001364: 3-17587">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001364/index.html#r1c1" target="_new">BGC0001364</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.45, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>RiPP</td>

<td>cinnamycin B</td>


<td>Actinomadura atramentaria DSM 43919</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001441: 0-27983">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001441/index.html#r1c1" target="_new">BGC0001441</a></td>





<td class="cc-heat-cell" title="0.43 (id:0.47, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>


<td>0.14</td>
<td>Other</td>

<td>belactosin A, belactosin C</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002016: 0-28409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002016/index.html#r1c1" target="_new">BGC0002016</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.46, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>Polyketide</td>

<td>lugdunomycin</td>


<td>Streptomyces sp. QL37</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001513: 120-11698">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001513/index.html#r1c1" target="_new">BGC0001513</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.43, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Polyketide</td>

<td>anthrabenzoxocinone</td>


<td>Actinobacteria bacterium</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r1c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002571: 103-67235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002571/index.html#r1c1" target="_new">BGC0002571</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.47, order:N/A, components:0.64)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>NRP</td>
<td>pyoverdine DC3000</td>
<td>Pseudomonas syringae pv. tomato str. DC3000</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002621: 254-8993">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002621/index.html#r1c1" target="_new">BGC0002621</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Alkaloid</td>
<td>angustmycin A, angustmycin B, angustmycin C, aristeromycin, arabinofuranosyladenine</td>
<td>Streptomyces angustmyceticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002407: 0-6138">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002407/index.html#r1c1" target="_new">BGC0002407</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.52, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Saccharide, Other</td>
<td>angustmycin A, angustmycin C</td>
<td>Streptomyces angustmyceticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001810: 122-16114">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001810/index.html#r1c1" target="_new">BGC0001810</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.47, order:N/A, components:0.44)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Polyketide</td>
<td>trehangelin</td>
<td>Polymorphospora rubra</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001972: 267-32147">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001972/index.html#r1c1" target="_new">BGC0001972</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.45, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Other</td>
<td>toxoflavin, fervenulin</td>
<td>Streptomyces hiroshimensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001959: 1219-11580">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001959/index.html#r1c1" target="_new">BGC0001959</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.44, order:N/A, components:N/A)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Other</td>
<td>taylorflavin A, taylorflavin B</td>
<td>Streptomyces hiroshimensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001364: 3-17587">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001364/index.html#r1c1" target="_new">BGC0001364</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.45, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>RiPP</td>
<td>cinnamycin B</td>
<td>Actinomadura atramentaria DSM 43919</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001441: 0-27983">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001441/index.html#r1c1" target="_new">BGC0001441</a></td>


<td class="cc-heat-cell" title="0.43 (id:0.47, order:N/A, components:0.39)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.43), rgb(0, 0, 0, 0.43) 43%, white 43%)"></td>
<td>0.43</td>
<td>Other</td>
<td>belactosin A, belactosin C</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002016: 0-28409">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002016/index.html#r1c1" target="_new">BGC0002016</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.46, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>Polyketide</td>
<td>lugdunomycin</td>
<td>Streptomyces sp. QL37</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001513: 120-11698">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001513/index.html#r1c1" target="_new">BGC0001513</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.43, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Polyketide</td>
<td>anthrabenzoxocinone</td>
<td>Actinobacteria bacterium</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r1c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-35"></div> <span class="help-tooltip" id="cb-known-help-35">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r1c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-36"></div> <span class="help-tooltip" id="cb-sub-help-36">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r1c5-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-38"></div> <span class="help-tooltip" id="tfbs-body-help-38">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ArgR
<div class="collapser collapser-target-ArgR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Regulator of arginine biosynthesis genes<br>
<span>Score</span>: 18.61 out of 20.01
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r1c5-2285408-ArgR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.21 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c5-2270937-ZuR"></div>

<div class="tfbs-hit-detail">
ArgR
<div class="collapser collapser-target-ArgR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Regulator of arginine biosynthesis genes<br>
<span>Score</span>: 17.17 out of 20.01
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r1c5-2278191-ArgR"></div>

</div>
</div>
</div>

</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r1c5-tfbs-finder" data-name="r1c5-tfbs-finder">
TFBS Finder
</div>


</div>








<div class="sidepanel-details-section r1c5-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-37"></div> <span class="help-tooltip" id="tfbs-side-help-37">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ArgR" target="_blank">ArgR</a></td>
<td>Regulator of arginine biosynthesis genes</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>


</div>

</div>

</div>





<div class="page" id='r2c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000002.1 - Region 1 - NRPS,NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-244"></div> <span class="help-tooltip" id="region_svg-help-244">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000002.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c1-svg" data-filename="JABTEZ010000002.1_r2c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 68,150 - 118,622 nt. (total: 50,473 nt)

<a class="cluster-rules-header" id="r2c1-rules-header" href="#r2c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c1-rules">NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r2c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-245"></div> <span class="help-tooltip" id="focus-panel-help-245">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c1-gene-table-details" data-name="r2c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c1-nrps_pks" data-name="r2c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c1-MIBiG-cluster-compare" data-name="r2c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c1-knownclusterblast" data-name="r2c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c1-subclusterblast" data-name="r2c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c1-tfbs-finder" data-name="r2c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r2c1-nrps-pks-bubbles" data-name="r2c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r2c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-49"></div> <span class="help-tooltip" id="gt-help-tooltip-help-49">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-40"></div> <span class="help-tooltip" id="nrps-domain-help-40">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-details-svg" data-filename="JABTEZ010000002.1_r2c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-41"></div> <span class="help-tooltip" id="MIBiG-help-41">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c1"></div>
<div id="comparison-MIBiG-r2c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS-like</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>1.03</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 10E9</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>1.03</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 22D11</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>1.02</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>1.02</td>
<td>NRP, Polyketide</td>

<td>thalassospiramide A</td>


<td>Tistrella bauzanensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.98</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.95</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.93</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.42, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.91</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000921: 0-15706">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000921/index.html#r1c1" target="_new">BGC0000921</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.88</td>
<td>Other (Shikimate-derived), Polyketide, NRP</td>

<td>promysalin</td>


<td>Pseudomonas putida</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876: 71-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.87</td>
<td>Other</td>

<td>neopolyoxin C</td>


<td>Streptomyces tendae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 10E9</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 22D11</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP, Polyketide</td>
<td>thalassospiramide A</td>
<td>Tistrella bauzanensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.41, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000921: 0-15706">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000921/index.html#r1c1" target="_new">BGC0000921</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Other (Shikimate-derived), Polyketide, NRP</td>
<td>promysalin</td>
<td>Pseudomonas putida</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000876: 71-9235">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000876/index.html#r1c1" target="_new">BGC0000876</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.46, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Other</td>
<td>neopolyoxin C</td>
<td>Streptomyces tendae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-42"></div> <span class="help-tooltip" id="cb-known-help-42">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c1_1.svg">
crochelin A
</option>

<option value = "svg/knownclusterblast_r2c1_2.svg">
bleomycin A2/bleomycin B2
</option>

<option value = "svg/knownclusterblast_r2c1_3.svg">
bleomycin
</option>

<option value = "svg/knownclusterblast_r2c1_4.svg">
microsclerodermin M
</option>

<option value = "svg/knownclusterblast_r2c1_5.svg">
microsclerodermin
</option>

<option value = "svg/knownclusterblast_r2c1_6.svg">
tubulysin A/tubulysin B/tubulysin C/tubulysin D/tubulysin E/tubulysin F/tubulysin G/tubulysin H/tubulysin I
</option>

<option value = "svg/knownclusterblast_r2c1_7.svg">
matlystatin A
</option>

<option value = "svg/knownclusterblast_r2c1_8.svg">
acidobactin A/acidobactin B
</option>

<option value = "svg/knownclusterblast_r2c1_9.svg">
delftibactin A/delftibactin B
</option>

<option value = "svg/knownclusterblast_r2c1_10.svg">
tallysomycin A/tallysomycin B
</option>

</select>
<button id="knownclusterblast-r2c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c1-svg"></div>

</div>
</div>

<div class="body-details-section r2c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-43"></div> <span class="help-tooltip" id="cb-sub-help-43">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-47"></div> <span class="help-tooltip" id="tfbs-body-help-47">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.16 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-78333-AbrC3"></div>

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 26.06 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-108667-NuR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 22.20 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-68222-CelR"></div>

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 22.10 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c1-68223-CelR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r2c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-48"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-48">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c1-domain-bubble-svg" data-filename="JABTEZ010000002.1_r2c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (68149 - 118622): neighbouring NRPS-like-T1PKS-NRPS
</option>



<option value="CC2">
Candidate 2 (68149 - 109928): single NRPS-like
</option>



<option value="CC3">
Candidate 3 (69932 - 115128): single T1PKS
</option>



<option value="CC4">
Candidate 4 (75175 - 118622): single NRPS
</option>


</select>

<div class="details-svg" id="r2c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c1-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c1-nrps_pks_products" data-name="r2c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c1-nrps_pks_monomers" data-name="r2c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r2c1-tfbs-finder" data-name="r2c1-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r2c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-44"></div> <span class="help-tooltip" id="nrps-structures-help-44">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 68149 - 118622:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c1-1" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CNC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*&amp;nrps3=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CNC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 2, location 68149 - 109928:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c1-2" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CNC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*&amp;nrps3=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c1-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CNC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 69932 - 115128:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c1-3" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CNC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*&amp;nrps3=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CNC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 4, location 75175 - 118622:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c1-4" class="smiles-canvas" data-smiles="NC([*])C(=O)C=CNC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (ccmal) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=ccmal*&amp;nrps3=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c1-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)C=CNC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-45"></div> <span class="help-tooltip" id="nrps-monomers-help-45">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">HS041_13060</span></strong>: X
<div class="collapser collapser-target-HS041_13060 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (11..393)</strong>: X
<div class="collapser collapser-target-HS041_13060 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-HS041_13060-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">--FFVHGGMK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">HS041_13125</span></strong>: X
<div class="collapser collapser-target-HS041_13125 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (20..398)</strong>: X
<div class="collapser collapser-target-HS041_13125 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-HS041_13125-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DMTHVGGFVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>D-bMe-Asp <span class="serif">DARHVGIFVK</span> (44% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_13130</span></strong>: mal
<div class="collapser collapser-target-HS041_13130 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (514..796)</strong>: mal
<div class="collapser collapser-target-HS041_13130 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-HS041_13130-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 91.7%</dd>
<dd>inactive: 83.3%</dd>
<dd>2-Rhyd-Malonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-HS041_13130-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 184.5</dt>

<dd></dd><dt>inactive: 110.8</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 69.8</dt>

<dd></dd><dt>Methylmalonyl-CoA: 66.5</dt>

<dd></dd><dt>Propionyl-CoA: 57.5</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 48.4</dt>

<dd></dd><dt>Isobutyryl-CoA: 33.1</dt>

<dd></dd><dt>CHC-CoA: 32.4</dt>

<dd></dd><dt>Benzoyl-CoA: 31.5</dt>

<dd></dd><dt>Acetyl-CoA: 30.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 24.1</dt>

<dd></dd><dt>fatty_acid: 23.7</dt>

<dd></dd><dt>trans-1,2-CPDA: 16.6</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 8.5</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

<strong><span class="serif">HS041_13135</span></strong>: X
<div class="collapser collapser-target-HS041_13135 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (9..387)</strong>: X
<div class="collapser collapser-target-HS041_13135 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-HS041_13135-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVHDSGYVAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Lys <span class="serif">DVFESGGVAK</span> (68% 8Å match)</dd>
<dd>Lys or Arg <span class="serif">DVEDIGSVAK</span> (50% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r2c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-46"></div> <span class="help-tooltip" id="tfbs-side-help-46">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CelR" target="_blank">CelR</a></td>
<td>Cellobiose uptake repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r2c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000002.1 - Region 2 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-246"></div> <span class="help-tooltip" id="region_svg-help-246">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000002.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c2-svg" data-filename="JABTEZ010000002.1_r2c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 215,400 - 270,759 nt. (total: 55,360 nt)

<a class="cluster-rules-header" id="r2c2-rules-header" href="#r2c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c2-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r2c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-247"></div> <span class="help-tooltip" id="focus-panel-help-247">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c2-gene-table-details" data-name="r2c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c2-nrps_pks" data-name="r2c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c2-MIBiG-cluster-compare" data-name="r2c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c2-knownclusterblast" data-name="r2c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c2-subclusterblast" data-name="r2c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c2-tfbs-finder" data-name="r2c2-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r2c2-nrps-pks-bubbles" data-name="r2c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r2c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-59"></div> <span class="help-tooltip" id="gt-help-tooltip-help-59">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-50"></div> <span class="help-tooltip" id="nrps-domain-help-50">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c2-details-svg" data-filename="JABTEZ010000002.1_r2c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-51"></div> <span class="help-tooltip" id="MIBiG-help-51">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c2"></div>
<div id="comparison-MIBiG-r2c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002688: 824-19983">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002688/index.html#r1c1" target="_new">BGC0002688</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>NRP</td>

<td>benarthin, dibenarthin</td>


<td>Planobispora rosea</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002685: 0-16397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002685/index.html#r1c1" target="_new">BGC0002685</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Rothia mucilaginosa ATCC 25296</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417: 0-12076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>rhodochelin</td>


<td>Rhodococcus jostii RHA1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001925: 0-20488">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001925/index.html#r1c1" target="_new">BGC0001925</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Alkaloid</td>

<td>altemicidin, SB-203207, SB-203208</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401: 0-12882">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>paenibactin</td>


<td>Paenibacillus elgii B69</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185: 0-11721">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>bacillibactin</td>


<td>Bacillus velezensis FZB42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.32</td>
<td>NRP, Alkaloid</td>

<td>pyreudione A, pyreudione B, pyreudione C, pyreudione D, pyreudione E</td>


<td>Pseudomonas fluorescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002495: 0-19671">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002495/index.html#r1c1" target="_new">BGC0002495</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>photobactin</td>


<td>Photorhabdus luminescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002413: 0-22337">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002413/index.html#r1c1" target="_new">BGC0002413</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.44, order:N/A, components:0.97)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>NRP</td>

<td>frederiksenibactin</td>


<td>Yersinia frederiksenii ATCC 33641</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002688: 824-19983">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002688/index.html#r1c1" target="_new">BGC0002688</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>NRP</td>
<td>benarthin, dibenarthin</td>
<td>Planobispora rosea</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002685: 0-16397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002685/index.html#r1c1" target="_new">BGC0002685</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Rothia mucilaginosa ATCC 25296</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417: 0-12076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>rhodochelin</td>
<td>Rhodococcus jostii RHA1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001925: 0-20488">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001925/index.html#r1c1" target="_new">BGC0001925</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Alkaloid</td>
<td>altemicidin, SB-203207, SB-203208</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401: 0-12882">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>paenibactin</td>
<td>Paenibacillus elgii B69</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185: 0-11721">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>bacillibactin</td>
<td>Bacillus velezensis FZB42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP, Alkaloid</td>
<td>pyreudione A, pyreudione B, pyreudione C, pyreudione D, pyreudione E</td>
<td>Pseudomonas fluorescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002495: 0-19671">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002495/index.html#r1c1" target="_new">BGC0002495</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>photobactin</td>
<td>Photorhabdus luminescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002413: 0-22337">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002413/index.html#r1c1" target="_new">BGC0002413</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.44, order:N/A, components:0.97)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>frederiksenibactin</td>
<td>Yersinia frederiksenii ATCC 33641</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-52"></div> <span class="help-tooltip" id="cb-known-help-52">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c2_1.svg">
tetronasin
</option>

<option value = "svg/knownclusterblast_r2c2_2.svg">
quinocarcin
</option>

<option value = "svg/knownclusterblast_r2c2_3.svg">
naphthyridinomycin
</option>

<option value = "svg/knownclusterblast_r2c2_4.svg">
paramagnetoquinone 1/paramagnetoquinone 2
</option>

<option value = "svg/knownclusterblast_r2c2_5.svg">
scleric acid
</option>

</select>
<button id="knownclusterblast-r2c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c2-svg"></div>

</div>
</div>

<div class="body-details-section r2c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-53"></div> <span class="help-tooltip" id="cb-sub-help-53">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r2c2-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-57"></div> <span class="help-tooltip" id="tfbs-body-help-57">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 26.66 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r2c2-221690-CelR"></div>

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 26.43 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r2c2-221691-CelR"></div>

<div class="tfbs-hit-detail">
AfsQ1
<div class="collapser collapser-target-AfsQ1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Two-component system AfsQ1-Q2, activator of antibiotic production<br>
<span>Score</span>: 19.52 out of 19.52
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r2c2-258934-AfsQ1"></div>

<div class="tfbs-hit-detail">
AfsQ1
<div class="collapser collapser-target-AfsQ1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Two-component system AfsQ1-Q2, activator of antibiotic production<br>
<span>Score</span>: 19.52 out of 19.52
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r2c2-258945-AfsQ1"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (5):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
BldD
<div class="collapser collapser-target-BldD collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and antibiotic global regulator<br>
<span>Score</span>: 18.70 out of 23.43
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c2-216240-BldD"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 17.34 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c2-231032-ZuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 17.14 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c2-238884-ZuR"></div>

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 22.65 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c2-259109-LexA"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 25.30 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c2-261696-NrtR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r2c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-58"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-58">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c2-domain-bubble-svg" data-filename="JABTEZ010000002.1_r2c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC5">
Candidate 5 (215399 - 270759): single NRPS
</option>


</select>

<div class="details-svg" id="r2c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c2-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c2-nrps_pks_products" data-name="r2c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c2-nrps_pks_monomers" data-name="r2c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r2c2-tfbs-finder" data-name="r2c2-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r2c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-54"></div> <span class="help-tooltip" id="nrps-structures-help-54">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 5, location 215399 - 270759:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c2-5" class="smiles-canvas" data-smiles="NCC(=O)NC(CC(=O)O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Gly) + (Asp)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly&amp;nrps2=asp" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*&amp;nrps2=asp*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c2-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NCC(=O)NC(CC(=O)O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-55"></div> <span class="help-tooltip" id="nrps-monomers-help-55">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">HS041_13790</span></strong>: Gly
<div class="collapser collapser-target-HS041_13790 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (468..835)</strong>: Gly
<div class="collapser collapser-target-HS041_13790 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly
<div class="collapser collapser-target-HS041_13790-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Tiny, hydrophilic, transition to aliphatic (Gly, Ala)</dd>
<dt>Single AA prediction:</dt>
<dd>gly (Gly)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DILQLGVIWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gly <span class="serif">DILQLGVIWK</span> (91% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_13795</span></strong>: X
<div class="collapser collapser-target-HS041_13795 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (11..395)</strong>: X
<div class="collapser collapser-target-HS041_13795 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-HS041_13795-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">NATFGAYPAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">HS041_13825</span></strong>: Asp
<div class="collapser collapser-target-HS041_13825 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asp" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=asp*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (477..892)</strong>: Asp
<div class="collapser collapser-target-HS041_13825 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Asp
<div class="collapser collapser-target-HS041_13825-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>asp,asn (Asp, Asn)</dd>
<dt>Single AA prediction:</dt>
<dd>asp (Asp)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DFTKIGHVGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Asp <span class="serif">DLTKIGHVGK</span> (74% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r2c2-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-56"></div> <span class="help-tooltip" id="tfbs-side-help-56">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CelR" target="_blank">CelR</a></td>
<td>Cellobiose uptake repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/BldD" target="_blank">BldD</a></td>
<td>Development and antibiotic global regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsQ1" target="_blank">AfsQ1</a></td>
<td>Two-component system AfsQ1-Q2, activator of antibiotic production</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r2c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000002.1 - Region 3 - NRPS,NRPS-like,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-248"></div> <span class="help-tooltip" id="region_svg-help-248">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000002.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c3-svg" data-filename="JABTEZ010000002.1_r2c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 479,865 - 573,731 nt. (total: 93,867 nt)

<a class="cluster-rules-header" id="r2c3-rules-header" href="#r2c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c3-rules">T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r2c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-249"></div> <span class="help-tooltip" id="focus-panel-help-249">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c3-gene-table-details" data-name="r2c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c3-nrps_pks" data-name="r2c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c3-MIBiG-cluster-compare" data-name="r2c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c3-knownclusterblast" data-name="r2c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c3-subclusterblast" data-name="r2c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r2c3-tfbs-finder" data-name="r2c3-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r2c3-nrps-pks-bubbles" data-name="r2c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r2c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-69"></div> <span class="help-tooltip" id="gt-help-tooltip-help-69">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-60"></div> <span class="help-tooltip" id="nrps-domain-help-60">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c3-details-svg" data-filename="JABTEZ010000002.1_r2c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-61"></div> <span class="help-tooltip" id="MIBiG-help-61">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c3"></div>
<div id="comparison-MIBiG-r2c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.42, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.65</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 10E9</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.42, order:N/A, components:0.40)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.65</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 22D11</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.65</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.64</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.64</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001925: 0-20488">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001925/index.html#r1c1" target="_new">BGC0001925</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.64</td>
<td>Alkaloid</td>

<td>altemicidin, SB-203207, SB-203208</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.45, order:N/A, components:0.96)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.63</td>
<td>NRP</td>

<td>vicibactin</td>


<td>Rhizobium etli CFN 42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.63</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000348: 0-13540">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000348/index.html#r1c1" target="_new">BGC0000348</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.63 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.62</td>
<td>NRP</td>

<td>ergovaline</td>


<td>Epichloe festucae var. lolii</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000201: 0-28528">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000201/index.html#r1c1" target="_new">BGC0000201</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.54, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.54, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.61</td>
<td>Polyketide, Saccharide</td>

<td>auricin</td>


<td>Streptomyces aureofaciens</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000056: 2-6689">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000056/index.html#r1c1" target="_new">BGC0000056</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.49, order:0.69, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Polyketide</td>
<td>esperamicin</td>
<td>Actinomadura verrucosospora</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002493: 1205-6076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002493/index.html#r1c1" target="_new">BGC0002493</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>parabactin</td>
<td>Paracoccus denitrificans PD1222</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000677: 120-4940">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000677/index.html#r1c1" target="_new">BGC0000677</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>Terpene</td>
<td>cyclooctatin</td>
<td>Streptomyces melanosporofaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561: 0-17578">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.48, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>curacomycin</td>
<td>Streptomyces curacoi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001374: 52-17210">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001374/index.html#r1c1" target="_new">BGC0001374</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.52, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Other</td>
<td>thienodolin</td>
<td>Streptomyces albogriseolus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000249: 0-12635">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000249/index.html#r1c1" target="_new">BGC0000249</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.60, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Polyketide</td>
<td>nogalamycin</td>
<td>Streptomyces nogalater</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>vicibactin</td>
<td>Rhizobium etli CFN 42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002031: 3-5918">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002031/index.html#r1c1" target="_new">BGC0002031</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Alkaloid</td>
<td>kitasetaline</td>
<td>Kitasatospora setae KM-6054</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131: 0-6280">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Polyketide</td>
<td>pyrrolomycin</td>
<td>Streptomyces sp. UC 11065</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-62"></div> <span class="help-tooltip" id="cb-known-help-62">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c3-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c3_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c3_1.svg">
maduropeptin
</option>

<option value = "svg/knownclusterblast_r2c3_2.svg">
sporolide A/sporolide B
</option>

<option value = "svg/knownclusterblast_r2c3_3.svg">
kedarcidin
</option>

<option value = "svg/knownclusterblast_r2c3_4.svg">
C-1027
</option>

<option value = "svg/knownclusterblast_r2c3_5.svg">
lidamycin
</option>

<option value = "svg/knownclusterblast_r2c3_6.svg">
C-1027
</option>

<option value = "svg/knownclusterblast_r2c3_7.svg">
neocarzinostatin
</option>

<option value = "svg/knownclusterblast_r2c3_8.svg">
amycolamycin A/amycolamycin B
</option>

<option value = "svg/knownclusterblast_r2c3_9.svg">
calicheamicin
</option>

<option value = "svg/knownclusterblast_r2c3_10.svg">
uncialamycin
</option>

</select>
<button id="knownclusterblast-r2c3-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c3-svg"></div>

</div>
</div>

<div class="body-details-section r2c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-63"></div> <span class="help-tooltip" id="cb-sub-help-63">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<select id= "subclusterblast-r2c3-select" class= "clusterblast-selector">
<option value="svg/subclusterblast_r2c3_all.svg">
All hits
</option>

<option value = "svg/subclusterblast_r2c3_1.svg">
AY271660_c2: maduropeptin enediyne (100% of genes show similarity), enediyne
</option>

<option value = "svg/subclusterblast_r2c3_2.svg">
AY117439_c3: neocarzinostatin enediyne (91% of genes show similarity), enediyne
</option>

<option value = "svg/subclusterblast_r2c3_3.svg">
AY048670_c3: C-1027 enediyne (91% of genes show similarity), enediyne
</option>

<option value = "svg/subclusterblast_r2c3_4.svg">
AF497482_c2: calicheamicin enediyne (38% of genes show similarity), enediyne
</option>

<option value = "svg/subclusterblast_r2c3_5.svg">
AF497482_c3: calicheamicin 4 nucleotide sugars (16% of genes show similarity), sugar
</option>

<option value = "svg/subclusterblast_r2c3_6.svg">
AY271660_c3: maduropeptin aminosugar (madurose) (40% of genes show similarity), sugar
</option>

<option value = "svg/subclusterblast_r2c3_7.svg">
DQ297453_c2: AT2433 aminopentose (25% of genes show similarity), sugar
</option>

</select>
<button id="subclusterblast-r2c3-download">Download graphic</button>
</div>
<div id="subclusterblast-r2c3-svg"></div>

</div>
</div>





<div class="body-details-section r2c3-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-67"></div> <span class="help-tooltip" id="tfbs-body-help-67">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (7):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
HypR
<div class="collapser collapser-target-HypR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: L-hydroxyproline utilization repressor<br>
<span>Score</span>: 18.94 out of 22.40
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c3-488372-HypR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.83 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c3-495875-ZuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 16.73 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c3-514169-ZuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.28 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c3-518231-ZuR"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 19.75 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c3-541471-NrtR"></div>

<div class="tfbs-hit-detail">
HypR
<div class="collapser collapser-target-HypR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: L-hydroxyproline utilization repressor<br>
<span>Score</span>: 18.59 out of 22.40
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c3-541743-HypR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 16.70 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c3-568108-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r2c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-68"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-68">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c3-domain-bubble-svg" data-filename="JABTEZ010000002.1_r2c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC6">
Candidate 6 (479864 - 573731): neighbouring T1PKS-NRPS-NRPS-like
</option>





<option value="CC8">
Candidate 8 (513205 - 556418): single NRPS
</option>



<option value="CC9">
Candidate 9 (532036 - 573731): single NRPS-like
</option>


</select>

<div class="details-svg" id="r2c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c3-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c3-nrps_pks_products" data-name="r2c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c3-nrps_pks_monomers" data-name="r2c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r2c3-tfbs-finder" data-name="r2c3-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r2c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-64"></div> <span class="help-tooltip" id="nrps-structures-help-64">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 6, location 479864 - 573731:
<div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c3-6" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(CC(C)C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (Leu)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=leu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=leu*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c3-6-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(CC(C)C)C(=O)O"></canvas>
</div>
</div>


<hr>






<div>
<div class="prediction-text">
For candidate cluster 8, location 513205 - 556418:
<div class="collapser collapser-target-CC8 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c3-8" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(CC(C)C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (Leu)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=leu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=leu*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c3-8-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(CC(C)C)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 9, location 532036 - 573731:
<div class="collapser collapser-target-CC9 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c3-9" class="smiles-canvas" data-smiles="NC([*])C(=O)NC(CC(C)C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X) + (Leu)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps2=leu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x&amp;nrps2=leu*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c3-9-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)NC(CC(C)C)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-65"></div> <span class="help-tooltip" id="nrps-monomers-help-65">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">HS041_14915</span></strong>: mal
<div class="collapser collapser-target-HS041_14915 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (619..844)</strong>: mal
<div class="collapser collapser-target-HS041_14915 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-HS041_14915-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 75.0%</dd>
<dd>inactive: 62.5%</dd>
<dd>Propionyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-HS041_14915-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 82.5</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 50.3</dt>

<dd></dd><dt>Methylmalonyl-CoA: 40.7</dt>

<dd></dd><dt>inactive: 30.5</dt>

<dd></dd><dt>Propionyl-CoA: 20.7</dt>

<dd></dd><dt>trans-1,2-CPDA: 19.2</dt>

<dd></dd><dt>fatty_acid: 19.2</dt>

<dd></dd><dt>Isobutyryl-CoA: 17.6</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 13.8</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 9.9</dt>

<dd></dd><dt>Benzoyl-CoA: 7.7</dt>

<dd></dd><dt>Acetyl-CoA: 1.8</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

<dd></dd><dt>CHC-CoA: -0.1</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">HS041_15050</span></strong>: X
<div class="collapser collapser-target-HS041_15050 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>AMP-binding (476..876)</strong>: X
<div class="collapser collapser-target-HS041_15050 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-HS041_15050-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAVQLALLAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>aza-bTyr <span class="serif">DACELALLAK</span> (68% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_15110</span></strong>: X
<div class="collapser collapser-target-HS041_15110 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (22..417)</strong>: X
<div class="collapser collapser-target-HS041_15110 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-HS041_15110-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">GYTVIAGHGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">HS041_15120</span></strong>: Leu
<div class="collapser collapser-target-HS041_15120 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (1..385)</strong>: Leu
<div class="collapser collapser-target-HS041_15120 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Leu
<div class="collapser collapser-target-HS041_15120-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DALFLGLVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DAWFLGQVVK</span> (79% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r2c3-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-66"></div> <span class="help-tooltip" id="tfbs-side-help-66">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/HypR" target="_blank">HypR</a></td>
<td>L-hydroxyproline utilization repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r2c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000002.1 - Region 4 - LAP,NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-250"></div> <span class="help-tooltip" id="region_svg-help-250">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000002.1.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c4-svg" data-filename="JABTEZ010000002.1_r2c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,285,856 - 1,347,562 nt. (total: 61,707 nt)

<a class="cluster-rules-header" id="r2c4-rules-header" href="#r2c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c4-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>LAP: ((((goadsporin_like or PF00881 or TIGR03605) and (YcaO or TIGR03604)) or TIGR03882 or TIGR03603 or TIGR03602) and not cyanobactin_synth)</div>

<div class="region-svg-container">
<div id='r2c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-251"></div> <span class="help-tooltip" id="focus-panel-help-251">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c4-gene-table-details" data-name="r2c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c4-nrps_pks" data-name="r2c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c4-MIBiG-cluster-compare" data-name="r2c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c4-knownclusterblast" data-name="r2c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c4-subclusterblast" data-name="r2c4-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r2c4-tfbs-finder" data-name="r2c4-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r2c4-nrps-pks-bubbles" data-name="r2c4-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r2c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-79"></div> <span class="help-tooltip" id="gt-help-tooltip-help-79">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-70"></div> <span class="help-tooltip" id="nrps-domain-help-70">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c4-details-svg" data-filename="JABTEZ010000002.1_r2c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-71"></div> <span class="help-tooltip" id="MIBiG-help-71">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c4"></div>
<div id="comparison-MIBiG-r2c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">LAP</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002584: 0-13427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002584/index.html#r1c1" target="_new">BGC0002584</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.61</td>
<td>RiPP</td>

<td>azolemycin B, azolemycin A, azolemycin D, azolemycin C</td>


<td>Streptomyces sp. FXJ1.264</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002013: 0-11564">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002013/index.html#r1c1" target="_new">BGC0002013</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.45, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.52</td>
<td>RiPP</td>

<td>curacozole</td>


<td>Streptomyces curacoi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002115: 0-14439">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002115/index.html#r1c1" target="_new">BGC0002115</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.47 (id:0.44, order:0.26, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.50</td>
<td>RiPP</td>

<td>streptamidine</td>


<td>Streptomyces albidoflavus J1074</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001173: 0-9945">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001173/index.html#r1c1" target="_new">BGC0001173</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.55 (id:0.44, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.48</td>
<td>RiPP</td>

<td>plantazolicin</td>


<td>Bacillus pumilus ATCC 7061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002359: 18-27968">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002359/index.html#r1c1" target="_new">BGC0002359</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.47, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.36 (id:0.45, order:N/A, components:0.30)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.42</td>
<td>NRP</td>

<td>dudomycin A</td>


<td>Streptomyces albus subsp. chlorinus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001174: 248-6052">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001174/index.html#r1c1" target="_new">BGC0001174</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.44, order:0.45, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.44, order:0.45, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.42</td>
<td>RiPP</td>

<td>corynazolicin</td>


<td>Corynebacterium urealyticum DSM 7111</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000329: 485-21536">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000329/index.html#r1c1" target="_new">BGC0000329</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.47, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.35 (id:0.45, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.42</td>
<td>NRP</td>

<td>corpeptin A, corpeptin B</td>


<td>Pseudomonas corrugata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002311: 0-5945">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002311/index.html#r1c1" target="_new">BGC0002311</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>


<td>0.41</td>
<td>RiPP</td>

<td>cutimycin</td>


<td>Cutibacterium acnes KPA171202</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000569: 0-9892">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000569/index.html#r1c1" target="_new">BGC0000569</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.44, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.44 (id:0.42, order:0.28, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.40</td>
<td>RiPP</td>

<td>plantazolicin</td>


<td>Bacillus velezensis FZB42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928: 108-8402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.40 (id:0.43, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.38</td>
<td>Other</td>

<td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>


<td>Streptomyces sp. MA37</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001137: 0-18476">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001137/index.html#r1c1" target="_new">BGC0001137</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Alkaloid</td>
<td>marinacarboline A, marinacarboline B, marinacarboline C, marinacarboline D</td>
<td>Marinactinospora thermotolerans</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561: 0-17578">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>NRP</td>
<td>curacomycin</td>
<td>Streptomyces curacoi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417: 0-12076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>rhodochelin</td>
<td>Rhodococcus jostii RHA1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:1.00, components:0.68)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP, Polyketide</td>
<td>thalassospiramide A</td>
<td>Tistrella bauzanensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001890: 1-28356">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001890/index.html#r1c1" target="_new">BGC0001890</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>amphi-enterobactin 1, amphi-enterobactin 2, amphi-enterobactin 3, amphi-enterobactin 4</td>
<td>Vibrio campbellii ATCC BAA-1116</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185: 0-11721">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>bacillibactin</td>
<td>Bacillus velezensis FZB42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002075: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002075/index.html#r1c1" target="_new">BGC0002075</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP, Alkaloid</td>
<td>pyreudione A, pyreudione B, pyreudione C, pyreudione D, pyreudione E</td>
<td>Pseudomonas fluorescens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>vicibactin</td>
<td>Rhizobium etli CFN 42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001925: 0-20488">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001925/index.html#r1c1" target="_new">BGC0001925</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Alkaloid</td>
<td>altemicidin, SB-203207, SB-203208</td>
<td>Streptomyces sp.</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-72"></div> <span class="help-tooltip" id="cb-known-help-72">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c4-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c4_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c4_1.svg">
qinichelins
</option>

<option value = "svg/knownclusterblast_r2c4_2.svg">
scabichelin
</option>

<option value = "svg/knownclusterblast_r2c4_3.svg">
amychelin A/amychelin B
</option>

<option value = "svg/knownclusterblast_r2c4_4.svg">
kanamycin
</option>

<option value = "svg/knownclusterblast_r2c4_5.svg">
kanamycin
</option>

<option value = "svg/knownclusterblast_r2c4_6.svg">
gobichelin A/gobichelin B
</option>

<option value = "svg/knownclusterblast_r2c4_7.svg">
peucechelin
</option>

<option value = "svg/knownclusterblast_r2c4_8.svg">
madurastatin D1/madurastatin D2/(-)-Madurastatin C1
</option>

<option value = "svg/knownclusterblast_r2c4_9.svg">
salinichelins
</option>

<option value = "svg/knownclusterblast_r2c4_10.svg">
coelichelin
</option>

</select>
<button id="knownclusterblast-r2c4-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c4-svg"></div>

</div>
</div>

<div class="body-details-section r2c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-73"></div> <span class="help-tooltip" id="cb-sub-help-73">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r2c4-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-77"></div> <span class="help-tooltip" id="tfbs-body-help-77">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 25.39 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r2c4-1323126-CelR"></div>

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 25.19 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r2c4-1323127-CelR"></div>

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 26.06 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r2c4-1323333-CelR"></div>

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 25.19 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r2c4-1323334-CelR"></div>

<div class="tfbs-hit-detail">
DmdR1
<div class="collapser collapser-target-DmdR1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Iron(II)-dependent repressor<br>
<span>Score</span>: 27.90 out of 36.53
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r2c4-1335155-DmdR1"></div>

<div class="tfbs-hit-detail">
CatR
<div class="collapser collapser-target-CatR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: H2O2-responsive repressor<br>
<span>Score</span>: 20.30 out of 25.31
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r2c4-1339360-CatR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
BldD
<div class="collapser collapser-target-BldD collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and antibiotic global regulator<br>
<span>Score</span>: 19.42 out of 23.43
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c4-1300341-BldD"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r2c4-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-78"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-78">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c4-domain-bubble-svg" data-filename="JABTEZ010000002.1_r2c4_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c4-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC10">
Candidate 10 (1285855 - 1347562): neighbouring NRPS-LAP
</option>



<option value="CC11">
Candidate 11 (1285855 - 1343035): single NRPS
</option>


</select>

<div class="details-svg" id="r2c4-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c4-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c4-nrps_pks_products" data-name="r2c4-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c4-nrps_pks_monomers" data-name="r2c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>





<div class="sidepanel-details-header r2c4-tfbs-finder" data-name="r2c4-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r2c4-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-74"></div> <span class="help-tooltip" id="nrps-structures-help-74">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 10, location 1285855 - 1347562:
<div class="collapser collapser-target-CC10 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c4-10" class="smiles-canvas" data-smiles="NC(CCCCN)C(=O)NCC(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Lys) + (Gly) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys&amp;nrps2=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys*&amp;nrps2=gly*&amp;nrps3=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c4-10-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCCCN)C(=O)NCC(=O)NC([*])C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 11, location 1285855 - 1343035:
<div class="collapser collapser-target-CC11 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c4-11" class="smiles-canvas" data-smiles="NC(CCCCN)C(=O)NCC(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Lys) + (Gly) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys&amp;nrps2=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys*&amp;nrps2=gly*&amp;nrps3=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c4-11-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCCCN)C(=O)NCC(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-75"></div> <span class="help-tooltip" id="nrps-monomers-help-75">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">HS041_18525</span></strong>: Lys
<div class="collapser collapser-target-HS041_18525 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=lys*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (472..871)</strong>: Lys
<div class="collapser collapser-target-HS041_18525 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Lys
<div class="collapser collapser-target-HS041_18525-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAEDIGTVIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Lys <span class="serif">DAESIGTVIK</span> (68% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_18530</span></strong>: Gly
<div class="collapser collapser-target-HS041_18530 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (8..421)</strong>: Gly
<div class="collapser collapser-target-HS041_18530 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly
<div class="collapser collapser-target-HS041_18530-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Tiny, hydrophilic, transition to aliphatic (Gly, Ala)</dd>
<dt>Single AA prediction:</dt>
<dd>gly (Gly)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DILQLGVVWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gly <span class="serif">DILQLGVVWK</span> (79% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_18560</span></strong>: X
<div class="collapser collapser-target-HS041_18560 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (17..398)</strong>: X
<div class="collapser collapser-target-HS041_18560 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-HS041_18560-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">NATFGSYPAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">HS041_18565</span></strong>: X
<div class="collapser collapser-target-HS041_18565 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>AMP-binding (531..923)</strong>: X
<div class="collapser collapser-target-HS041_18565 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-HS041_18565-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DTEDIGTIIK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Lys <span class="serif">DTENIGTISK</span> (62% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">HS041_18580</span></strong>: X
<div class="collapser collapser-target-HS041_18580 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (25..431)</strong>: X
<div class="collapser collapser-target-HS041_18580 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Asp, Asn, Glu, Gln, Aad
<div class="collapser collapser-target-HS041_18580-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">GGWMAGNAMK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

</dl>
</div>
</div>





<div class="sidepanel-details-section r2c4-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-76"></div> <span class="help-tooltip" id="tfbs-side-help-76">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CelR" target="_blank">CelR</a></td>
<td>Cellobiose uptake repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DmdR1" target="_blank">DmdR1</a></td>
<td>Iron(II)-dependent repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CatR" target="_blank">CatR</a></td>
<td>H2O2-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/BldD" target="_blank">BldD</a></td>
<td>Development and antibiotic global regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r2c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000002.1 - Region 5 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-252"></div> <span class="help-tooltip" id="region_svg-help-252">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000002.1.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c5-svg" data-filename="JABTEZ010000002.1_r2c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,372,493 - 1,393,338 nt. (total: 20,846 nt)

<a class="cluster-rules-header" id="r2c5-rules-header" href="#r2c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c5-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r2c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-253"></div> <span class="help-tooltip" id="focus-panel-help-253">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c5-gene-table-details" data-name="r2c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c5-MIBiG-cluster-compare" data-name="r2c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c5-knownclusterblast" data-name="r2c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c5-subclusterblast" data-name="r2c5-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r2c5-tfbs-finder" data-name="r2c5-tfbs-finder">
TFBS Finder
</div>


</div>


<div class="body-details-section r2c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-85"></div> <span class="help-tooltip" id="gt-help-tooltip-help-85">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-80"></div> <span class="help-tooltip" id="MIBiG-help-80">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c5"></div>
<div id="comparison-MIBiG-r2c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911: 591-10299">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.46, order:0.36, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>


<td>0.18</td>
<td>Terpene</td>

<td>raimonol</td>


<td>Streptomyces anulatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910: 237-11923">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.47, order:0.33, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Terpene</td>

<td>cyslabdan</td>


<td>Streptomyces cyslabdanicus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002643: 0-9229">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002643/index.html#r1c1" target="_new">BGC0002643</a></td>





<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>


<td>0.15</td>
<td>Alkaloid</td>

<td>aetokthonotoxin</td>


<td>Aetokthonos hydrillicola Thurmond2011</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>





<td class="cc-heat-cell" title="0.44 (id:0.51, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>


<td>0.14</td>
<td>Other</td>

<td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>


<td>Streptomyces sp. RM-5-8</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286: 17-17862">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286</a></td>





<td class="cc-heat-cell" title="0.41 (id:0.46, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.13</td>
<td>Polyketide</td>

<td>viguiepinol</td>


<td>Streptomyces sp. KO-3988</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001595: 67-28084">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001595/index.html#r1c1" target="_new">BGC0001595</a></td>





<td class="cc-heat-cell" title="0.35 (id:0.41, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>


<td>0.10</td>
<td>Alkaloid</td>

<td>fischerindole</td>


<td>Fischerella sp. SAG 46.79</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197: 0-12679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.50, order:0.45, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.09</td>
<td>Polyketide</td>

<td>frankiamicin</td>


<td>Frankia sp. EAN1pec</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001605: 0-9118">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001605/index.html#r1c1" target="_new">BGC0001605</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.43, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>Terpene</td>

<td>gibberellin</td>


<td>Sinorhizobium fredii NGR234</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000706: 350-14906">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000706/index.html#r1c1" target="_new">BGC0000706</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.46, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Saccharide</td>

<td>kanamycin</td>


<td>Streptomyces kanamyceticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002372: 389-22072">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002372/index.html#r1c1" target="_new">BGC0002372</a></td>





<td class="cc-heat-cell" title="0.28 (id:0.44, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>


<td>0.07</td>
<td>Polyketide, Terpene, Alkaloid</td>

<td>marinoterpin B, marinoterpin A, marinoterpin C</td>


<td>Streptomyces sp.</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911: 591-10299">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911</a></td>


<td class="cc-heat-cell" title="0.50 (id:0.46, order:0.36, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>
<td>0.50</td>
<td>Terpene</td>
<td>raimonol</td>
<td>Streptomyces anulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001910: 237-11923">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001910/index.html#r1c1" target="_new">BGC0001910</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.47, order:0.33, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Terpene</td>
<td>cyslabdan</td>
<td>Streptomyces cyslabdanicus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002643: 0-9229">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002643/index.html#r1c1" target="_new">BGC0002643</a></td>


<td class="cc-heat-cell" title="0.45 (id:0.41, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.45), rgb(0, 0, 0, 0.45) 45%, white 45%)"></td>
<td>0.45</td>
<td>Alkaloid</td>
<td>aetokthonotoxin</td>
<td>Aetokthonos hydrillicola Thurmond2011</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001483: 0-25255">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001483/index.html#r1c1" target="_new">BGC0001483</a></td>


<td class="cc-heat-cell" title="0.44 (id:0.51, order:N/A, components:0.38)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.44), rgb(0, 0, 0, 0.44) 44%, white 44%)"></td>
<td>0.44</td>
<td>Other</td>
<td>5-isoprenylindole-3-carboxylate β-D-glycosyl ester</td>
<td>Streptomyces sp. RM-5-8</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000286: 17-17862">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000286/index.html#r1c1" target="_new">BGC0000286</a></td>


<td class="cc-heat-cell" title="0.41 (id:0.46, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>
<td>0.41</td>
<td>Polyketide</td>
<td>viguiepinol</td>
<td>Streptomyces sp. KO-3988</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001595: 67-28084">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001595/index.html#r1c1" target="_new">BGC0001595</a></td>


<td class="cc-heat-cell" title="0.35 (id:0.41, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.35), rgb(0, 0, 0, 0.35) 35%, white 35%)"></td>
<td>0.35</td>
<td>Alkaloid</td>
<td>fischerindole</td>
<td>Fischerella sp. SAG 46.79</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001197: 0-12679">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001197/index.html#r1c1" target="_new">BGC0001197</a></td>


<td class="cc-heat-cell" title="0.33 (id:0.50, order:0.45, components:0.17)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>
<td>0.33</td>
<td>Polyketide</td>
<td>frankiamicin</td>
<td>Frankia sp. EAN1pec</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001605: 0-9118">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001605/index.html#r1c1" target="_new">BGC0001605</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.43, order:N/A, components:0.23)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>Terpene</td>
<td>gibberellin</td>
<td>Sinorhizobium fredii NGR234</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000706: 350-14906">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000706/index.html#r1c1" target="_new">BGC0000706</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.46, order:N/A, components:0.20)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Saccharide</td>
<td>kanamycin</td>
<td>Streptomyces kanamyceticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002372: 389-22072">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002372/index.html#r1c1" target="_new">BGC0002372</a></td>


<td class="cc-heat-cell" title="0.28 (id:0.44, order:N/A, components:0.18)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.28), rgb(0, 0, 0, 0.28) 28%, white 28%)"></td>
<td>0.28</td>
<td>Polyketide, Terpene, Alkaloid</td>
<td>marinoterpin B, marinoterpin A, marinoterpin C</td>
<td>Streptomyces sp.</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-81"></div> <span class="help-tooltip" id="cb-known-help-81">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c5-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c5_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c5_1.svg">
kendomycin B
</option>

<option value = "svg/knownclusterblast_r2c5_2.svg">
frankiamicin
</option>

</select>
<button id="knownclusterblast-r2c5-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c5-svg"></div>

</div>
</div>

<div class="body-details-section r2c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-82"></div> <span class="help-tooltip" id="cb-sub-help-82">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r2c5-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-84"></div> <span class="help-tooltip" id="tfbs-body-help-84">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 19.01 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c5-1374440-ZuR"></div>

<div class="tfbs-hit-detail">
ArgR
<div class="collapser collapser-target-ArgR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Regulator of arginine biosynthesis genes<br>
<span>Score</span>: 16.39 out of 20.01
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c5-1375355-ArgR"></div>

</div>
</div>
</div>

</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r2c5-tfbs-finder" data-name="r2c5-tfbs-finder">
TFBS Finder
</div>


</div>








<div class="sidepanel-details-section r2c5-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-83"></div> <span class="help-tooltip" id="tfbs-side-help-83">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ArgR" target="_blank">ArgR</a></td>
<td>Regulator of arginine biosynthesis genes</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>


</div>

</div>

</div>

<div class="page" id='r2c6' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000002.1 - Region 6 - NRP-metallophore,NRPS,lanthipeptide-class-ii <div class="help-container"> <div class="help-icon" data-id="region_svg-help-254"></div> <span class="help-tooltip" id="region_svg-help-254">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000002.1.region006.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c6-svg" data-filename="JABTEZ010000002.1_r2c6.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,729,536 - 1,806,044 nt. (total: 76,509 nt)

<a class="cluster-rules-header" id="r2c6-rules-header" href="#r2c6">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c6-rules">lanthipeptide-class-ii: cds(LANC_like and DUF4135)<br>NRP-metallophore: (VibH_like or Cy_tandem or (cds(Condensation and AMP-binding) and ((IBH_Asp and not SBH_Asp) or IBH_His or TBH_Asp or CyanoBH_Asp1 or CyanoBH_Asp2 or IPL or SalSyn or (EntA and EntC) or (GrbD and GrbE) or (FbnL and FbnM) or PvdO or PvdP or (Orn_monoox and not (KtzT or MetRS-like)) or Lys_monoox or VbsL)))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r2c6-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-255"></div> <span class="help-tooltip" id="focus-panel-help-255">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c6">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c6-gene-table-details" data-name="r2c6-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c6-nrps_pks" data-name="r2c6-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c6-MIBiG-cluster-compare" data-name="r2c6-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c6-knownclusterblast" data-name="r2c6-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c6-subclusterblast" data-name="r2c6-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r2c6-lanthipeptides" data-name="r2c6-lanthipeptides">
Lanthipeptides
</div>







<div class="body-details-header r2c6-tfbs-finder" data-name="r2c6-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r2c6-nrps-pks-bubbles" data-name="r2c6-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r2c6-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-97"></div> <span class="help-tooltip" id="gt-help-tooltip-help-97">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c6">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c6-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c6-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c6">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c6-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-86"></div> <span class="help-tooltip" id="nrps-domain-help-86">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c6-details-svg" data-filename="JABTEZ010000002.1_r2c6_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c6-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c6-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-87"></div> <span class="help-tooltip" id="MIBiG-help-87">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c6-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c6-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c6"></div>
<div id="comparison-MIBiG-r2c6-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">lanthipeptide-class-ii</span></th>

<th><span class="cc-heat-product">NRP-metallophore</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349: 0-24145">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.45, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>






<td class="cc-heat-cell" title="0.73 (id:0.54, order:0.79, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>






<td class="cc-heat-cell" title="0.73 (id:0.54, order:0.79, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.92</td>
<td>NRP</td>

<td>erythrochelin</td>


<td>Saccharopolyspora erythraea NRRL 2338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002424: 0-19621">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002424/index.html#r1c1" target="_new">BGC0002424</a></td>





<td class="cc-heat-cell" title="0.32 (id:0.46, order:N/A, components:0.22)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>






<td class="cc-heat-cell" title="0.72 (id:0.54, order:0.76, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>






<td class="cc-heat-cell" title="0.72 (id:0.54, order:0.76, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>


<td>0.90</td>
<td>NRP</td>

<td>saccharochelin A, saccharochelin B, saccharochelin C, saccharochelin D, saccharochelin E, saccharochelin F, saccharochelin G, saccharochelin H</td>


<td>Saccharothrix sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>






<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>


<td>0.77</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Escherichia coli str. K-12 substr. MG1655</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002467: 0-30976">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002467/index.html#r1c1" target="_new">BGC0002467</a></td>





<td class="cc-heat-cell" title="0.33 (id:0.46, order:N/A, components:0.24)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.54, order:0.52, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.54, order:0.52, components:0.85)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.66</td>
<td>NRP</td>

<td>thermochelin</td>


<td>Thermocrispum agreste DSM 44070</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.66</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.65</td>
<td>NRP</td>

<td>thermoactinoamide A</td>


<td>Thermoactinomyces sp. AS95</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000321: 1657-43703">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000321/index.html#r1c1" target="_new">BGC0000321</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.44, order:N/A, components:0.29)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.63</td>
<td>NRP</td>

<td>chaetocin</td>


<td>Collariella virescens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.63</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.60</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.45, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.47 (id:0.44, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.60</td>
<td>RiPP</td>

<td>SRO15-3108</td>


<td>Streptomyces filamentosus NRRL 15998</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c6-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000349: 0-24145">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000349/index.html#r1c1" target="_new">BGC0000349</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.54, order:0.79, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>NRP</td>
<td>erythrochelin</td>
<td>Saccharopolyspora erythraea NRRL 2338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002424: 0-19621">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002424/index.html#r1c1" target="_new">BGC0002424</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.54, order:0.76, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>NRP</td>
<td>saccharochelin A, saccharochelin B, saccharochelin C, saccharochelin D, saccharochelin E, saccharochelin F, saccharochelin G, saccharochelin H</td>
<td>Saccharothrix sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>


<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
<td>0.71</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Escherichia coli str. K-12 substr. MG1655</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>thermoactinoamide A</td>
<td>Thermoactinomyces sp. AS95</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000528: 0-8183">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000528/index.html#r1c1" target="_new">BGC0000528</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>RiPP</td>
<td>michiganin A</td>
<td>Clavibacter michiganensis subsp. michiganensis NCPPB 382</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000503: 0-17083">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000503/index.html#r1c1" target="_new">BGC0000503</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>RiPP</td>
<td>cinnamycin</td>
<td>Streptomyces cinnamoneus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001579: 0-8838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001579/index.html#r1c1" target="_new">BGC0001579</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>RiPP</td>
<td>duramycin</td>
<td>Streptomyces cinnamoneus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.44, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c6-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-88"></div> <span class="help-tooltip" id="cb-known-help-88">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c6-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c6_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c6_1.svg">
peucechelin
</option>

<option value = "svg/knownclusterblast_r2c6_2.svg">
scabichelin
</option>

<option value = "svg/knownclusterblast_r2c6_3.svg">
erythrochelin
</option>

<option value = "svg/knownclusterblast_r2c6_4.svg">
thermochelin
</option>

<option value = "svg/knownclusterblast_r2c6_5.svg">
saccharochelin A/saccharochelin B/saccharochelin C/saccharochelin D/saccharochelin E/saccharochelin F/saccharochelin G/saccharochelin H
</option>

<option value = "svg/knownclusterblast_r2c6_6.svg">
salinichelins
</option>

<option value = "svg/knownclusterblast_r2c6_7.svg">
qinichelins
</option>

<option value = "svg/knownclusterblast_r2c6_8.svg">
albachelin
</option>

<option value = "svg/knownclusterblast_r2c6_9.svg">
amychelin A/amychelin B
</option>

<option value = "svg/knownclusterblast_r2c6_10.svg">
coelichelin
</option>

</select>
<button id="knownclusterblast-r2c6-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c6-svg"></div>

</div>
</div>

<div class="body-details-section r2c6-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-89"></div> <span class="help-tooltip" id="cb-sub-help-89">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r2c6-lanthipeptides">
<div class="details">
<div class="heading">
<span>Lanthipeptide predictions</span>
<div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-90"></div> <span class="help-tooltip" id="lanthi-body-help-90">Shows the possible core peptides for each biosynthetic enzyme.<br>Includes CompaRiPPson results for any available databases, with a detailed explanation <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/comparippson' target='_blank'>here</a>.</span></div>
</div>
<div class="details-text ripp-body">


<h4>Lanthipeptide(s) for HS041_20450</h4>
<hr>

<div class="ripp-motif">


<div class="ripp-name">
<span class="serif">HS041_20445</span>
</div>
<div class="ripp-segment-type">Leader:</div>
<div class="ripp-segment-sequence">MTVTLERAEEIIAA</div>

<div class="ripp-segment-type">Core:</div>
<div class="ripp-segment-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="ripp-segment-type">Core with dehydrations:</div>
<div class="ripp-segment-sequence ripp-dehydrated"><span>W</span><span>R</span><span class="dha">Dha</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">Dha</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">Dha</span><span>E</span><span>I</span><span class="dhb">Dhb</span><span>M</span><span class="dhb">Dhb</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">Dha</span><span>G</span><span>R</span><span class="cys">Cys</span><span>G</span><span class="dhb">Dhb</span><span>A</span><span class="cys">Cys</span><span class="dha">Dha</span><span>G</span><span class="dha">Dha</span><span>R</span><span class="dhb">Dhb</span><span>Y</span><span>M</span><span class="cys">Cys</span><span class="cys">Cys</span></div>

<div class="comparippson-details">
<h4>antiSMASH-DB 4.0 matches for HS041_20445</h4>
<div class="comparippson-block">
<div class="comparippson-container">
<span class="comparippson-similarity">95.8%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JACHJJ010000004.1&start=244644&end=244833">NZ_JACHJJ010000004.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: FHS22_RS08790_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span>R</span><span class="dha near-match-cycle">S</span><span>G</span><span>D</span><span>E</span><span> </span><span>D</span><span class="dha near-match-cycle">S</span><span>P</span><span> </span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha near-match-cycle">S</span><span>E</span><span>I</span><span class="dhb near-match-cycle">T</span><span>M</span><span class="dhb near-match-cycle">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha near-match-cycle">S</span><span>G</span><span>R</span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span>G</span><span class="dha near-match-cycle">S</span><span>R</span><span class="dhb near-match-cycle">T</span><span>Y</span><span>M</span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>A</span><span>D</span><span class="dha">S</span><span>P</span><span>A</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
</div>
</div><div class="comparippson-container">
<span class="comparippson-similarity">89.6%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_FNVT01000006.1&start=357268&end=357457">NZ_FNVT01000006.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: C4V41_RS24485_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span>R</span><span class="near-match-cycle">+</span><span>G</span><span>D</span><span>+</span><span>P</span><span>D</span><span class="dha near-match-cycle">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>+</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha near-match-cycle">S</span><span>E</span><span>I</span><span class="dhb near-match-cycle">T</span><span>M</span><span class="dhb near-match-cycle">T</span><span>G</span><span>P</span><span>G</span><span> </span><span class="dha near-match-cycle">S</span><span>G</span><span>R</span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span>G</span><span class="dha near-match-cycle">S</span><span>R</span><span class="dhb near-match-cycle">T</span><span>Y</span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dhb">T</span><span>G</span><span>D</span><span>D</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>L</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>G</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>Q</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
</div>
</div><div class="comparippson-container">
<span class="comparippson-similarity">85.4%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JACHIN010000006.1&start=233361&end=233550">NZ_JACHIN010000006.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: HNR40_RS23265_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span>R</span><span class="near-match-cycle">+</span><span>G</span><span>+</span><span>E</span><span>P</span><span>D</span><span class="dha near-match-cycle">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>+</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha near-match-cycle">S</span><span>E</span><span>I</span><span class="dhb near-match-cycle">T</span><span>M</span><span class="dhb near-match-cycle">T</span><span>G</span><span>P</span><span>G</span><span>+</span><span class="dha near-match-cycle">S</span><span>G</span><span>+</span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span>G</span><span class="dha near-match-cycle">S</span><span>R</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dhb">T</span><span>G</span><span>E</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>L</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>H</span><span class="dha">S</span><span>G</span><span>K</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>R</span><span>Y</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
</div>
</div>
</div><div class="comparippson-block comparippson-extension">
with 57 more sequence matches <div class="collapser collapser-target-comparippson-worse-hits collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-container">
<span class="comparippson-similarity">85.4%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_KZ559487.1&start=71163&end=71352">NZ_KZ559487.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: CXR44_RS34455_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span>R</span><span class="near-match-cycle">+</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha near-match-cycle">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>+</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span> </span><span>E</span><span class="dha near-match-cycle">S</span><span>E</span><span>I</span><span class="dhb near-match-cycle">T</span><span>M</span><span class="dhb near-match-cycle">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha near-match-cycle">S</span><span>G</span><span>+</span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span>G</span><span class="dha near-match-cycle">S</span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dhb">T</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>M</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>A</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>K</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>Y</span><span class="dhb">T</span><span>R</span><span>Y</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">85.4%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_VRLV01000011.1&start=90705&end=90894">NZ_VRLV01000011.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: FVE25_RS35215_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span>R</span><span>+</span><span>G</span><span>+</span><span>E</span><span> </span><span>D</span><span class="dha near-match-cycle">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>+</span><span>L</span><span>P</span><span>G</span><span>+</span><span>Y</span><span>V</span><span>E</span><span class="dha near-match-cycle">S</span><span>E</span><span>I</span><span class="dhb near-match-cycle">T</span><span>M</span><span class="dhb near-match-cycle">T</span><span>G</span><span>P</span><span>G</span><span>+</span><span class="dha near-match-cycle">S</span><span>G</span><span>R</span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span>G</span><span class="dha near-match-cycle">S</span><span>R</span><span class="dhb near-match-cycle">T</span><span> </span><span>M</span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span>A</span><span>G</span><span>E</span><span>E</span><span class="dha">S</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>L</span><span>L</span><span>P</span><span>G</span><span>L</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>H</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>R</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">54.2%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_QFWW01000008.1&start=127445&end=127631">NZ_QFWW01000008.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: DI276_RS14925_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>-</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span>R</span><span class="near-match-cycle">+</span><span>G</span><span>D</span><span> </span><span> </span><span>D</span><span> </span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span>P</span><span> </span><span>G</span><span> </span><span>Y</span><span> </span><span>E</span><span> </span><span>E</span><span>+</span><span class="dhb near-match-cycle">T</span><span>M</span><span class="dhb near-match-cycle">T</span><span>G</span><span> </span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span>G</span><span>+</span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span class="dha near-match-cycle">S</span><span>+</span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dhb">T</span><span>G</span><span>D</span><span>-</span><span>V</span><span>D</span><span>G</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>F</span><span>P</span><span class="dha">S</span><span>G</span><span>E</span><span>Y</span><span>A</span><span>E</span><span>H</span><span>E</span><span>L</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>-</span><span>-</span><span>H</span><span class="dha">S</span><span>G</span><span>K</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>V</span><span class="cys">C</span><span class="dhb">T</span><span>Y</span><span class="dha">S</span><span>Q</span><span>G</span><span>R</span><span>Q</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">46</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">50.0%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP040709.1&start=4065223&end=4065418">NZ_CP040709.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: FF090_RS18940_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>-</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span>R</span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span>+</span><span> </span><span> </span><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span>+</span><span> </span><span>G</span><span> </span><span>Y</span><span> </span><span>E</span><span>+</span><span>+</span><span>I</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="dha near-match-cycle">S</span><span>G</span><span>+</span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span>G</span><span class="dha near-match-cycle">S</span><span>R</span><span class="dhb near-match-cycle">T</span><span> </span><span>M</span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>Q</span><span>A</span><span>D</span><span>A</span><span class="dha">S</span><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>F</span><span>I</span><span>G</span><span>G</span><span>E</span><span>Y</span><span>A</span><span>E</span><span>A</span><span>D</span><span>I</span><span>I</span><span>G</span><span>D</span><span class="dhb">T</span><span class="dhb">T</span><span class="dha">S</span><span>I</span><span class="dha">S</span><span>G</span><span>K</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>R</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">50</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">47.2%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JAIQCM010000003.1&start=193251&end=193458">NZ_JAIQCM010000003.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: K9V56_RS08295_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>-</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span>R</span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span>+</span><span> </span><span> </span><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span>+</span><span> </span><span>G</span><span> </span><span>Y</span><span> </span><span>E</span><span>+</span><span>+</span><span>I</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="dha near-match-cycle">S</span><span>G</span><span>+</span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span>G</span><span class="dha near-match-cycle">S</span><span>R</span><span class="dhb near-match-cycle">T</span><span> </span><span>M</span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">5</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>Q</span><span>A</span><span>D</span><span>G</span><span class="dha">S</span><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>F</span><span>I</span><span>G</span><span>G</span><span>E</span><span>Y</span><span>A</span><span>E</span><span>A</span><span>D</span><span>I</span><span>V</span><span>G</span><span>D</span><span class="dhb">T</span><span class="dha">S</span><span class="dhb">T</span><span>I</span><span class="dha">S</span><span>G</span><span>K</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>R</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">53</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">45.8%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_BMQJ01000023.1&start=60234&end=60420">NZ_BMQJ01000023.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: IEZ10_RS33030_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>-</span><span>-</span><span>-</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span>R</span><span class="dha near-match-cycle">S</span><span>G</span><span> </span><span>+</span><span> </span><span> </span><span> </span><span> </span><span>D</span><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span> </span><span>G</span><span> </span><span>Y</span><span> </span><span>E</span><span class="dha near-match-cycle">S</span><span>E</span><span>I</span><span class="dhb near-match-cycle">T</span><span>+</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span>R</span><span> </span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span class="dha near-match-cycle">S</span><span>R</span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>A</span><span>D</span><span>V</span><span>E</span><span>G</span><span>W</span><span>D</span><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>F</span><span>I</span><span>G</span><span>D</span><span>Y</span><span>A</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>L</span><span class="dhb">T</span><span>-</span><span>-</span><span>A</span><span>L</span><span>D</span><span class="dhb">T</span><span>R</span><span>L</span><span>A</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dhb">T</span><span class="cys">C</span><span class="dha">S</span><span>R</span><span>-</span><span>-</span><span>F</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">47</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">44.9%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBWY010000002.1&start=687721&end=687913">NZ_JABBWY010000002.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: HHX37_RS08020_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>-</span><span>-</span><span>-</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span>R</span><span class="dha near-match-cycle">S</span><span>G</span><span> </span><span>+</span><span> </span><span> </span><span> </span><span> </span><span>D</span><span class="near-match-cycle"> </span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span> </span><span>G</span><span> </span><span>Y</span><span> </span><span>E</span><span class="dha near-match-cycle">S</span><span>E</span><span>I</span><span class="dhb near-match-cycle">T</span><span>+</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span>R</span><span> </span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>G</span><span>+</span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>A</span><span>D</span><span>V</span><span>E</span><span>G</span><span>W</span><span>D</span><span class="cys">C</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>F</span><span>I</span><span>G</span><span>E</span><span>Y</span><span>A</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>L</span><span class="dhb">T</span><span>-</span><span>-</span><span>A</span><span>L</span><span>D</span><span class="dhb">T</span><span>R</span><span>R</span><span>P</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dhb">T</span><span>G</span><span>A</span><span>N</span><span class="dhb">T</span><span>I</span><span>P</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">49</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">40.7%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP012109.1&start=4714246&end=4714435">NZ_CP012109.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: A176_RS18020_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>-</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span>R</span><span>+</span><span> </span><span> </span><span>+</span><span> </span><span> </span><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span> </span><span>G</span><span> </span><span> </span><span>+</span><span> </span><span>E</span><span class="dha near-match-cycle">S</span><span>+</span><span>I</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="dha near-match-cycle">S</span><span>G</span><span>R</span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span> </span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span>G</span><span class="dha near-match-cycle">S</span><span>+</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">6</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span>N</span><span class="dha">S</span><span>A</span><span>D</span><span>K</span><span class="dha">S</span><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>F</span><span>V</span><span>G</span><span>G</span><span>E</span><span>F</span><span class="dhb">T</span><span>E</span><span class="dha">S</span><span>D</span><span>I</span><span>I</span><span>H</span><span>E</span><span class="dhb">T</span><span>I</span><span>V</span><span>I</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>G</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>Q</span><span class="dhb">T</span><span>R</span><span>Q</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">54</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">39.6%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP041730.1&start=3715327&end=3715516">NZ_CP041730.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: FNU76_RS16940_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">15</div>
<div class="comparippson-sequence"><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>-</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>+</span><span> </span><span>G</span><span> </span><span>Y</span><span> </span><span>E</span><span>+</span><span>+</span><span>I</span><span> </span><span>M</span><span class="near-match-cycle">+</span><span> </span><span> </span><span>P</span><span> </span><span> </span><span> </span><span>G</span><span> </span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span>G</span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span>Y</span><span>M</span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">3</div>
<div class="comparippson-sequence"><span>V</span><span class="dha">S</span><span>G</span><span>E</span><span>Y</span><span>A</span><span>E</span><span>A</span><span>D</span><span>I</span><span>A</span><span>M</span><span class="dha">S</span><span>I</span><span>R</span><span>P</span><span>L</span><span class="dhb">T</span><span>W</span><span>G</span><span>G</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span class="dha">S</span><span>R</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">37</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">39.0%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_AP023359.1&start=7977441&end=7977666">NZ_AP023359.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: Prubr_RS34935_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>-</span><span>-</span><span>-</span><span>-</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>-</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span>-</span><span>-</span><span>-</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">42</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span> </span><span class="near-match-cycle">+</span><span>G</span><span> </span><span>E</span><span>P</span><span> </span><span> </span><span> </span><span> </span><span>+</span><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span>L</span><span> </span><span>G</span><span> </span><span>Y</span><span> </span><span>E</span><span>+</span><span>+</span><span>I</span><span class="dhb near-match-cycle">T</span><span> </span><span class="dhb near-match-cycle">T</span><span>G</span><span> </span><span>G</span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span>G</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span>G</span><span class="dha near-match-cycle">S</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>L</span><span class="dhb">T</span><span>G</span><span class="dhb">T</span><span>E</span><span>P</span><span>V</span><span>D</span><span>G</span><span>W</span><span>E</span><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>F</span><span>L</span><span>G</span><span>G</span><span>K</span><span>Y</span><span>A</span><span>E</span><span>A</span><span>D</span><span>I</span><span class="dhb">T</span><span>A</span><span class="dhb">T</span><span>G</span><span class="dha">S</span><span>G</span><span>G</span><span class="dha">S</span><span>N</span><span class="dha">S</span><span>V</span><span class="dhb">T</span><span>G</span><span class="cys">C</span><span>G</span><span>-</span><span>W</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">49</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">37.5%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JADIKF010000040.1&start=249445&end=249646">NZ_JADIKF010000040.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: ISS99_RS17245_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">15</div>
<div class="comparippson-sequence"><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span>-</span><span>-</span><span>-</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span>-</span><span>-</span><span>-</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>+</span><span> </span><span>G</span><span> </span><span>+</span><span> </span><span>E</span><span class="dha near-match-cycle">S</span><span>+</span><span>I</span><span> </span><span> </span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span> </span><span> </span><span> </span><span>Y</span><span> </span><span> </span><span> </span><span class="dha near-match-cycle">S</span><span>G</span><span>R</span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span> </span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span>G</span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">4</div>
<div class="comparippson-sequence"><span>V</span><span class="dha">S</span><span>G</span><span>R</span><span>F</span><span>A</span><span>E</span><span class="dha">S</span><span>D</span><span>I</span><span>V</span><span class="dha">S</span><span class="dha">S</span><span class="dhb">T</span><span>N</span><span>I</span><span>V</span><span class="dhb">T</span><span>A</span><span>Y</span><span>A</span><span>V</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>I</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span class="dha">S</span><span>H</span><span>A</span><span>E</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">43</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">37.5%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_KB896392.1&start=135054&end=135273">NZ_KB896392.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_KB898432.1&start=11438&end=11657">NZ_KB898432.1</a></span>
<span class="comparippson-description">Lanthipeptides: D462_RS1000000129050_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: C576_RS27880_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">8</div>
<div class="comparippson-sequence"><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>-</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">42</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>D</span><span class="dha near-match-cycle">S</span><span>P</span><span> </span><span>G</span><span>P</span><span>V</span><span> </span><span> </span><span> </span><span>G</span><span> </span><span>Y</span><span>V</span><span> </span><span> </span><span>E</span><span>I</span><span class="dhb near-match-cycle">T</span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="near-match-cycle">+</span><span>A</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>G</span><span class="dha near-match-cycle">S</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>D</span><span class="dha">S</span><span>P</span><span>A</span><span>G</span><span>P</span><span>V</span><span>F</span><span class="dhb">T</span><span>G</span><span>G</span><span>R</span><span>Y</span><span>V</span><span>V</span><span>Q</span><span>E</span><span>I</span><span class="dhb">T</span><span>A</span><span class="dhb">T</span><span>A</span><span>V</span><span>A</span><span>N</span><span>W</span><span class="dhb">T</span><span>Y</span><span class="cys">C</span><span>-</span><span class="dha">S</span><span>A</span><span class="cys">C</span><span class="dhb">T</span><span>G</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">35</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">35.4%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP064030.1&start=1950646&end=1950847">NZ_CP064030.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: ISN74_RS08625_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">15</div>
<div class="comparippson-sequence"><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>-</span><span>-</span><span>-</span><span>-</span><span>-</span><span>-</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>+</span><span> </span><span>G</span><span> </span><span>+</span><span> </span><span>E</span><span class="dha near-match-cycle">S</span><span>+</span><span>I</span><span> </span><span> </span><span class="near-match-cycle">+</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span>G</span><span> </span><span> </span><span> </span><span class="dha near-match-cycle">S</span><span>G</span><span>R</span><span class="cys near-match-cycle">C</span><span>G</span><span class="near-match-cycle">+</span><span>A</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span>G</span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">4</div>
<div class="comparippson-sequence"><span>V</span><span class="dha">S</span><span>G</span><span>Q</span><span>F</span><span>A</span><span>E</span><span class="dha">S</span><span>D</span><span>I</span><span>V</span><span class="dha">S</span><span class="dha">S</span><span class="dhb">T</span><span>N</span><span>V</span><span>L</span><span class="dhb">T</span><span>L</span><span>G</span><span>A</span><span>V</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dha">S</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>A</span><span>H</span><span>A</span><span>E</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">43</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">35.4%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP072597.1&start=380587&end=380764">NZ_CP072597.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: J5226_RS01600_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>-</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span>R</span><span>+</span><span> </span><span> </span><span> </span><span> </span><span>D</span><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span>+</span><span> </span><span>G</span><span> </span><span>+</span><span> </span><span>E</span><span>+</span><span>+</span><span>I</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span>R</span><span> </span><span> </span><span class="near-match-cycle">+</span><span>A</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>G</span><span class="near-match-cycle"> </span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span>N</span><span>D</span><span>A</span><span>G</span><span>A</span><span>D</span><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>F</span><span>I</span><span class="dha">S</span><span>G</span><span>E</span><span>F</span><span>A</span><span>E</span><span>A</span><span>D</span><span>I</span><span>A</span><span>G</span><span>D</span><span>N</span><span>E</span><span>I</span><span>I</span><span class="dha">S</span><span>-</span><span>R</span><span>N</span><span class="dhb">T</span><span class="dha">S</span><span>A</span><span class="cys">C</span><span class="dhb">T</span><span>G</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>I</span><span>Q</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">35.4%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_FOZI01000001.1&start=181647&end=181830">NZ_FOZI01000001.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: BM338_RS00935_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">12</div>
<div class="comparippson-sequence"><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>-</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>-</span><span>-</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>G</span><span>P</span><span>+</span><span> </span><span> </span><span>G</span><span> </span><span> </span><span>+</span><span> </span><span>E</span><span>+</span><span>E</span><span>I</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="near-match-cycle">+</span><span> </span><span> </span><span>G</span><span> </span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>G</span><span class="dha near-match-cycle">S</span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>G</span><span>P</span><span>L</span><span>F</span><span class="dhb">T</span><span>G</span><span>G</span><span>Q</span><span>F</span><span>A</span><span>E</span><span>A</span><span>E</span><span>I</span><span>I</span><span class="dha">S</span><span>E</span><span>A</span><span class="dha">S</span><span class="dha">S</span><span>L</span><span class="dhb">T</span><span class="dhb">T</span><span>F</span><span>G</span><span>G</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dhb">T</span><span>G</span><span class="dha">S</span><span>H</span><span class="dhb">T</span><span>R</span><span>Y</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">40</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">35.2%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP011130.1&start=718185&end=718374">NZ_CP011130.1</a><span class="comparippson-extra-indicator">and 4 others</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP023465.1&start=683650&end=683839">NZ_CP023465.1</a></span>
<span class="comparippson-description">Lanthipeptides: CNO08_RS25510_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP094357.1&start=684803&end=684992">NZ_CP094357.1</a></span>
<span class="comparippson-description">Lanthipeptides: IEQ11_RS02795_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_KQ061218.1&start=215890&end=216079">NZ_KQ061218.1</a></span>
<span class="comparippson-description">Lanthipeptides: WN54_RS23295_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_LMHM01000017.1&start=214697&end=214886">NZ_LMHM01000017.1</a></span>
<span class="comparippson-description">Lanthipeptides: ASD86_RS21560_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: LC55x_RS02840_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>-</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>-</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span>R</span><span class="near-match-cycle">+</span><span> </span><span> </span><span>E</span><span> </span><span>+</span><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span>+</span><span> </span><span>G</span><span> </span><span>Y</span><span>V</span><span>E</span><span class="dha near-match-cycle">S</span><span>+</span><span>I</span><span> </span><span> </span><span> </span><span> </span><span> </span><span>P</span><span> </span><span> </span><span class="near-match-cycle">+</span><span>G</span><span> </span><span> </span><span> </span><span class="near-match-cycle">+</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>G</span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">6</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dhb">T</span><span>D</span><span>A</span><span>E</span><span class="dha">S</span><span>N</span><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>F</span><span>I</span><span>G</span><span>G</span><span>E</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>D</span><span>I</span><span>I</span><span>G</span><span>E</span><span>V</span><span>A</span><span>P</span><span>A</span><span>I</span><span class="dhb">T</span><span>G</span><span>-</span><span>-</span><span class="cys">C</span><span class="dha">S</span><span>L</span><span class="cys">C</span><span class="dhb">T</span><span>G</span><span class="dha">S</span><span>M</span><span>R</span><span>I</span><span>N</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">53</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">35.2%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP076103.1&start=702047&end=702236">NZ_CP076103.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: KME82_RS02870_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>-</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>-</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span>R</span><span class="near-match-cycle">+</span><span> </span><span> </span><span>E</span><span> </span><span>+</span><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span>+</span><span> </span><span>G</span><span> </span><span>Y</span><span>V</span><span>E</span><span class="dha near-match-cycle">S</span><span>+</span><span>I</span><span> </span><span> </span><span> </span><span> </span><span> </span><span>P</span><span> </span><span> </span><span class="near-match-cycle">+</span><span>G</span><span> </span><span> </span><span> </span><span class="near-match-cycle">+</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>G</span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">6</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dhb">T</span><span>D</span><span>A</span><span>E</span><span class="dha">S</span><span>N</span><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>F</span><span>I</span><span>G</span><span>G</span><span>E</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>D</span><span>I</span><span>I</span><span>G</span><span>E</span><span>V</span><span>A</span><span>P</span><span>A</span><span>I</span><span class="dhb">T</span><span>G</span><span>-</span><span>-</span><span class="cys">C</span><span class="dha">S</span><span>L</span><span class="cys">C</span><span class="dhb">T</span><span>G</span><span class="dha">S</span><span>I</span><span>R</span><span>I</span><span>N</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">53</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">33.3%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP048405.1&start=3109241&end=3109433">NZ_CP048405.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: GXW83_RS14070_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">17</div>
<div class="comparippson-sequence"><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>G</span><span> </span><span>Y</span><span> </span><span>E</span><span class="dha near-match-cycle">S</span><span>E</span><span>+</span><span class="dhb near-match-cycle">T</span><span> </span><span class="near-match-cycle">+</span><span>G</span><span> </span><span> </span><span> </span><span class="near-match-cycle">+</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">6</div>
<div class="comparippson-sequence"><span>G</span><span>A</span><span>Y</span><span>A</span><span>E</span><span class="dha">S</span><span>E</span><span>L</span><span class="dhb">T</span><span>A</span><span class="dha">S</span><span>G</span><span>A</span><span>V</span><span>R</span><span class="dhb">T</span><span>I</span><span>Q</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>Y</span><span class="dha">S</span><span>A</span><span>G</span><span>R</span><span>E</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">37</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">32.7%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP069338.1&start=1707133&end=1707328">NZ_CP069338.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: JQX13_RS07220_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>-</span><span>-</span><span>-</span><span>-</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>-</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span> </span><span class="dha near-match-cycle">S</span><span>G</span><span> </span><span>E</span><span> </span><span> </span><span> </span><span> </span><span> </span><span>+</span><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span>L</span><span> </span><span>G</span><span> </span><span> </span><span> </span><span>E</span><span>+</span><span> </span><span>+</span><span class="dhb near-match-cycle">T</span><span> </span><span class="near-match-cycle">+</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>G</span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>L</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>E</span><span class="dha">S</span><span>A</span><span>F</span><span>G</span><span>M</span><span>N</span><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>Y</span><span>L</span><span>G</span><span>G</span><span>E</span><span>A</span><span class="dhb">T</span><span>E</span><span>A</span><span>A</span><span>L</span><span class="dhb">T</span><span>N</span><span class="dha">S</span><span class="dhb">T</span><span>D</span><span>A</span><span>L</span><span>Y</span><span class="dhb">T</span><span>G</span><span class="cys">C</span><span>-</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dhb">T</span><span>G</span><span class="dha">S</span><span>L</span><span>H</span><span class="dha">S</span><span>Y</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">52</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">32.7%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP071090.1&start=9699086&end=9699290">NZ_CP071090.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: JY651_RS36970_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>-</span><span>-</span><span>-</span><span>-</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>-</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span> </span><span class="dha near-match-cycle">S</span><span>G</span><span> </span><span>E</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span>+</span><span> </span><span>G</span><span> </span><span> </span><span> </span><span>E</span><span>+</span><span> </span><span>+</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span> </span><span>P</span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="near-match-cycle">+</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span class="dha near-match-cycle">S</span><span>R</span><span> </span><span> </span><span>+</span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>L</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>E</span><span class="dha">S</span><span>V</span><span>D</span><span>G</span><span>L</span><span>A</span><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>Y</span><span>V</span><span>G</span><span>G</span><span class="dhb">T</span><span>A</span><span class="dhb">T</span><span>E</span><span>A</span><span>A</span><span>L</span><span class="dhb">T</span><span>D</span><span>A</span><span>A</span><span>P</span><span>M</span><span>M</span><span>G</span><span class="dhb">T</span><span>G</span><span class="cys">C</span><span>-</span><span class="dha">S</span><span>I</span><span class="cys">C</span><span class="dhb">T</span><span>A</span><span class="dha">S</span><span>R</span><span>P</span><span>V</span><span>V</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">52</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">31.2%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NC_017030.1&start=5061419&end=5061617">NC_017030.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: COCOR_RS20320_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">9</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>-</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>V</span><span> </span><span> </span><span>G</span><span> </span><span>M</span><span> </span><span> </span><span>E</span><span> </span><span> </span><span>+</span><span class="dhb near-match-cycle">T</span><span> </span><span class="near-match-cycle">+</span><span> </span><span> </span><span> </span><span> </span><span>+</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>V</span><span>F</span><span>V</span><span>G</span><span>G</span><span>M</span><span>R</span><span class="dhb">T</span><span>E</span><span>Q</span><span>A</span><span>L</span><span class="dhb">T</span><span>E</span><span class="dha">S</span><span>N</span><span>L</span><span>V</span><span>E</span><span>N</span><span class="dhb">T</span><span>G</span><span class="cys">C</span><span>-</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dha">S</span><span>A</span><span class="dha">S</span><span class="dhb">T</span><span class="dhb">T</span><span>G</span><span>F</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">41</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">31.2%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_AP014940.1&start=5510291&end=5510465">NZ_AP014940.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: LEN_RS25395_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>-</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span>R</span><span>+</span><span> </span><span> </span><span> </span><span> </span><span>D</span><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span>+</span><span> </span><span>G</span><span> </span><span>+</span><span> </span><span>E</span><span>+</span><span>+</span><span>I</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span> </span><span> </span><span class="near-match-cycle">+</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>G</span><span class="near-match-cycle"> </span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span>N</span><span>E</span><span>A</span><span>G</span><span class="dha">S</span><span>D</span><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>F</span><span>I</span><span>G</span><span>G</span><span>E</span><span>F</span><span>A</span><span>E</span><span>A</span><span>D</span><span>I</span><span>A</span><span>G</span><span>D</span><span class="dha">S</span><span>E</span><span>A</span><span>I</span><span class="dha">S</span><span>-</span><span>M</span><span>I</span><span class="dhb">T</span><span class="dha">S</span><span>V</span><span class="cys">C</span><span class="dhb">T</span><span>G</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>I</span><span>Q</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">31.2%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_AZWQ01000024.1&start=122349&end=122565">NZ_AZWQ01000024.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: B116_RS0113780_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">17</div>
<div class="comparippson-sequence"><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>G</span><span> </span><span>Y</span><span>V</span><span> </span><span> </span><span>E</span><span>I</span><span class="dhb near-match-cycle">T</span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span>G</span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>G</span><span class="dha near-match-cycle">S</span><span>+</span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">6</div>
<div class="comparippson-sequence"><span>G</span><span>K</span><span>Y</span><span>V</span><span>V</span><span>Q</span><span>E</span><span>I</span><span class="dhb">T</span><span class="dha">S</span><span class="dhb">T</span><span>W</span><span>G</span><span>G</span><span class="cys">C</span><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="cys">C</span><span>-</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dhb">T</span><span>G</span><span class="dha">S</span><span>Q</span><span>R</span><span>V</span><span class="dha">S</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">36</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">31.2%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_FNOG01000007.1&start=237589&end=237763">NZ_FNOG01000007.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: BLU84_RS27270_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>-</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span>R</span><span>+</span><span> </span><span> </span><span> </span><span> </span><span>D</span><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span>+</span><span> </span><span>G</span><span> </span><span>+</span><span> </span><span>E</span><span>+</span><span>+</span><span>I</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span> </span><span> </span><span class="near-match-cycle">+</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>G</span><span class="near-match-cycle"> </span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span>N</span><span>E</span><span>A</span><span>G</span><span class="dha">S</span><span>D</span><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>F</span><span>I</span><span>G</span><span>G</span><span>E</span><span>F</span><span>A</span><span>E</span><span>A</span><span>D</span><span>I</span><span>A</span><span>G</span><span>D</span><span class="dha">S</span><span>E</span><span>A</span><span>V</span><span class="dha">S</span><span>-</span><span>M</span><span>I</span><span class="dhb">T</span><span class="dha">S</span><span>V</span><span class="cys">C</span><span class="dhb">T</span><span>G</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>I</span><span>H</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">31.2%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBWY010000002.1&start=687967&end=688153">NZ_JABBWY010000002.1</a><span class="comparippson-extra-indicator">and 3 others</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBWZ010000011.1&start=16983&end=17169">NZ_JABBWZ010000011.1</a></span>
<span class="comparippson-description">Lanthipeptides: HHX40_RS26720_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_VANP01000002.1&start=15814&end=16000">NZ_VANP01000002.1</a></span>
<span class="comparippson-description">Lanthipeptides: FED44_RS05225_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_VYTZ01000002.1&start=525084&end=525270">NZ_VYTZ01000002.1</a></span>
<span class="comparippson-description">Lanthipeptides: F5972_RS06340_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: HHX37_RS08025_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">13</div>
<div class="comparippson-sequence"><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>P</span><span>+</span><span> </span><span> </span><span>G</span><span> </span><span>Y</span><span> </span><span>E</span><span class="dha near-match-cycle">S</span><span>E</span><span>I</span><span class="dhb near-match-cycle">T</span><span>+</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span>R</span><span> </span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span class="dha near-match-cycle">S</span><span>R</span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>P</span><span>L</span><span>F</span><span>I</span><span>G</span><span>E</span><span>Y</span><span>A</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>L</span><span class="dhb">T</span><span>-</span><span>-</span><span>A</span><span>L</span><span>D</span><span class="dhb">T</span><span>R</span><span>L</span><span>A</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dhb">T</span><span class="cys">C</span><span class="dha">S</span><span>R</span><span>-</span><span>-</span><span>F</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">32</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">31.2%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBWZ010000011.1&start=16737&end=16929">NZ_JABBWZ010000011.1</a><span class="comparippson-extra-indicator">and 2 others</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_VANP01000002.1&start=16054&end=16246">NZ_VANP01000002.1</a></span>
<span class="comparippson-description">Lanthipeptides: FED44_RS05230_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_VYTZ01000002.1&start=524838&end=525030">NZ_VYTZ01000002.1</a></span>
<span class="comparippson-description">Lanthipeptides: F5972_RS06335_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: HHX40_RS26715_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">13</div>
<div class="comparippson-sequence"><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>P</span><span>+</span><span> </span><span> </span><span>G</span><span> </span><span>Y</span><span> </span><span>E</span><span class="dha near-match-cycle">S</span><span>E</span><span>I</span><span class="dhb near-match-cycle">T</span><span>+</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span>R</span><span> </span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>G</span><span>+</span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>P</span><span>L</span><span>F</span><span>I</span><span>G</span><span>E</span><span>Y</span><span>A</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>L</span><span class="dhb">T</span><span>-</span><span>-</span><span>A</span><span>L</span><span>D</span><span class="dhb">T</span><span>R</span><span>R</span><span>P</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dhb">T</span><span>G</span><span>A</span><span>N</span><span class="dhb">T</span><span>I</span><span>P</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">34</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">31.2%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JALPRT010000019.1&start=60451&end=60667">NZ_JALPRT010000019.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_KB895919.1&start=3421&end=3637">NZ_KB895919.1</a></span>
<span class="comparippson-description">Lanthipeptides: C570_RS1000000130195_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: M0M45_RS13255_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">17</div>
<div class="comparippson-sequence"><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>G</span><span> </span><span>Y</span><span>V</span><span> </span><span> </span><span>E</span><span>I</span><span class="dhb near-match-cycle">T</span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span>G</span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>G</span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">6</div>
<div class="comparippson-sequence"><span>G</span><span>R</span><span>Y</span><span>V</span><span>V</span><span>Q</span><span>E</span><span>I</span><span class="dhb">T</span><span class="dha">S</span><span class="dhb">T</span><span>W</span><span>G</span><span>G</span><span class="cys">C</span><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="cys">C</span><span>-</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dhb">T</span><span>G</span><span class="dha">S</span><span>V</span><span>P</span><span>I</span><span>Q</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">36</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">29.2%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP034669.1&start=4844534&end=4844732">NZ_CP034669.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: EJ065_RS19470_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">9</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>-</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>V</span><span> </span><span> </span><span>G</span><span> </span><span>+</span><span> </span><span> </span><span>E</span><span> </span><span> </span><span>+</span><span class="dhb near-match-cycle">T</span><span> </span><span class="near-match-cycle">+</span><span> </span><span> </span><span> </span><span> </span><span>+</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>V</span><span>F</span><span>V</span><span>G</span><span>G</span><span>L</span><span>R</span><span class="dhb">T</span><span>E</span><span>Q</span><span>A</span><span>L</span><span class="dhb">T</span><span>E</span><span class="dha">S</span><span>N</span><span>L</span><span>V</span><span>E</span><span>N</span><span class="dhb">T</span><span>G</span><span class="cys">C</span><span>-</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dha">S</span><span>A</span><span class="dha">S</span><span class="dhb">T</span><span class="dhb">T</span><span>G</span><span>F</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">41</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">29.2%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP114039.1&start=7142215&end=7142413">NZ_CP114039.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: O0N60_RS29095_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">9</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>-</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>V</span><span> </span><span> </span><span>G</span><span> </span><span> </span><span> </span><span> </span><span>E</span><span> </span><span> </span><span>+</span><span class="dhb near-match-cycle">T</span><span> </span><span class="near-match-cycle">+</span><span> </span><span> </span><span> </span><span> </span><span>+</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>V</span><span>F</span><span>V</span><span>G</span><span>G</span><span>Q</span><span>R</span><span class="dhb">T</span><span>E</span><span>Q</span><span>A</span><span>L</span><span class="dhb">T</span><span>E</span><span class="dha">S</span><span>N</span><span>L</span><span>V</span><span>E</span><span>N</span><span class="dhb">T</span><span>G</span><span class="cys">C</span><span>-</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dha">S</span><span>A</span><span class="dha">S</span><span class="dhb">T</span><span class="dhb">T</span><span>G</span><span>F</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">41</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">29.2%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JANZKX010000002.1&start=602502&end=602691">NZ_JANZKX010000002.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: O9Y01_RS08280_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">17</div>
<div class="comparippson-sequence"><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>G</span><span>+</span><span>Y</span><span> </span><span>E</span><span>+</span><span>E</span><span>I</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="near-match-cycle">+</span><span> </span><span>+</span><span> </span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span>G</span><span class="dha near-match-cycle">S</span><span>R</span><span class="dhb near-match-cycle">T</span><span> </span><span>M</span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">6</div>
<div class="comparippson-sequence"><span>G</span><span>V</span><span>Y</span><span class="dha">S</span><span>E</span><span>A</span><span>E</span><span>I</span><span>A</span><span>G</span><span>D</span><span>V</span><span class="dha">S</span><span>R</span><span>E</span><span class="dhb">T</span><span>R</span><span>Q</span><span>G</span><span class="cys">C</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>I</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">37</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">28.8%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_FOSG01000018.1&start=65571&end=65775">NZ_FOSG01000018.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_PDKQ02000037.1&start=34701&end=34905">NZ_PDKQ02000037.1</a></span>
<span class="comparippson-description">Lanthipeptides: BFM75_RS14295_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: BM237_RS22045_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>-</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span> </span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span>E</span><span>P</span><span>+</span><span> </span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span>+</span><span> </span><span>G</span><span> </span><span> </span><span> </span><span>E</span><span>+</span><span> </span><span>+</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>G</span><span class="dha near-match-cycle">S</span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">7</div>
<div class="comparippson-sequence"><span>N</span><span class="dha">S</span><span>H</span><span>G</span><span>E</span><span>P</span><span>N</span><span>-</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>Y</span><span>V</span><span>G</span><span>G</span><span>A</span><span>A</span><span class="dhb">T</span><span>E</span><span>A</span><span>A</span><span>L</span><span>V</span><span>D</span><span>A</span><span class="dhb">T</span><span>D</span><span>A</span><span>L</span><span>F</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span>-</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dhb">T</span><span>G</span><span class="dha">S</span><span>L</span><span class="dhb">T</span><span class="dhb">T</span><span>F</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">52</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">28.3%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP063231.1&start=2141316&end=2141505">NZ_CP063231.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_FPIS01000001.1&start=577991&end=578180">NZ_FPIS01000001.1</a></span>
<span class="comparippson-description">Lanthipeptides: BT115_RS02490_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: IM816_RS09045_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>-</span><span>-</span><span>-</span><span>-</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>-</span><span>-</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span> </span><span class="dha near-match-cycle">S</span><span>G</span><span> </span><span>+</span><span> </span><span> </span><span> </span><span> </span><span> </span><span>D</span><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span> </span><span> </span><span> </span><span>+</span><span> </span><span>+</span><span class="near-match-cycle">+</span><span> </span><span>+</span><span> </span><span>M</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span> </span><span> </span><span>+</span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span class="dha near-match-cycle">S</span><span>R</span><span> </span><span> </span><span>+</span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>L</span><span class="dha">S</span><span>G</span><span>A</span><span>D</span><span>N</span><span>V</span><span>G</span><span>G</span><span>Y</span><span>D</span><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>Y</span><span class="dhb">T</span><span>E</span><span>G</span><span>H</span><span>N</span><span>Q</span><span class="dhb">T</span><span>L</span><span>M</span><span>A</span><span>M</span><span class="dhb">T</span><span class="dha">S</span><span class="dhb">T</span><span>D</span><span>I</span><span>A</span><span>L</span><span>I</span><span class="dha">S</span><span>N</span><span class="cys">C</span><span>-</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dhb">T</span><span>A</span><span class="dha">S</span><span>R</span><span>P</span><span>G</span><span>V</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">53</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">27.1%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP079946.1&start=4852506&end=4852704">NZ_CP079946.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: KYK13_RS20175_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">9</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span>-</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>V</span><span> </span><span> </span><span>G</span><span> </span><span> </span><span> </span><span> </span><span class="near-match-cycle">+</span><span>E</span><span> </span><span> </span><span>+</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="near-match-cycle"> </span><span>G</span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>V</span><span>F</span><span>V</span><span>G</span><span>G</span><span>-</span><span class="dhb">T</span><span class="dha">S</span><span class="dhb">T</span><span>E</span><span>K</span><span>A</span><span>L</span><span class="dhb">T</span><span>E</span><span class="dha">S</span><span>N</span><span>L</span><span>V</span><span>A</span><span>D</span><span class="dhb">T</span><span>G</span><span class="cys">C</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dhb">T</span><span>A</span><span class="dha">S</span><span class="dhb">T</span><span class="dhb">T</span><span>A</span><span>F</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">41</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">27.1%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JANYMP010000013.1&start=64951&end=65137">NZ_JANYMP010000013.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: NZH93_RS25860_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">17</div>
<div class="comparippson-sequence"><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>G</span><span> </span><span>+</span><span> </span><span>E</span><span> </span><span>+</span><span>I</span><span> </span><span> </span><span class="near-match-cycle"> </span><span> </span><span>P</span><span> </span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span>R</span><span class="cys near-match-cycle">C</span><span> </span><span class="near-match-cycle">+</span><span>A</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span class="dha near-match-cycle">S</span><span>R</span><span> </span><span> </span><span>+</span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">6</div>
<div class="comparippson-sequence"><span>G</span><span>E</span><span>F</span><span>A</span><span>E</span><span>G</span><span>D</span><span>I</span><span>V</span><span>A</span><span class="cys">C</span><span>N</span><span>P</span><span>P</span><span>G</span><span class="dha">S</span><span>-</span><span>R</span><span class="cys">C</span><span>-</span><span class="dha">S</span><span>A</span><span class="cys">C</span><span class="dhb">T</span><span>A</span><span class="dha">S</span><span>R</span><span>P</span><span>V</span><span>V</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">35</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">26.2%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_AP023359.1&start=7976701&end=7976932">NZ_AP023359.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: Prubr_RS34925_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>-</span><span>-</span><span>-</span><span>-</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>-</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span></div>
<div class="comparippson-coordinate">30</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>W</span><span> </span><span class="near-match-cycle">+</span><span>G</span><span> </span><span>+</span><span>P</span><span> </span><span> </span><span> </span><span> </span><span>D</span><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span>+</span><span> </span><span>L</span><span> </span><span>G</span><span> </span><span>Y</span><span> </span><span>E</span><span>+</span><span>+</span><span>I</span><span class="dhb near-match-cycle">T</span><span> </span><span class="dhb near-match-cycle">T</span><span>G</span><span> </span><span>G</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>W</span><span>L</span><span class="dhb">T</span><span>G</span><span class="dhb">T</span><span>D</span><span>P</span><span>V</span><span>D</span><span>G</span><span>W</span><span>D</span><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>L</span><span>F</span><span>L</span><span>G</span><span>G</span><span>K</span><span>Y</span><span>A</span><span>E</span><span>A</span><span>D</span><span>I</span><span class="dhb">T</span><span>A</span><span class="dhb">T</span><span>G</span><span class="dha">S</span><span>G</span></div>
<div class="comparippson-coordinate">35</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">25.0%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NC_015711.1&start=5704833&end=5705022">NC_015711.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP022203.1&start=3833082&end=3833271">NZ_CP022203.1</a></span>
<span class="comparippson-description">Lanthipeptides: MYMAC_RS16130_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: LILAB_RS23630_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">33</div>
<div class="comparippson-sequence"><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>G</span><span>R</span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span> </span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span>G</span><span class="dha near-match-cycle">S</span><span>+</span><span class="dhb near-match-cycle">T</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>G</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>Q</span><span class="dhb">T</span><span>R</span><span>L</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">16</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">25.0%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_AZVD01000036.1&start=15625&end=15844">NZ_AZVD01000036.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_KB913036.1&start=1236177&end=1236396">NZ_KB913036.1</a></span>
<span class="comparippson-description">Lanthipeptides: A39M_RS0105590_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: K389_RS0119060_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">17</div>
<div class="comparippson-sequence"><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">42</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>G</span><span> </span><span>Y</span><span>V</span><span> </span><span> </span><span>E</span><span>I</span><span class="dhb near-match-cycle">T</span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="near-match-cycle">+</span><span>A</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>G</span><span class="dha near-match-cycle">S</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">6</div>
<div class="comparippson-sequence"><span>G</span><span>R</span><span>Y</span><span>V</span><span>V</span><span>Q</span><span>E</span><span>I</span><span class="dhb">T</span><span>A</span><span class="dhb">T</span><span>A</span><span>V</span><span>A</span><span>N</span><span>W</span><span class="dhb">T</span><span>Y</span><span class="cys">C</span><span>-</span><span class="dha">S</span><span>A</span><span class="cys">C</span><span class="dhb">T</span><span>G</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">30</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">25.0%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP077072.1&start=2280343&end=2280532">NZ_CP077072.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: KR767_RS10085_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">8</div>
<div class="comparippson-sequence"><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>-</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>D</span><span>+</span><span>P</span><span> </span><span>G</span><span> </span><span>+</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span>+</span><span class="near-match-cycle">+</span><span> </span><span> </span><span> </span><span>M</span><span class="dhb near-match-cycle">T</span><span> </span><span>P</span><span> </span><span> </span><span>+</span><span> </span><span>G</span><span> </span><span> </span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span class="dha near-match-cycle">S</span><span>R</span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span>D</span><span>N</span><span>P</span><span>A</span><span>G</span><span class="dha">S</span><span>L</span><span>Y</span><span class="dhb">T</span><span>E</span><span>G</span><span>L</span><span>A</span><span>K</span><span class="dhb">T</span><span>M</span><span>N</span><span>A</span><span>M</span><span class="dhb">T</span><span>N</span><span>P</span><span>E</span><span>V</span><span>A</span><span>I</span><span>G</span><span class="dhb">T</span><span>N</span><span class="cys">C</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dhb">T</span><span>A</span><span class="dha">S</span><span>R</span><span>P</span><span>G</span><span>F</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">43</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">25.0%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_FWXV01000001.1&start=757419&end=757608">NZ_FWXV01000001.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: B9C43_RS03005_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">17</div>
<div class="comparippson-sequence"><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>G</span><span> </span><span>+</span><span> </span><span>E</span><span> </span><span>+</span><span>I</span><span> </span><span> </span><span class="near-match-cycle"> </span><span> </span><span>P</span><span> </span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span>R</span><span class="cys near-match-cycle">C</span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span class="dha near-match-cycle">S</span><span>R</span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">6</div>
<div class="comparippson-sequence"><span>G</span><span>E</span><span>F</span><span>A</span><span>E</span><span>G</span><span>D</span><span>I</span><span>V</span><span>A</span><span class="cys">C</span><span>N</span><span>P</span><span>P</span><span>G</span><span class="dha">S</span><span>-</span><span>R</span><span class="cys">C</span><span>-</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dhb">T</span><span>A</span><span class="dha">S</span><span>R</span><span>P</span><span>V</span><span>F</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">35</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">24.5%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP070499.1&start=3232735&end=3232933">NZ_CP070499.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP070499.1&start=3232735&end=3232933">NZ_CP070499.1</a></span>
<span class="comparippson-description">Lanthipeptides: JQS43_RS14385_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: JQS43_RS14385_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">4</div>
<div class="comparippson-sequence"><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">47</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span> </span><span>D</span><span>+</span><span> </span><span>+</span><span> </span><span> </span><span>+</span><span> </span><span> </span><span>+</span><span> </span><span> </span><span>G</span><span> </span><span> </span><span>V</span><span>+</span><span> </span><span> </span><span>I</span><span> </span><span>M</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span> </span><span> </span><span>+</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span> </span><span>+</span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">8</div>
<div class="comparippson-sequence"><span>A</span><span>D</span><span>D</span><span>W</span><span>E</span><span>L</span><span>D</span><span>I</span><span>K</span><span>F</span><span>I</span><span>E</span><span>A</span><span>G</span><span>D</span><span>A</span><span>V</span><span>K</span><span>H</span><span>L</span><span>I</span><span>Y</span><span>M</span><span class="dhb">T</span><span>-</span><span>-</span><span>-</span><span>-</span><span>N</span><span>D</span><span>Q</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span class="dhb">T</span><span class="cys">C</span><span>Q</span><span class="dha">S</span><span>A</span><span class="cys">C</span><span class="dhb">T</span><span>N</span><span class="dhb">T</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">47</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">22.9%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_BIFH01000038.1&start=20442&end=20646">NZ_BIFH01000038.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: EMA17_RS39120_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">35</div>
<div class="comparippson-sequence"><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span> </span><span class="dha near-match-cycle">S</span><span>R</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">22</div>
<div class="comparippson-sequence"><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>W</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>R</span><span>A</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">35</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">22.9%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP095741.1&start=823965&end=824154">NZ_CP095741.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: L2Y96_RS03630_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">8</div>
<div class="comparippson-sequence"><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>-</span><span>-</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>D</span><span>+</span><span>P</span><span> </span><span>G</span><span> </span><span>+</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span>+</span><span class="near-match-cycle">+</span><span> </span><span>+</span><span> </span><span>M</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span>+</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="near-match-cycle">+</span><span>+</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span class="dha near-match-cycle">S</span><span>R</span><span> </span><span> </span><span>+</span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span>D</span><span>N</span><span>P</span><span>A</span><span>G</span><span class="dha">S</span><span>L</span><span>Y</span><span class="dhb">T</span><span>E</span><span>G</span><span>Q</span><span>A</span><span>Q</span><span class="dhb">T</span><span>M</span><span>V</span><span>A</span><span>M</span><span class="dhb">T</span><span>N</span><span>A</span><span>E</span><span>M</span><span>A</span><span>L</span><span>A</span><span class="dha">S</span><span>N</span><span class="cys">C</span><span>-</span><span class="dha">S</span><span class="dha">S</span><span class="cys">C</span><span class="dhb">T</span><span>A</span><span class="dha">S</span><span>R</span><span>P</span><span>G</span><span>L</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">43</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">22.9%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP097263.1&start=2522981&end=2523161">NZ_CP097263.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: M3Q35_RS11595_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">35</div>
<div class="comparippson-sequence"><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>G</span><span class="dha near-match-cycle">S</span><span>R</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">11</div>
<div class="comparippson-sequence"><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dhb">T</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>R</span><span>Q</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">24</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">22.4%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_QOIL01000017.1&start=25195&end=25387">NZ_QOIL01000017.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: DQ384_RS40545_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">8</div>
<div class="comparippson-sequence"><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span></div>
<div class="comparippson-coordinate">27</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>+</span><span>+</span><span>P</span><span> </span><span>G</span><span>P</span><span> </span><span>L</span><span> </span><span>G</span><span> </span><span>Y</span><span> </span><span>E</span><span> </span><span>E</span><span>I</span><span class="dhb near-match-cycle">T</span><span>M</span><span class="near-match-cycle">+</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">12</div>
<div class="comparippson-sequence"><span>E</span><span>N</span><span>P</span><span>A</span><span>G</span><span>P</span><span>P</span><span>L</span><span>I</span><span>G</span><span>E</span><span>Y</span><span class="dhb">T</span><span>E</span><span>E</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">31</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">20.8%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_LMUE01000002.1&start=93060&end=93246">NZ_LMUE01000002.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: allorf_0093061_0093246_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">35</div>
<div class="comparippson-sequence"><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>G</span><span class="dha near-match-cycle">S</span><span>+</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dhb">T</span><span>G</span><span class="dha">S</span><span>Q</span><span class="dhb">T</span><span>R</span><span>Q</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">14</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">18.8%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP045572.1&start=8129061&end=8129265">NZ_CP045572.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP045572.1&start=8129061&end=8129265">NZ_CP045572.1</a></span>
<span class="comparippson-description">Lanthipeptides: GBF35_RS37485_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: GBF35_RS37485_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">24</div>
<div class="comparippson-sequence"><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">39</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>I</span><span> </span><span>M</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span>G</span><span> </span><span>+</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">3</div>
<div class="comparippson-sequence"><span>I</span><span>R</span><span>M</span><span class="dhb">T</span><span>D</span><span>D</span><span>G</span><span class="cys">C</span><span>N</span><span class="dha">S</span><span class="dhb">T</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">18</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">18.8%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP048405.1&start=3121647&end=3121833">NZ_CP048405.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: allorf_3121648_3121833_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">13</div>
<div class="comparippson-sequence"><span>P</span><span>V</span><span>L</span><span>P</span><span>G</span><span>M</span><span>Y</span><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span></div>
<div class="comparippson-coordinate">38</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>P</span><span>+</span><span> </span><span> </span><span>G</span><span>+</span><span> </span><span>+</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span>G</span><span>P</span><span> </span><span> </span><span>+</span><span>G</span><span>R</span><span class="cys near-match-cycle">C</span><span>G</span><span> </span><span>A</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">12</div>
<div class="comparippson-sequence"><span>P</span><span>L</span><span>R</span><span>A</span><span>G</span><span>V</span><span>K</span><span>L</span><span>P</span><span>R</span><span>L</span><span class="dha">S</span><span>R</span><span>A</span><span>A</span><span>G</span><span>P</span><span>P</span><span class="dhb">T</span><span>A</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span>R</span><span>A</span></div>
<div class="comparippson-coordinate">37</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">18.8%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP084058.1&start=8408864&end=8409101">NZ_CP084058.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: LCN96_RS39120_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">29</div>
<div class="comparippson-sequence"><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>P</span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span>G</span><span class="dhb near-match-cycle">T</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span class="dha near-match-cycle">S</span><span>R</span><span> </span><span> </span><span>+</span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">7</div>
<div class="comparippson-sequence"><span>P</span><span>A</span><span>A</span><span>I</span><span class="dhb">T</span><span>V</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>V</span><span class="cys">C</span><span class="dhb">T</span><span>D</span><span class="dha">S</span><span>R</span><span>P</span><span>R</span><span>V</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">26</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">18.8%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JAHQCX010000010.1&start=106387&end=106531">NZ_JAHQCX010000010.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: KTH90_RS14990_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">23</div>
<div class="comparippson-sequence"><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>E</span><span> </span><span class="dhb near-match-cycle">T</span><span>+</span><span> </span><span> </span><span>P</span><span> </span><span>+</span><span class="near-match-cycle">+</span><span> </span><span> </span><span> </span><span>G</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span> </span><span class="dhb near-match-cycle">T</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">14</div>
<div class="comparippson-sequence"><span>E</span><span>N</span><span class="dhb">T</span><span>V</span><span>Q</span><span>A</span><span>P</span><span>D</span><span>F</span><span class="dhb">T</span><span>F</span><span class="dhb">T</span><span>V</span><span>G</span><span>-</span><span>-</span><span class="cys">C</span><span class="dha">S</span><span class="dhb">T</span><span>V</span><span>M</span><span class="dhb">T</span><span>I</span><span>I</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">37</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">16.7%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_BCBX01000061.1&start=23453&end=23654">NZ_BCBX01000061.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_BCBX01000061.1&start=23453&end=23654">NZ_BCBX01000061.1</a></span>
<span class="comparippson-description">Lanthipeptides: AP234_RS35640_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: AP234_RS35640_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">24</div>
<div class="comparippson-sequence"><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">39</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>I</span><span> </span><span>M</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span>G</span><span> </span><span>+</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">3</div>
<div class="comparippson-sequence"><span>I</span><span>R</span><span>M</span><span class="dhb">T</span><span>D</span><span>D</span><span>G</span><span class="cys">C</span><span>N</span><span class="dha">S</span><span class="dhb">T</span><span class="cys">C</span><span>A</span><span class="dhb">T</span><span>A</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">18</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">16.7%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP021121.1&start=4287016&end=4287208">NZ_CP021121.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP021121.1&start=4287016&end=4287208">NZ_CP021121.1</a></span>
<span class="comparippson-description">Lanthipeptides: CAG99_RS19010_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: CAG99_RS19010_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">24</div>
<div class="comparippson-sequence"><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">39</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>I</span><span> </span><span>+</span><span class="dhb near-match-cycle">T</span><span>G</span><span> </span><span>G</span><span> </span><span>+</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">3</div>
<div class="comparippson-sequence"><span>I</span><span>R</span><span>L</span><span class="dhb">T</span><span>G</span><span>D</span><span>G</span><span class="cys">C</span><span>N</span><span class="dha">S</span><span class="dhb">T</span><span class="cys">C</span><span>A</span><span class="dhb">T</span><span>A</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">18</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">16.7%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP045572.1&start=3549131&end=3549329">NZ_CP045572.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP045572.1&start=3549131&end=3549329">NZ_CP045572.1</a></span>
<span class="comparippson-description">Lanthipeptides: GBF35_RS16225_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: GBF35_RS16225_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">24</div>
<div class="comparippson-sequence"><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span></div>
<div class="comparippson-coordinate">41</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>I</span><span> </span><span>M</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span>G</span><span> </span><span>+</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span><span>+</span><span> </span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">3</div>
<div class="comparippson-sequence"><span>I</span><span>R</span><span>M</span><span class="dhb">T</span><span>D</span><span>D</span><span>G</span><span class="cys">C</span><span>N</span><span class="dha">S</span><span class="dhb">T</span><span class="cys">C</span><span>A</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span>N</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">20</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">16.7%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP099837.1&start=4973408&end=4973555">NZ_CP099837.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP099837.1&start=4973408&end=4973555">NZ_CP099837.1</a></span>
<span class="comparippson-description">Lanthipeptides: NE857_RS22000_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: NE857_RS22000_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">20</div>
<div class="comparippson-sequence"><span>V</span><span>E</span><span class="dha">S</span><span>E</span><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">42</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>+</span><span>+</span><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span> </span><span>+</span><span class="dhb near-match-cycle">T</span><span>G</span><span> </span><span> </span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="near-match-cycle">+</span><span>A</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span> </span><span class="dha near-match-cycle">S</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>L</span><span>D</span><span class="dha">S</span><span>L</span><span>L</span><span>N</span><span>L</span><span class="dhb">T</span><span>G</span><span>D</span><span>N</span><span class="cys">C</span><span>G</span><span class="dha">S</span><span class="dhb">T</span><span class="cys">C</span><span>E</span><span class="dha">S</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span class="dhb">T</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">23</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">16.7%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP104318.1&start=249270&end=249450">NZ_CP104318.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP104318.1&start=249270&end=249450">NZ_CP104318.1</a></span>
<span class="comparippson-description">Lanthipeptides: NW605_RS33555_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: NW605_RS33555_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">24</div>
<div class="comparippson-sequence"><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">42</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>+</span><span> </span><span> </span><span class="dhb near-match-cycle">T</span><span>G</span><span> </span><span>G</span><span> </span><span class="near-match-cycle">+</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="near-match-cycle">+</span><span>A</span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span> </span><span class="dha near-match-cycle">S</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>L</span><span>N</span><span>D</span><span class="dhb">T</span><span>G</span><span>D</span><span>G</span><span class="cys">C</span><span class="dhb">T</span><span>A</span><span class="dhb">T</span><span class="cys">C</span><span>E</span><span class="dha">S</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>N</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">19</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">16.7%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP119143.1&start=7573413&end=7573599">NZ_CP119143.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP119143.1&start=7573413&end=7573599">NZ_CP119143.1</a></span>
<span class="comparippson-description">Lanthipeptides: PYR72_RS32440_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: PYR72_RS32440_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">24</div>
<div class="comparippson-sequence"><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span></div>
<div class="comparippson-coordinate">41</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>I</span><span> </span><span>M</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span>G</span><span> </span><span>+</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">3</div>
<div class="comparippson-sequence"><span>I</span><span>R</span><span>M</span><span class="dhb">T</span><span>D</span><span>D</span><span>G</span><span class="cys">C</span><span>N</span><span class="dha">S</span><span class="dhb">T</span><span class="cys">C</span><span>A</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dhb">T</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">20</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">14.6%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NC_016109.1&start=6590401&end=6590584">NC_016109.1</a><span class="comparippson-extra-indicator">and 4 others</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NC_016109.1&start=6590401&end=6590584">NC_016109.1</a></span>
<span class="comparippson-description">Lanthipeptides: KSE_RS29105_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_AP023440.1&start=4042989&end=4043172">NZ_AP023440.1</a></span>
<span class="comparippson-description">Lanthipeptides: IGS71_RS16730_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_AP023440.1&start=4042989&end=4043172">NZ_AP023440.1</a></span>
<span class="comparippson-description">Lanthipeptides: IGS71_RS16730_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_AP023440.1&start=4042989&end=4043172">NZ_AP023440.1</a></span>
<span class="comparippson-description">Lanthipeptides: IGS71_RS16730_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: KSE_RS29105_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">24</div>
<div class="comparippson-sequence"><span>I</span><span class="dhb">T</span><span>M</span><span class="dhb">T</span><span>G</span><span>P</span><span>G</span><span>Y</span><span class="dha">S</span><span>G</span><span>R</span><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">39</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>I</span><span> </span><span>+</span><span class="dhb near-match-cycle">T</span><span> </span><span> </span><span>G</span><span> </span><span>+</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span class="dhb near-match-cycle">T</span><span>A</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">3</div>
<div class="comparippson-sequence"><span>I</span><span>R</span><span>L</span><span class="dhb">T</span><span>N</span><span>D</span><span>G</span><span class="cys">C</span><span>N</span><span class="dha">S</span><span class="dhb">T</span><span class="cys">C</span><span>A</span><span class="dhb">T</span><span>A</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">18</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">14.6%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP026498.1&start=9420594&end=9420819">NZ_CP026498.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: C2142_RS40575_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span>R</span><span class="dha">S</span><span>G</span><span>D</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span>P</span><span>V</span><span>G</span><span>P</span><span>V</span><span>L</span></div>
<div class="comparippson-coordinate">15</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>R</span><span> </span><span>G</span><span> </span><span>E</span><span>P</span><span>D</span><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span> </span><span> </span><span>V</span><span>+</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">17</div>
<div class="comparippson-sequence"><span>R</span><span>E</span><span>G</span><span>R</span><span>E</span><span>P</span><span>D</span><span class="dha">S</span><span class="dha">S</span><span>I</span><span>A</span><span>R</span><span>V</span><span>M</span></div>
<div class="comparippson-coordinate">30</div>
</div>
</div>
</div>
</div>
</div>
<div class="comparippson-details">
<h4>MIBiG 3.1 matches for HS041_20445</h4>
<div class="comparippson-block">
<div class="comparippson-container">
<span class="comparippson-similarity">10.4%</span>
<span class="comparippson-link"><a href="https://mibig.secondarymetabolites.org/go/BGC0001155">BGC0001155 (AGY49587.1)</a>:</span>
<span class="comparippson-description">Thiopeptide: GE2270</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">35</div>
<div class="comparippson-sequence"><span class="cys">C</span><span>G</span><span class="dhb">T</span><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">42</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="cys near-match-cycle">C</span><span>G</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="dha near-match-cycle">S</span><span> </span><span class="dha near-match-cycle">S</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">6</div>
<div class="comparippson-sequence"><span class="cys">C</span><span>G</span><span>F</span><span class="cys">C</span><span class="cys">C</span><span class="dha">S</span><span class="cys">C</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">13</div>
</div>
</div><div class="comparippson-container">
<span class="comparippson-similarity">7.8%</span>
<span class="comparippson-link"><a href="https://mibig.secondarymetabolites.org/go/BGC0002120">BGC0002120 (NQE92413.1)</a>:</span>
<span class="comparippson-description">Lanthipeptide: nocathioamide A</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">38</div>
<div class="comparippson-sequence"><span>A</span><span class="cys">C</span><span class="dha">S</span><span>G</span><span class="dha">S</span><span>R</span><span class="dhb">T</span><span>Y</span><span>M</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">48</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>A</span><span class="cys near-match-cycle">C</span><span> </span><span> </span><span> </span><span> </span><span class="near-match-cycle">+</span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">39</div>
<div class="comparippson-sequence"><span>A</span><span class="cys">C</span><span>H</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span class="dha">S</span><span>L</span><span>P</span><span class="cys">C</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">49</div>
</div>
</div>
</div>
</div>
</div>

<div class="ripp-motif">


<div class="ripp-name">
<span class="serif">allorf_1751188_1751322</span>
</div>
<div class="ripp-segment-type">Leader:</div>
<div class="ripp-segment-sequence">MSLPCRTPSSGRGWGEVSSGA</div>

<div class="ripp-segment-type">Core:</div>
<div class="ripp-segment-sequence"><span>G</span><span class="dhb">T</span><span>P</span><span>I</span><span class="dhb">T</span><span>F</span><span>H</span><span class="cys">C</span><span>L</span><span class="dha">S</span><span>V</span><span>P</span><span>V</span><span>H</span><span>F</span><span class="dha">S</span><span>R</span><span>A</span><span>L</span><span>F</span><span>P</span><span>A</span><span>R</span></div>
<div class="ripp-segment-type">Core with dehydrations:</div>
<div class="ripp-segment-sequence ripp-dehydrated"><span>G</span><span class="dhb">Dhb</span><span>P</span><span>I</span><span class="dhb">Dhb</span><span>F</span><span>H</span><span class="cys">Cys</span><span>L</span><span class="dha">Dha</span><span>V</span><span>P</span><span>V</span><span>H</span><span>F</span><span class="dha">Dha</span><span>R</span><span>A</span><span>L</span><span>F</span><span>P</span><span>A</span><span>R</span></div>

<div class="comparippson-details">
<h4>antiSMASH-DB 4.0 matches for allorf_1751188_1751322</h4>
<div class="comparippson-block">
<div class="comparippson-container">
<span class="comparippson-similarity">30.3%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABRUZ010000012.1&start=1111&end=1255">NZ_JABRUZ010000012.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JALATF010000002.1&start=54351&end=54495">NZ_JALATF010000002.1</a></span>
<span class="comparippson-description">Lanthipeptides: allorf_0054352_0054495_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: allorf_001112_001255_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dhb">T</span><span>P</span><span>I</span><span class="dhb">T</span><span>F</span><span>H</span><span class="cys">C</span><span>L</span><span class="dha">S</span><span>V</span><span>P</span><span>V</span><span>H</span><span>F</span><span class="dha">S</span><span>R</span><span>A</span><span>L</span><span>F</span></div>
<div class="comparippson-coordinate">20</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dhb near-match-cycle">T</span><span>P</span><span>I</span><span> </span><span>F</span><span>H</span><span class="cys near-match-cycle">C</span><span> </span><span> </span><span>+</span><span>P</span><span>+</span><span> </span><span>F</span><span> </span><span>R</span><span> </span><span> </span><span>F</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">13</div>
<div class="comparippson-sequence"><span class="dhb">T</span><span>P</span><span>I</span><span>-</span><span>F</span><span>H</span><span class="cys">C</span><span>W</span><span>K</span><span>I</span><span>P</span><span>I</span><span>D</span><span>F</span><span>I</span><span>R</span><span>F</span><span>Y</span><span>F</span></div>
<div class="comparippson-coordinate">30</div>
</div>
</div><div class="comparippson-container">
<span class="comparippson-similarity">27.3%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JH600294.1&start=63657&end=63801">NZ_JH600294.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: allorf_063658_063801_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dhb">T</span><span>P</span><span>I</span><span class="dhb">T</span><span>F</span><span>H</span><span class="cys">C</span><span>L</span><span class="dha">S</span><span>V</span><span>P</span><span>V</span><span>H</span><span>F</span><span class="dha">S</span><span>R</span><span>A</span><span>L</span><span>F</span></div>
<div class="comparippson-coordinate">20</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dhb near-match-cycle">T</span><span>P</span><span>I</span><span> </span><span> </span><span>H</span><span class="cys near-match-cycle">C</span><span> </span><span> </span><span>+</span><span>P</span><span>+</span><span> </span><span>F</span><span> </span><span>R</span><span> </span><span> </span><span>F</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">13</div>
<div class="comparippson-sequence"><span class="dhb">T</span><span>P</span><span>I</span><span>-</span><span>L</span><span>H</span><span class="cys">C</span><span>W</span><span>K</span><span>I</span><span>P</span><span>I</span><span>D</span><span>F</span><span>I</span><span>R</span><span>F</span><span>Y</span><span>F</span></div>
<div class="comparippson-coordinate">30</div>
</div>
</div><div class="comparippson-container">
<span class="comparippson-similarity">20.3%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_BROP01000004.1&start=14629&end=14866">NZ_BROP01000004.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: allorf_014630_014866_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>G</span><span class="dhb">T</span><span>P</span><span>I</span><span class="dhb">T</span><span>F</span><span>H</span><span class="cys">C</span><span>L</span><span class="dha">S</span><span>V</span><span>P</span><span>V</span><span>H</span><span>F</span><span class="dha">S</span><span>R</span><span>A</span><span>L</span><span>F</span><span>P</span><span>A</span><span>R</span></div>
<div class="comparippson-coordinate">23</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>G</span><span> </span><span>P</span><span> </span><span> </span><span> </span><span> </span><span class="cys near-match-cycle">C</span><span>L</span><span class="dha near-match-cycle">S</span><span>V</span><span>P</span><span>+</span><span> </span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span>A</span><span>+</span><span> </span><span>P</span><span>A</span><span>R</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">32</div>
<div class="comparippson-sequence"><span>G</span><span>R</span><span>P</span><span>R</span><span>R</span><span>L</span><span class="dha">S</span><span class="cys">C</span><span>L</span><span class="dha">S</span><span>V</span><span>P</span><span>I</span><span class="dha">S</span><span>L</span><span class="dha">S</span><span>P</span><span>A</span><span>V</span><span>-</span><span>P</span><span>A</span><span>R</span></div>
<div class="comparippson-coordinate">53</div>
</div>
</div>
</div>
</div>
<div class="comparippson-details">
<h4>MIBiG 3.1 matches for allorf_1751188_1751322</h4>
<div class="comparippson-block">
<div class="comparippson-container">
<span class="comparippson-similarity">17.4%</span>
<span class="comparippson-link"><a href="https://mibig.secondarymetabolites.org/go/BGC0000503">BGC0000503 (cinA)</a>:</span>
<span class="comparippson-description">Lanthipeptide: cinnamycin</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">3</div>
<div class="comparippson-sequence"><span>P</span><span>I</span><span class="dhb">T</span><span>F</span><span>H</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">8</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>P</span><span> </span><span class="dhb near-match-cycle">T</span><span>F</span><span> </span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">9</div>
<div class="comparippson-sequence"><span>P</span><span>F</span><span class="dhb">T</span><span>F</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">14</div>
</div>
</div>
</div>
</div>
</div>


<div class="ripp-legend">
<h4> Legend: </h4>
<div>
<span class="dha">Dha</span>: Didehydroalanine<br>
<span class="dhb">Dhb</span>: Didehydrobutyrine<br>
</div>
</div>

</div>
</div>
</div>







<div class="body-details-section r2c6-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-95"></div> <span class="help-tooltip" id="tfbs-body-help-95">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
DmdR1
<div class="collapser collapser-target-DmdR1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Iron(II)-dependent repressor<br>
<span>Score</span>: 33.07 out of 36.53
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r2c6-1765225-DmdR1"></div>

<div class="tfbs-hit-detail">
DmdR1
<div class="collapser collapser-target-DmdR1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Iron(II)-dependent repressor<br>
<span>Score</span>: 27.84 out of 36.53
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r2c6-1768938-DmdR1"></div>

<div class="tfbs-hit-detail">
DmdR1
<div class="collapser collapser-target-DmdR1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Iron(II)-dependent repressor<br>
<span>Score</span>: 28.86 out of 36.53
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r2c6-1768995-DmdR1"></div>

<div class="tfbs-hit-detail">
DmdR1
<div class="collapser collapser-target-DmdR1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Iron(II)-dependent repressor<br>
<span>Score</span>: 30.62 out of 36.53
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r2c6-1790725-DmdR1"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AfsQ1
<div class="collapser collapser-target-AfsQ1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Two-component system AfsQ1-Q2, activator of antibiotic production<br>
<span>Score</span>: 16.94 out of 19.52
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c6-1749729-AfsQ1"></div>

<div class="tfbs-hit-detail">
DmdR1
<div class="collapser collapser-target-DmdR1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Iron(II)-dependent repressor<br>
<span>Score</span>: 25.99 out of 36.53
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c6-1765230-DmdR1"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.93 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r2c6-1770229-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r2c6-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-96"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-96">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c6-domain-bubble-svg" data-filename="JABTEZ010000002.1_r2c6_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c6-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC14">
Candidate 14 (1729535 - 1806044): neighbouring lanthipeptide-class-ii-NRP-metallophore-NRPS
</option>



<option value="CC16">
Candidate 16 (1745349 - 1806044): chemical hybrid NRP-metallophore-NRPS
</option>


</select>

<div class="details-svg" id="r2c6-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c6-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c6-lanthipeptides" data-name="r2c6-lanthipeptides">
Lanthipeptides
</div>



<div class="sidepanel-details-header r2c6-nrps_pks_products" data-name="r2c6-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c6-nrps_pks_monomers" data-name="r2c6-nrps_pks_monomers">
NRPS/PKS substrates
</div>





<div class="sidepanel-details-header r2c6-tfbs-finder" data-name="r2c6-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r2c6-lanthipeptides">
<div class="more-details">
<div class="heading">
<span>Core peptide predictions</span>
<div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-91"></div> <span class="help-tooltip" id="lanthi-side-help-91">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
</div>

<dl class ="prediction-text">
<dt><span class="serif">HS041_20445</span> - Class II</dt>
<dd>Cleavage pHMM score: -0.10</dd>
<dd>RODEO score: 0</dd>
<dd>Monoisotopic mass: 4866.2 Da</dd>
<dd>Molecular weight: 4869.4 Da</dd>
<dd>Number of bridges: 4</dd>
<dd>
<dl>
<dt>Alternative weights</dt>
<dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>

<dd>
<span class="alt-weight"> 4887.5 Da</span>
<span class="alt-weight-n"> N = 1</span>
</dd>

<dd>
<span class="alt-weight"> 4905.5 Da</span>
<span class="alt-weight-n"> N = 2</span>
</dd>

<dd>
<span class="alt-weight"> 4923.5 Da</span>
<span class="alt-weight-n"> N = 3</span>
</dd>

<dd>
<span class="alt-weight"> 4941.5 Da</span>
<span class="alt-weight-n"> N = 4</span>
</dd>

<dd>
<span class="alt-weight"> 4959.5 Da</span>
<span class="alt-weight-n"> N = 5</span>
</dd>

<dd>
<span class="alt-weight"> 4977.6 Da</span>
<span class="alt-weight-n"> N = 6</span>
</dd>

</dl>
</dd>
<dt><span class="serif">allorf_1751188_1751322</span> - Class II</dt>
<dd>Cleavage pHMM score: -6.60</dd>
<dd>RODEO score: 0</dd>
<dd>Monoisotopic mass: 2480.3 Da</dd>
<dd>Molecular weight: 2481.9 Da</dd>
<dd>Number of bridges: 1</dd>
<dd>
<dl>
<dt>Alternative weights</dt>
<dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>

<dd>
<span class="alt-weight"> 2499.9 Da</span>
<span class="alt-weight-n"> N = 1</span>
</dd>

<dd>
<span class="alt-weight"> 2517.9 Da</span>
<span class="alt-weight-n"> N = 2</span>
</dd>

<dd>
<span class="alt-weight"> 2536.0 Da</span>
<span class="alt-weight-n"> N = 3</span>
</dd>

</dl>
</dd>

</dl>

</div>

</div>



<div class="sidepanel-details-section r2c6-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-92"></div> <span class="help-tooltip" id="nrps-structures-help-92">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 14, location 1729535 - 1806044:
<div class="collapser collapser-target-CC14 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c6-14" class="smiles-canvas" data-smiles="NC(CCCN(O)C=O)C(=O)NC([*])C(=O)NC(CCCNO)C(=O)NC(CCCNO)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(D-Fo-OH-Orn - X - OH-Orn - OH-Orn)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-fo-oh-orn,oh-orn,oh-orn" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-fo-oh-orn*,x,oh-orn*,oh-orn*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c6-14-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCCN(O)C=O)C(=O)NC([*])C(=O)NC(CCCNO)C(=O)NC(CCCNO)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 16, location 1745349 - 1806044:
<div class="collapser collapser-target-CC16 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c6-16" class="smiles-canvas" data-smiles="NC(CCCN(O)C=O)C(=O)NC([*])C(=O)NC(CCCNO)C(=O)NC(CCCNO)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(D-Fo-OH-Orn - X - OH-Orn - OH-Orn)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-fo-oh-orn,oh-orn,oh-orn" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-fo-oh-orn*,x,oh-orn*,oh-orn*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c6-16-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCCN(O)C=O)C(=O)NC([*])C(=O)NC(CCCNO)C(=O)NC(CCCNO)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c6-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-93"></div> <span class="help-tooltip" id="nrps-monomers-help-93">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">HS041_20495</span></strong>: X
<div class="collapser collapser-target-HS041_20495 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (18..395)</strong>: X
<div class="collapser collapser-target-HS041_20495 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-HS041_20495-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">PFAYWLLATK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>

<dt>Stachelhaus code match:</dt>
<dd>0% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">HS041_20565</span></strong>: Fo-OH-Orn - X - OH-Orn - OH-Orn
<div class="collapser collapser-target-HS041_20565 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=fo-oh-orn,oh-orn,oh-orn" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=fo-oh-orn*,oh-orn*,oh-orn*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (478..878)</strong>: Fo-OH-Orn
<div class="collapser collapser-target-HS041_20565 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Fo-OH-Orn
<div class="collapser collapser-target-HS041_20565-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVWIVGAVNK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Fo-OH-Orn <span class="serif">DVWIIGAVNK</span> (91% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2051..2474)</strong>: X
<div class="collapser collapser-target-HS041_20565 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad
<div class="collapser collapser-target-HS041_20565-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAWDFGCVQK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gln <span class="serif">DAWQFGVVDK</span> (59% 8Å match)</dd>
<dd>Arg <span class="serif">DADDSGCVDK</span> (50% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (3142..3555)</strong>: OH-Orn
<div class="collapser collapser-target-HS041_20565 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  OH-Orn
<div class="collapser collapser-target-HS041_20565-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Long positively charged side chain (Orn, Lys, Arg)</dd>
<dt>Small clusters prediction:</dt>
<dd>Orn and hydroxy-Orn specific (Orn, OH-Orn)</dd>
<dt>Single AA prediction:</dt>
<dd>orn (Orn)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DMENLGLINK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>OH-Orn <span class="serif">DMENLGLINK</span> (100% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (4222..4618)</strong>: OH-Orn
<div class="collapser collapser-target-HS041_20565 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  OH-Orn
<div class="collapser collapser-target-HS041_20565-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVFILGAVNK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>OH-Orn <span class="serif">DVFILGAVNK</span> (88% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>





<div class="sidepanel-details-section r2c6-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-94"></div> <span class="help-tooltip" id="tfbs-side-help-94">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DmdR1" target="_blank">DmdR1</a></td>
<td>Iron(II)-dependent repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsQ1" target="_blank">AfsQ1</a></td>
<td>Two-component system AfsQ1-Q2, activator of antibiotic production</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r2c7' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000002.1 - Region 7 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-256"></div> <span class="help-tooltip" id="region_svg-help-256">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000002.1.region007.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r2c7-svg" data-filename="JABTEZ010000002.1_r2c7.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1,936,660 - 1,998,654 nt. (total: 61,995 nt)

<a class="cluster-rules-header" id="r2c7-rules-header" href="#r2c7">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r2c7-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r2c7-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-257"></div> <span class="help-tooltip" id="focus-panel-help-257">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r2c7">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r2c7-gene-table-details" data-name="r2c7-gene-table-details">
Gene overview
</div>



<div class="body-details-header r2c7-nrps_pks" data-name="r2c7-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r2c7-MIBiG-cluster-compare" data-name="r2c7-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r2c7-knownclusterblast" data-name="r2c7-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r2c7-subclusterblast" data-name="r2c7-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r2c7-nrps-pks-bubbles" data-name="r2c7-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r2c7-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-105"></div> <span class="help-tooltip" id="gt-help-tooltip-help-105">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r2c7">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r2c7-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r2c7-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r2c7">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r2c7-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-98"></div> <span class="help-tooltip" id="nrps-domain-help-98">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c7-details-svg" data-filename="JABTEZ010000002.1_r2c7_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r2c7-details-svg"></div>
</div>
</div>



<div class="body-details-section r2c7-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-99"></div> <span class="help-tooltip" id="MIBiG-help-99">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r2c7-selector">Analysis type:</label>
<select id="comparison-MIBiG-r2c7-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r2c7"></div>
<div id="comparison-MIBiG-r2c7-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>





<td class="cc-heat-cell" title="0.73 (id:0.46, order:1.00, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>






<td class="cc-heat-cell" title="0.73 (id:0.46, order:1.00, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.83</td>
<td>NRP, Polyketide</td>

<td>thalassospiramide A</td>


<td>Tistrella bauzanensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>





<td class="cc-heat-cell" title="0.71 (id:0.45, order:1.00, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>






<td class="cc-heat-cell" title="0.71 (id:0.45, order:1.00, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>


<td>0.79</td>
<td>NRP</td>

<td>taxlllaid A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.47, order:1.00, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>






<td class="cc-heat-cell" title="0.70 (id:0.47, order:1.00, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.76</td>
<td>NRP</td>

<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>


<td>Pseudomonas sp. QS1027</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>






<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.72</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.71</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.70</td>
<td>NRP</td>

<td>syringafactin A, syringafactin C</td>


<td>Pseudomonas sp. SZ57</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002017: 0-12514">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002017/index.html#r1c1" target="_new">BGC0002017</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.69</td>
<td>NRP</td>

<td>potensibactin</td>


<td>Nocardiopsis potens DSM 45234</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.68</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 10E9</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.68</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 22D11</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>NRP</td>

<td>sevadicin</td>


<td>Paenibacillus larvae</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r2c7-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.46, order:1.00, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>NRP, Polyketide</td>
<td>thalassospiramide A</td>
<td>Tistrella bauzanensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>


<td class="cc-heat-cell" title="0.71 (id:0.45, order:1.00, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
<td>0.71</td>
<td>NRP</td>
<td>taxlllaid A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.47, order:1.00, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>
<td>Pseudomonas sp. QS1027</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002518: 0-10561">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002518/index.html#r1c1" target="_new">BGC0002518</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>syringafactin A, syringafactin C</td>
<td>Pseudomonas sp. SZ57</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002017: 0-12514">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002017/index.html#r1c1" target="_new">BGC0002017</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>potensibactin</td>
<td>Nocardiopsis potens DSM 45234</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 10E9</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 22D11</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>sevadicin</td>
<td>Paenibacillus larvae</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r2c7-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-100"></div> <span class="help-tooltip" id="cb-known-help-100">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r2c7-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r2c7_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r2c7_1.svg">
crochelin A
</option>

<option value = "svg/knownclusterblast_r2c7_2.svg">
ralsolamycin
</option>

<option value = "svg/knownclusterblast_r2c7_3.svg">
BE-43547A1/BE-43547A2/BE-43547B1/BE-43547B2/BE-43547B3/BE-43547C1/BE-43547C2
</option>

<option value = "svg/knownclusterblast_r2c7_4.svg">
microsclerodermin M
</option>

<option value = "svg/knownclusterblast_r2c7_5.svg">
vacidobactin A/vacidobactin B
</option>

<option value = "svg/knownclusterblast_r2c7_6.svg">
nostopeptolide A1/nostopeptolide 1052
</option>

<option value = "svg/knownclusterblast_r2c7_7.svg">
nostopeptolide A2
</option>

<option value = "svg/knownclusterblast_r2c7_8.svg">
rakicidin A/rakicidin B
</option>

<option value = "svg/knownclusterblast_r2c7_9.svg">
rakicidin A/rakicidin B
</option>

<option value = "svg/knownclusterblast_r2c7_10.svg">
A-500359 A/A-500359 B
</option>

</select>
<button id="knownclusterblast-r2c7-download">Download graphic</button>
</div>
<div id="knownclusterblast-r2c7-svg"></div>

</div>
</div>

<div class="body-details-section r2c7-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-101"></div> <span class="help-tooltip" id="cb-sub-help-101">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r2c7-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-104"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-104">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r2c7-domain-bubble-svg" data-filename="JABTEZ010000002.1_r2c7_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r2c7-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC17">
Candidate 17 (1936659 - 1998654): interleaved NRPS-T1PKS
</option>


</select>

<div class="details-svg" id="r2c7-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r2c7-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r2c7-nrps_pks_products" data-name="r2c7-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r2c7-nrps_pks_monomers" data-name="r2c7-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r2c7-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-102"></div> <span class="help-tooltip" id="nrps-structures-help-102">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 17, location 1936659 - 1998654:
<div class="collapser collapser-target-CC17 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r2c7-17" class="smiles-canvas" data-smiles="NCC(=O)NC(CO)C(=O)NC([*])C(=O)NC([*])C(=O)C=CC(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Gly) + (Ser) + (X - X) + (ccmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly&amp;nrps2=ser&amp;nrps4=ccmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*&amp;nrps2=ser*&amp;nrps3=x,x&amp;nrps4=ccmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r2c7-17-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NCC(=O)NC(CO)C(=O)NC([*])C(=O)NC([*])C(=O)C=CC(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r2c7-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-103"></div> <span class="help-tooltip" id="nrps-monomers-help-103">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">HS041_21295</span></strong>: Gly
<div class="collapser collapser-target-HS041_21295 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (463..847)</strong>: Gly
<div class="collapser collapser-target-HS041_21295 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly
<div class="collapser collapser-target-HS041_21295-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>gly (Gly)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DILQFGLIWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gly <span class="serif">DILQFGLIWK</span> (82% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_21300</span></strong>: Ser
<div class="collapser collapser-target-HS041_21300 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (513..917)</strong>: Ser
<div class="collapser collapser-target-HS041_21300 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ser
<div class="collapser collapser-target-HS041_21300-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>ser (Ser)</dd>
<dt>Single AA prediction:</dt>
<dd>ser (Ser)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVWHVSLVDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ser <span class="serif">DVWHVSLVDK</span> (97% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_21310</span></strong>: X - X
<div class="collapser collapser-target-HS041_21310 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (13..420)</strong>: X
<div class="collapser collapser-target-HS041_21310 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-HS041_21310-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVLTLAWVCK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Pro <span class="serif">DLLYLALVCK</span> (68% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1102..1508)</strong>: X
<div class="collapser collapser-target-HS041_21310 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Asp, Asn, Glu, Gln, Aad
<div class="collapser collapser-target-HS041_21310-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DPRHLGAVQK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Glu or D-Glu <span class="serif">DPRHSGVVGK</span> (47% 8Å match)</dd>
<dd>D-Glu <span class="serif">DPRHSGVVGK</span> (47% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_21315</span></strong>: mal
<div class="collapser collapser-target-HS041_21315 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (531..818)</strong>: mal
<div class="collapser collapser-target-HS041_21315 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-HS041_21315-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 91.7%</dd>
<dd>2-Rhyd-Malonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-HS041_21315-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 169.0</dt>

<dd></dd><dt>inactive: 112.1</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 68.8</dt>

<dd></dd><dt>Methylmalonyl-CoA: 61.6</dt>

<dd></dd><dt>Propionyl-CoA: 45.0</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 35.8</dt>

<dd></dd><dt>Acetyl-CoA: 33.5</dt>

<dd></dd><dt>Isobutyryl-CoA: 32.4</dt>

<dd></dd><dt>Benzoyl-CoA: 28.7</dt>

<dd></dd><dt>CHC-CoA: 24.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 23.2</dt>

<dd></dd><dt>fatty_acid: 18.0</dt>

<dd></dd><dt>trans-1,2-CPDA: 12.9</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 8.3</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>





<div class="page" id='r3c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000003.1 - Region 1 - terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-258"></div> <span class="help-tooltip" id="region_svg-help-258">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000003.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c1-svg" data-filename="JABTEZ010000003.1_r3c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 373,105 - 394,103 nt. (total: 20,999 nt)

<a class="cluster-rules-header" id="r3c1-rules-header" href="#r3c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c1-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)</div>

<div class="region-svg-container">
<div id='r3c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-259"></div> <span class="help-tooltip" id="focus-panel-help-259">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c1-gene-table-details" data-name="r3c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c1-MIBiG-cluster-compare" data-name="r3c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c1-knownclusterblast" data-name="r3c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c1-subclusterblast" data-name="r3c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r3c1-tfbs-finder" data-name="r3c1-tfbs-finder">
TFBS Finder
</div>


</div>


<div class="body-details-section r3c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-111"></div> <span class="help-tooltip" id="gt-help-tooltip-help-111">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-106"></div> <span class="help-tooltip" id="MIBiG-help-106">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c1"></div>
<div id="comparison-MIBiG-r3c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001201: 0-2291">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001201/index.html#r1c1" target="_new">BGC0001201</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Terpene</td>

<td>eremophilene</td>


<td>Sorangium cellulosum So ce56</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131: 0-6280">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.19</td>
<td>Polyketide</td>

<td>pyrrolomycin</td>


<td>Streptomyces sp. UC 11065</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294: 0-9019">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>


<td>0.16</td>
<td>Other</td>

<td>7-prenylisatin</td>


<td>Streptomyces sp. MBT28</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002412: 0-12427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002412/index.html#r1c1" target="_new">BGC0002412</a></td>





<td class="cc-heat-cell" title="0.42 (id:0.48, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.13</td>
<td>NRP</td>

<td>holomycin</td>


<td>Photobacterium galatheae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002387: 1-41257">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002387/index.html#r1c1" target="_new">BGC0002387</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>yangpumicin A</td>


<td>Micromonospora yangpuensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002091: 1-17973">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002091/index.html#r1c1" target="_new">BGC0002091</a></td>





<td class="cc-heat-cell" title="0.40 (id:0.48, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>


<td>0.12</td>
<td>NRP</td>

<td>holomycin</td>


<td>Yersinia ruckeri ATCC 29473</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000890: 0-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000890/index.html#r1c1" target="_new">BGC0000890</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.41, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Other (Aminocoumarin)</td>

<td>pentabromopseudilin</td>


<td>Pseudoalteromonas phenolica O-BC30</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000891: 0-11263">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000891/index.html#r1c1" target="_new">BGC0000891</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.41, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Other (Aminocoumarin)</td>

<td>pentabromopseudilin</td>


<td>Pseudoalteromonas luteoviolacea 2ta16</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000274: 0-34431">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000274/index.html#r1c1" target="_new">BGC0000274</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Polyketide</td>

<td>tetarimycin A, tetarimycin B</td>


<td>uncultured bacterium</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001465: 0-12220">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001465/index.html#r1c1" target="_new">BGC0001465</a></td>





<td class="cc-heat-cell" title="0.39 (id:0.41, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.12</td>
<td>Other</td>

<td>bromophene, bistribromopyrrole, pentabromopseudilin</td>


<td>Marinomonas mediterranea MMB-1</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001201: 0-2291">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001201/index.html#r1c1" target="_new">BGC0001201</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene</td>
<td>eremophilene</td>
<td>Sorangium cellulosum So ce56</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000131: 0-6280">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000131/index.html#r1c1" target="_new">BGC0000131</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.44, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Polyketide</td>
<td>pyrrolomycin</td>
<td>Streptomyces sp. UC 11065</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001294: 0-9019">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001294/index.html#r1c1" target="_new">BGC0001294</a></td>


<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>
<td>0.47</td>
<td>Other</td>
<td>7-prenylisatin</td>
<td>Streptomyces sp. MBT28</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002412: 0-12427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002412/index.html#r1c1" target="_new">BGC0002412</a></td>


<td class="cc-heat-cell" title="0.42 (id:0.48, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>
<td>0.42</td>
<td>NRP</td>
<td>holomycin</td>
<td>Photobacterium galatheae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002387: 1-41257">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002387/index.html#r1c1" target="_new">BGC0002387</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.44, order:N/A, components:0.36)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>Polyketide</td>
<td>yangpumicin A</td>
<td>Micromonospora yangpuensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002091: 1-17973">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002091/index.html#r1c1" target="_new">BGC0002091</a></td>


<td class="cc-heat-cell" title="0.40 (id:0.48, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.40), rgb(0, 0, 0, 0.40) 40%, white 40%)"></td>
<td>0.40</td>
<td>NRP</td>
<td>holomycin</td>
<td>Yersinia ruckeri ATCC 29473</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000890: 0-11090">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000890/index.html#r1c1" target="_new">BGC0000890</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.41, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Other (Aminocoumarin)</td>
<td>pentabromopseudilin</td>
<td>Pseudoalteromonas phenolica O-BC30</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000891: 0-11263">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000891/index.html#r1c1" target="_new">BGC0000891</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.41, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Other (Aminocoumarin)</td>
<td>pentabromopseudilin</td>
<td>Pseudoalteromonas luteoviolacea 2ta16</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000274: 0-34431">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000274/index.html#r1c1" target="_new">BGC0000274</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.33)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Polyketide</td>
<td>tetarimycin A, tetarimycin B</td>
<td>uncultured bacterium</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001465: 0-12220">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001465/index.html#r1c1" target="_new">BGC0001465</a></td>


<td class="cc-heat-cell" title="0.39 (id:0.41, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>
<td>0.39</td>
<td>Other</td>
<td>bromophene, bistribromopyrrole, pentabromopseudilin</td>
<td>Marinomonas mediterranea MMB-1</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-107"></div> <span class="help-tooltip" id="cb-known-help-107">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r3c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-108"></div> <span class="help-tooltip" id="cb-sub-help-108">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r3c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-110"></div> <span class="help-tooltip" id="tfbs-body-help-110">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 20.07 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r3c1-373447-LexA"></div>

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 20.34 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r3c1-373832-LexA"></div>

</div>
</div>
</div>

</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r3c1-tfbs-finder" data-name="r3c1-tfbs-finder">
TFBS Finder
</div>


</div>








<div class="sidepanel-details-section r3c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-109"></div> <span class="help-tooltip" id="tfbs-side-help-109">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>


</div>

</div>

</div>

<div class="page" id='r3c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000003.1 - Region 2 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-260"></div> <span class="help-tooltip" id="region_svg-help-260">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000003.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c2-svg" data-filename="JABTEZ010000003.1_r3c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 622,779 - 687,796 nt. (total: 65,018 nt)

<a class="cluster-rules-header" id="r3c2-rules-header" href="#r3c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c2-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r3c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-261"></div> <span class="help-tooltip" id="focus-panel-help-261">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c2-gene-table-details" data-name="r3c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c2-nrps_pks" data-name="r3c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c2-MIBiG-cluster-compare" data-name="r3c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c2-knownclusterblast" data-name="r3c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c2-subclusterblast" data-name="r3c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c2-tfbs-finder" data-name="r3c2-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r3c2-nrps-pks-bubbles" data-name="r3c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r3c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-121"></div> <span class="help-tooltip" id="gt-help-tooltip-help-121">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-112"></div> <span class="help-tooltip" id="nrps-domain-help-112">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c2-details-svg" data-filename="JABTEZ010000003.1_r3c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-113"></div> <span class="help-tooltip" id="MIBiG-help-113">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c2"></div>
<div id="comparison-MIBiG-r3c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>





<td class="cc-heat-cell" title="0.75 (id:0.48, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>


<td>0.45</td>
<td>NRP</td>

<td>sevadicin</td>


<td>Paenibacillus larvae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.37</td>
<td>NRP</td>

<td>thermoactinoamide A</td>


<td>Thermoactinomyces sp. AS95</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002017: 0-12514">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002017/index.html#r1c1" target="_new">BGC0002017</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>NRP</td>

<td>potensibactin</td>


<td>Nocardiopsis potens DSM 45234</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:0.75, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>NRP</td>

<td>taxlllaid A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.35</td>
<td>NRP</td>

<td>icosalide A, icosalide B</td>


<td>Burkholderia gladioli</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185: 0-11721">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>bacillibactin</td>


<td>Bacillus velezensis FZB42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001796: 0-40582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001796/index.html#r1c1" target="_new">BGC0001796</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>tauramamide</td>


<td>Brevibacillus laterosporus DSM 25</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>bovienimide A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351: 1-24369">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.49, order:0.69, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>exochelin</td>


<td>Mycolicibacterium smegmatis MC2 155</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401: 0-12882">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>paenibactin</td>


<td>Paenibacillus elgii B69</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>


<td class="cc-heat-cell" title="0.75 (id:0.48, order:1.00, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.75), rgb(0, 0, 0, 0.75) 75%, white 75%)"></td>
<td>0.75</td>
<td>NRP</td>
<td>sevadicin</td>
<td>Paenibacillus larvae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002437: 0-10596">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002437/index.html#r1c1" target="_new">BGC0002437</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>NRP</td>
<td>thermoactinoamide A</td>
<td>Thermoactinomyces sp. AS95</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002017: 0-12514">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002017/index.html#r1c1" target="_new">BGC0002017</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>potensibactin</td>
<td>Nocardiopsis potens DSM 45234</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:0.75, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>taxlllaid A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001833: 14964-30015">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001833/index.html#r1c1" target="_new">BGC0001833</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>icosalide A, icosalide B</td>
<td>Burkholderia gladioli</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185: 0-11721">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.48, order:N/A, components:0.94)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>bacillibactin</td>
<td>Bacillus velezensis FZB42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001796: 0-40582">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001796/index.html#r1c1" target="_new">BGC0001796</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>tauramamide</td>
<td>Brevibacillus laterosporus DSM 25</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002135: 0-11508">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002135/index.html#r1c1" target="_new">BGC0002135</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>bovienimide A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000351: 1-24369">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000351/index.html#r1c1" target="_new">BGC0000351</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.49, order:0.69, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>exochelin</td>
<td>Mycolicibacterium smegmatis MC2 155</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401: 0-12882">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.49, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>paenibactin</td>
<td>Paenibacillus elgii B69</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-114"></div> <span class="help-tooltip" id="cb-known-help-114">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r3c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r3c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r3c2_1.svg">
friulimicin A/friulimicin B/friulimicin C/friulimicin D
</option>

<option value = "svg/knownclusterblast_r3c2_2.svg">
coprisamide C/coprisamide D
</option>

<option value = "svg/knownclusterblast_r3c2_3.svg">
FR900359
</option>

<option value = "svg/knownclusterblast_r3c2_4.svg">
cysteoamide
</option>

<option value = "svg/knownclusterblast_r3c2_5.svg">
gobichelin A/gobichelin B
</option>

<option value = "svg/knownclusterblast_r3c2_6.svg">
actinomycin D
</option>

<option value = "svg/knownclusterblast_r3c2_7.svg">
JBIR-78/JBIR-95
</option>

<option value = "svg/knownclusterblast_r3c2_8.svg">
cadaside A/cadaside B
</option>

<option value = "svg/knownclusterblast_r3c2_9.svg">
hormaomycin/hormaomycin A1/hormaomycin A2/hormaomycin A3/hormaomycin A4/hormaomycin A5/hormaomycin A6
</option>

<option value = "svg/knownclusterblast_r3c2_10.svg">
WAP-8294A2
</option>

</select>
<button id="knownclusterblast-r3c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r3c2-svg"></div>

</div>
</div>

<div class="body-details-section r3c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-115"></div> <span class="help-tooltip" id="cb-sub-help-115">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c2-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-119"></div> <span class="help-tooltip" id="tfbs-body-help-119">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 27.53 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r3c2-624295-CelR"></div>

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 25.19 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r3c2-624296-CelR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ArgR
<div class="collapser collapser-target-ArgR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Regulator of arginine biosynthesis genes<br>
<span>Score</span>: 17.21 out of 20.01
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r3c2-687361-ArgR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r3c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-120"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-120">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c2-domain-bubble-svg" data-filename="JABTEZ010000003.1_r3c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r3c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (622778 - 687796): single NRPS
</option>


</select>

<div class="details-svg" id="r3c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r3c2-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c2-nrps_pks_products" data-name="r3c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r3c2-nrps_pks_monomers" data-name="r3c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r3c2-tfbs-finder" data-name="r3c2-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r3c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-116"></div> <span class="help-tooltip" id="nrps-structures-help-116">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 622778 - 687796:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c2-2" class="smiles-canvas" data-smiles="NC(C(C)CC)C(=O)NC(CO)C(=O)NC(C(O)C)C(=O)NC(CC(C)C)C(=O)NC(C(O)C)C(=O)NC(C(C)C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(D-Ile - Ser - D-Thr) + (Leu - Thr) + (Val)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-ile,ser,d-thr&amp;nrps2=leu,thr&amp;nrps3=val" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=d-ile*,ser*,d-thr*&amp;nrps2=leu*,thr*&amp;nrps3=val*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(C(C)CC)C(=O)NC(CO)C(=O)NC(C(O)C)C(=O)NC(CC(C)C)C(=O)NC(C(O)C)C(=O)NC(C(C)C)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r3c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-117"></div> <span class="help-tooltip" id="nrps-monomers-help-117">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">HS041_24560</span></strong>: pk
<div class="collapser collapser-target-HS041_24560 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>CAL_domain (35..477)</strong>: pk
<div class="collapser collapser-target-HS041_24560 collapser-level-candidate"> </div><div class="collapser-content">

minowa:  fatty_acid
<div class="collapser collapser-target-HS041_24560-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>fatty_acid: 24.2</dt>

<dd></dd><dt>NH2: 15.9</dt>

<dd></dd><dt>AHBA: 13.8</dt>

<dd></dd><dt>Acetyl-CoA: 0.1</dt>

<dd></dd><dt>shikimic_acid: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

<strong><span class="serif">HS041_24575</span></strong>: Ile - Ser - Thr
<div class="collapser collapser-target-HS041_24575 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile,ser,thr" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile*,ser*,thr*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (453..848)</strong>: Ile
<div class="collapser collapser-target-HS041_24575 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ile
<div class="collapser collapser-target-HS041_24575-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DGLFLGVVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ile <span class="serif">DGFFLGVVYK</span> (56% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>











<dd><strong>AMP-binding (2019..2400)</strong>: Ser
<div class="collapser collapser-target-HS041_24575 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ser
<div class="collapser collapser-target-HS041_24575-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>ser (Ser)</dd>
<dt>Single AA prediction:</dt>
<dd>ser (Ser)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVWHLSLVDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ser <span class="serif">DVWHLSLVDK</span> (97% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (3106..3545)</strong>: Thr
<div class="collapser collapser-target-HS041_24575 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Thr
<div class="collapser collapser-target-HS041_24575-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>thr (Thr)</dd>
<dt>Single AA prediction:</dt>
<dd>thr (Thr)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DFWNIGMVHK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Thr <span class="serif">DFWNIGMVHK</span> (100% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>








</div>
<br>

<strong><span class="serif">HS041_24580</span></strong>: Leu - Thr
<div class="collapser collapser-target-HS041_24580 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu,thr" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=leu*,thr*" target="_new">relaxed</a>
</dd>
<br>







<dd><strong>AMP-binding (441..836)</strong>: Leu
<div class="collapser collapser-target-HS041_24580 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Leu
<div class="collapser collapser-target-HS041_24580-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAMLVGAIAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Leu <span class="serif">DAMLVGAICK</span> (76% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1501..1940)</strong>: Thr
<div class="collapser collapser-target-HS041_24580 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Thr
<div class="collapser collapser-target-HS041_24580-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>thr (Thr)</dd>
<dt>Single AA prediction:</dt>
<dd>thr (Thr)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DFWNIGMVHK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Thr <span class="serif">DFWNIGMVHK</span> (100% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_24595</span></strong>: Val
<div class="collapser collapser-target-HS041_24595 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*" target="_new">relaxed</a>
</dd>
<br>







<dd><strong>AMP-binding (474..871)</strong>: Val
<div class="collapser collapser-target-HS041_24595 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Val
<div class="collapser collapser-target-HS041_24595-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>val (Val)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAYFLGVTYK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val or aIle <span class="serif">DAYFLGVTFK</span> (82% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r3c2-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-118"></div> <span class="help-tooltip" id="tfbs-side-help-118">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CelR" target="_blank">CelR</a></td>
<td>Cellobiose uptake repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ArgR" target="_blank">ArgR</a></td>
<td>Regulator of arginine biosynthesis genes</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r3c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000003.1 - Region 3 - lanthipeptide-class-iii <div class="help-container"> <div class="help-icon" data-id="region_svg-help-262"></div> <span class="help-tooltip" id="region_svg-help-262">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000003.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c3-svg" data-filename="JABTEZ010000003.1_r3c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 750,481 - 773,255 nt. (total: 22,775 nt)

<a class="cluster-rules-header" id="r3c3-rules-header" href="#r3c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c3-rules">lanthipeptide-class-iii: (micKC and not PKS_KS)</div>

<div class="region-svg-container">
<div id='r3c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-263"></div> <span class="help-tooltip" id="focus-panel-help-263">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c3-gene-table-details" data-name="r3c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c3-MIBiG-cluster-compare" data-name="r3c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c3-knownclusterblast" data-name="r3c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c3-subclusterblast" data-name="r3c3-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r3c3-lanthipeptides" data-name="r3c3-lanthipeptides">
Lanthipeptides
</div>





<div class="body-details-header r3c3-tfbs-finder" data-name="r3c3-tfbs-finder">
TFBS Finder
</div>


</div>


<div class="body-details-section r3c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-129"></div> <span class="help-tooltip" id="gt-help-tooltip-help-129">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-122"></div> <span class="help-tooltip" id="MIBiG-help-122">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c3"></div>
<div id="comparison-MIBiG-r3c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">lanthipeptide-class-iii</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496: 0-7345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.48, order:0.64, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>RiPP</td>

<td>AmfS</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563: 185-5330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>RiPP</td>

<td>venezuelin</td>


<td>Streptomyces venezuelae ATCC 10712</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>RiPP</td>

<td>SapB</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001226: 0-6018">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001226/index.html#r1c1" target="_new">BGC0001226</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.48, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>RiPP</td>

<td>streptocollin</td>


<td>Streptomyces collinus Tu 365</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513: 0-6203">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.48, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>RiPP</td>

<td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>


<td>Saccharopolyspora erythraea NRRL 2338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501: 0-7208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.50, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>RiPP</td>

<td>catenulipeptin</td>


<td>Catenulispora acidiphila DSM 44928</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519: 0-6400">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.49, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.27</td>
<td>RiPP</td>

<td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>


<td>Actinomadura namibiensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000528: 0-8183">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000528/index.html#r1c1" target="_new">BGC0000528</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>RiPP</td>

<td>michiganin A</td>


<td>Clavibacter michiganensis subsp. michiganensis NCPPB 382</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002700: 0-5832">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002700/index.html#r1c1" target="_new">BGC0002700</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.42, order:0.43, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>RiPP</td>

<td>bacinapeptin</td>


<td>Bacillus nakamurai</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002657: 479-79914">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002657/index.html#r1c1" target="_new">BGC0002657</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide, NRP</td>

<td>deoxyhangtaimycin</td>


<td>Streptomyces spectabilis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496: 0-7345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.48, order:0.64, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>RiPP</td>
<td>AmfS</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000563: 185-5330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000563/index.html#r1c1" target="_new">BGC0000563</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>RiPP</td>
<td>venezuelin</td>
<td>Streptomyces venezuelae ATCC 10712</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.49, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>RiPP</td>
<td>SapB</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001226: 0-6018">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001226/index.html#r1c1" target="_new">BGC0001226</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.48, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>RiPP</td>
<td>streptocollin</td>
<td>Streptomyces collinus Tu 365</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513: 0-6203">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.48, order:0.57, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>RiPP</td>
<td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>
<td>Saccharopolyspora erythraea NRRL 2338</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501: 0-7208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.50, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>RiPP</td>
<td>catenulipeptin</td>
<td>Catenulispora acidiphila DSM 44928</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519: 0-6400">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.49, order:0.45, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>RiPP</td>
<td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
<td>Actinomadura namibiensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000528: 0-8183">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000528/index.html#r1c1" target="_new">BGC0000528</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>RiPP</td>
<td>michiganin A</td>
<td>Clavibacter michiganensis subsp. michiganensis NCPPB 382</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002700: 0-5832">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002700/index.html#r1c1" target="_new">BGC0002700</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.42, order:0.43, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>RiPP</td>
<td>bacinapeptin</td>
<td>Bacillus nakamurai</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002657: 479-79914">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002657/index.html#r1c1" target="_new">BGC0002657</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.44, order:N/A, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide, NRP</td>
<td>deoxyhangtaimycin</td>
<td>Streptomyces spectabilis</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-123"></div> <span class="help-tooltip" id="cb-known-help-123">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r3c3-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r3c3_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r3c3_1.svg">
catenulipeptin
</option>

<option value = "svg/knownclusterblast_r3c3_2.svg">
SapB
</option>

<option value = "svg/knownclusterblast_r3c3_3.svg">
labyrinthopeptin A2/labyrinthopeptin A1/labyrinthopeptin A3
</option>

<option value = "svg/knownclusterblast_r3c3_4.svg">
SAL-2242
</option>

<option value = "svg/knownclusterblast_r3c3_5.svg">
corynecin III/corynecin I/corynecin II
</option>

<option value = "svg/knownclusterblast_r3c3_6.svg">
duramycin
</option>

<option value = "svg/knownclusterblast_r3c3_7.svg">
cinnamycin
</option>

</select>
<button id="knownclusterblast-r3c3-download">Download graphic</button>
</div>
<div id="knownclusterblast-r3c3-svg"></div>

</div>
</div>

<div class="body-details-section r3c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-124"></div> <span class="help-tooltip" id="cb-sub-help-124">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r3c3-lanthipeptides">
<div class="details">
<div class="heading">
<span>Lanthipeptide predictions</span>
<div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-125"></div> <span class="help-tooltip" id="lanthi-body-help-125">Shows the possible core peptides for each biosynthetic enzyme.<br>Includes CompaRiPPson results for any available databases, with a detailed explanation <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/comparippson' target='_blank'>here</a>.</span></div>
</div>
<div class="details-text ripp-body">


<h4>Lanthipeptide(s) for HS041_24990</h4>
<hr>

<div class="ripp-motif">


<div class="ripp-name">
<span class="serif">HS041_24985</span>
</div>
<div class="ripp-segment-type">Leader:</div>
<div class="ripp-segment-sequence">MALLDLQALESATKGGPD</div>

<div class="ripp-segment-type">Core:</div>
<div class="ripp-segment-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="ripp-segment-type">Core with dehydrations:</div>
<div class="ripp-segment-sequence ripp-dehydrated"><span class="dha">Dha</span><span class="dhb">Dhb</span><span>V</span><span class="dha">Dha</span><span>L</span><span>L</span><span>L</span><span class="cys">Cys</span><span class="dha">Dha</span><span>H</span><span>H</span><span>D</span><span class="dha">Dha</span><span class="dhb">Dhb</span><span>L</span><span class="dha">Dha</span><span>A</span><span>V</span><span class="cys">Cys</span></div>

<div class="comparippson-details">
<h4>antiSMASH-DB 4.0 matches for HS041_24985</h4>
<div class="comparippson-block">
<div class="comparippson-container">
<span class="comparippson-similarity">57.9%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_BMXJ01000002.1&start=92792&end=92918">NZ_BMXJ01000002.1</a><span class="comparippson-extra-indicator">and 3 others</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP099837.1&start=4086089&end=4086215">NZ_CP099837.1</a></span>
<span class="comparippson-description">Lanthipeptides: NE857_RS18075_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_KI911536.1&start=26386&end=26512">NZ_KI911536.1</a></span>
<span class="comparippson-description">Lanthipeptides: B042_RS0117070_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_VWVU01000023.1&start=24758&end=24884">NZ_VWVU01000023.1</a></span>
<span class="comparippson-description">Lanthipeptides: FYX01_RS21045_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: IE195_RS04480_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">16</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span class="dhb near-match-cycle">T</span><span> </span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span>L</span><span class="cys near-match-cycle">C</span><span> </span><span> </span><span> </span><span>D</span><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span>L</span><span class="dha near-match-cycle">S</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>A</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span>-</span><span>-</span><span>G</span><span>D</span><span class="dha">S</span><span class="dha">S</span><span>L</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">15</div>
</div>
</div><div class="comparippson-container">
<span class="comparippson-similarity">55.0%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP077658.1&start=7768222&end=7768351">NZ_CP077658.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JACHJG010000001.1&start=590951&end=591092">NZ_JACHJG010000001.1</a></span>
<span class="comparippson-description">Lanthipeptides: FHS38_RS02665_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: KQY30_RS33045_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span></div>
<div class="comparippson-coordinate">18</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span> </span><span> </span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span>L</span><span class="cys near-match-cycle">C</span><span> </span><span> </span><span> </span><span>D</span><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span>L</span><span class="dha near-match-cycle">S</span><span> </span><span>V</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>R</span><span>A</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span>G</span><span>-</span><span>-</span><span>D</span><span class="dha">S</span><span class="dha">S</span><span>L</span><span class="dha">S</span><span>V</span><span>V</span></div>
<div class="comparippson-coordinate">17</div>
</div>
</div><div class="comparippson-container">
<span class="comparippson-similarity">50.0%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBWU010000019.1&start=75650&end=75779">NZ_JABBWU010000019.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: HHX30_RS29855_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>-</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span> </span><span>V</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span>H</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span>L</span><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>N</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>G</span><span class="cys">C</span><span class="dhb">T</span><span>P</span><span>H</span><span>E</span><span>N</span><span class="dha">S</span><span class="dha">S</span><span>L</span><span class="dha">S</span><span>I</span><span>I</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
</div>
</div>
</div><div class="comparippson-block comparippson-extension">
with 23 more sequence matches <div class="collapser collapser-target-comparippson-worse-hits collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-container">
<span class="comparippson-similarity">50.0%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBWV010000002.1&start=113906&end=114035">NZ_JABBWV010000002.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: HHX31_RS05740_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>-</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span>H</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span>L</span><span class="dha near-match-cycle">S</span><span> </span><span>V</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>N</span><span>L</span><span class="dha">S</span><span>L</span><span>L</span><span>G</span><span class="cys">C</span><span class="dhb">T</span><span>P</span><span>H</span><span>E</span><span>N</span><span class="dha">S</span><span class="dha">S</span><span>L</span><span class="dha">S</span><span>L</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">50.0%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBWX010000013.1&start=197732&end=197861">NZ_JABBWX010000013.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: HHX35_RS30140_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>-</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span>H</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span>L</span><span class="dha near-match-cycle">S</span><span> </span><span>V</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>N</span><span>L</span><span class="dha">S</span><span>L</span><span>L</span><span>G</span><span class="cys">C</span><span class="dhb">T</span><span>P</span><span>H</span><span>E</span><span>N</span><span class="dha">S</span><span class="dha">S</span><span>L</span><span class="dha">S</span><span>I</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">47.4%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_FNVO01000002.1&start=121232&end=121349">NZ_FNVO01000002.1</a><span class="comparippson-extra-indicator">and 2 others</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_FNVO01000002.1&start=121232&end=121349">NZ_FNVO01000002.1</a></span>
<span class="comparippson-description">Lanthipeptides: C4U03_RS04695_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_FNVO01000002.1&start=121232&end=121349">NZ_FNVO01000002.1</a></span>
<span class="comparippson-description">Lanthipeptides: C4U03_RS04695_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: C4U03_RS04695_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="near-match-cycle">+</span><span>V</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span>L</span><span class="cys near-match-cycle">C</span><span> </span><span> </span><span>H</span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span class="near-match-cycle">+</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span>D</span><span>K</span><span>H</span><span class="dha">S</span><span class="dha">S</span><span>H</span><span class="dha">S</span><span class="dhb">T</span><span>I</span><span>L</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">18</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">47.4%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBWY010000006.1&start=416038&end=416164">NZ_JABBWY010000006.1</a><span class="comparippson-extra-indicator">and 3 others</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBWZ010000004.1&start=581961&end=582087">NZ_JABBWZ010000004.1</a></span>
<span class="comparippson-description">Lanthipeptides: HHX40_RS12960_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_VANP01000005.1&start=473289&end=473415">NZ_VANP01000005.1</a></span>
<span class="comparippson-description">Lanthipeptides: FED44_RS16250_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_VYTZ01000006.1&start=123724&end=123850">NZ_VYTZ01000006.1</a></span>
<span class="comparippson-description">Lanthipeptides: F5972_RS17725_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: HHX37_RS18530_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">3</div>
<div class="comparippson-sequence"><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>-</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span>+</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span>H</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span>L</span><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>L</span><span class="dha">S</span><span>L</span><span>L</span><span>A</span><span class="cys">C</span><span class="dhb">T</span><span>P</span><span>H</span><span>E</span><span>N</span><span class="dha">S</span><span class="dha">S</span><span>L</span><span class="dha">S</span><span>L</span><span>I</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">18</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">47.4%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_MTBP01000002.1&start=2091249&end=2091378">NZ_MTBP01000002.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: BTM25_RS21150_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span></div>
<div class="comparippson-coordinate">15</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span> </span><span>V</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span>L</span><span class="cys near-match-cycle">C</span><span> </span><span> </span><span>+</span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span>L</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>Y</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span>F</span><span class="dha">S</span><span>Y</span><span>A</span><span class="dha">S</span><span>V</span><span>L</span></div>
<div class="comparippson-coordinate">16</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">45.5%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_BMMN01000011.1&start=118150&end=118291">NZ_BMMN01000011.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_FTNI01000016.1&start=189071&end=189203">NZ_FTNI01000016.1</a></span>
<span class="comparippson-description">Lanthipeptides: BXA11_RS42315_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: IEY80_RS26005_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>-</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span>H</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span>L</span><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>N</span><span>L</span><span class="dha">S</span><span>L</span><span>L</span><span>A</span><span class="cys">C</span><span class="dhb">T</span><span>P</span><span>H</span><span>E</span><span>N</span><span class="dha">S</span><span class="dha">S</span><span>L</span><span class="dha">S</span><span>L</span><span>I</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">21</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">45.5%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_BMMN01000011.1&start=118350&end=118482">NZ_BMMN01000011.1</a><span class="comparippson-extra-indicator">and 6 others</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_FTNI01000016.1&start=188870&end=189002">NZ_FTNI01000016.1</a></span>
<span class="comparippson-description">Lanthipeptides: BXA11_RS42310_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBWY010000006.1&start=416233&end=416365">NZ_JABBWY010000006.1</a></span>
<span class="comparippson-description">Lanthipeptides: HHX37_RS18535_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBWZ010000004.1&start=582156&end=582288">NZ_JABBWZ010000004.1</a></span>
<span class="comparippson-description">Lanthipeptides: HHX40_RS12965_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_VANP01000005.1&start=473497&end=473629">NZ_VANP01000005.1</a></span>
<span class="comparippson-description">Lanthipeptides: FED44_RS16255_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_VYTZ01000006.1&start=123526&end=123652">NZ_VYTZ01000006.1</a></span>
<span class="comparippson-description">Lanthipeptides: F5972_RS17720_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_VYTZ01000006.1&start=123273&end=123402">NZ_VYTZ01000006.1</a></span>
<span class="comparippson-description">Lanthipeptides: F5972_RS17715_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: IEY80_RS26010_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>-</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span>H</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span>L</span><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>N</span><span>L</span><span class="dha">S</span><span>L</span><span>L</span><span>G</span><span class="cys">C</span><span class="dhb">T</span><span>P</span><span>H</span><span>E</span><span>N</span><span class="dha">S</span><span class="dha">S</span><span>L</span><span class="dha">S</span><span>I</span><span>I</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">21</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">45.0%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_BBQG01000022.1&start=94184&end=94307">NZ_BBQG01000022.1</a><span class="comparippson-extra-indicator">and 5 others</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP048875.1&start=7347918&end=7348041">NZ_CP048875.1</a></span>
<span class="comparippson-description">Lanthipeptides: G3260_RS30690_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP050693.1&start=865873&end=865996">NZ_CP050693.1</a></span>
<span class="comparippson-description">Lanthipeptides: HC362_RS03880_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP103060.1&start=743296&end=743419">NZ_CP103060.1</a></span>
<span class="comparippson-description">Lanthipeptides: NR995_RS03375_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_KE150472.1&start=2145219&end=2145342">NZ_KE150472.1</a></span>
<span class="comparippson-description">Lanthipeptides: HMPREF1486_RS35560_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_VJOK01000001.1&start=6674732&end=6674855">NZ_VJOK01000001.1</a></span>
<span class="comparippson-description">Lanthipeptides: FM076_RS28910_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: TU22_RS34220_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span> </span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span>L</span><span class="cys near-match-cycle">C</span><span> </span><span> </span><span>H</span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span class="near-match-cycle">+</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dha">S</span><span>A</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span>D</span><span>K</span><span>H</span><span class="dha">S</span><span class="dha">S</span><span>H</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span>L</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">45.0%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBXA010000003.1&start=91878&end=92007">NZ_JABBXA010000003.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: HHX41_RS08065_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>-</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span>H</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span>L</span><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>N</span><span>L</span><span class="dha">S</span><span>L</span><span>L</span><span>G</span><span class="cys">C</span><span class="dhb">T</span><span>P</span><span>H</span><span>E</span><span>N</span><span class="dha">S</span><span class="dha">S</span><span>L</span><span class="dha">S</span><span>L</span><span>I</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">43.5%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NC_016582.1&start=9014835&end=9014970">NC_016582.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_FOLM01000007.1&start=131174&end=131300">NZ_FOLM01000007.1</a></span>
<span class="comparippson-description">Lanthipeptides: BM242_RS14445_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: SBI_RS53340_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span>V</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span>L</span><span class="cys near-match-cycle">C</span><span> </span><span> </span><span>H</span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span>+</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">4</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dha">S</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span>D</span><span>K</span><span>H</span><span class="dha">S</span><span class="dha">S</span><span>H</span><span class="dha">S</span><span>N</span><span>L</span><span>L</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">22</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">43.5%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_BCBX01000043.1&start=72448&end=72583">NZ_BCBX01000043.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_BCBX01000043.1&start=72239&end=72371">NZ_BCBX01000043.1</a></span>
<span class="comparippson-description">Lanthipeptides: AP234_RS36580_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: AP234_RS36585_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span>-</span><span class="dha">S</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span> </span><span>H</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span>L</span><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">3</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>N</span><span>L</span><span class="dha">S</span><span>L</span><span>L</span><span>A</span><span class="cys">C</span><span>V</span><span>P</span><span>H</span><span>E</span><span>N</span><span class="dha">S</span><span class="dha">S</span><span>L</span><span class="dha">S</span><span>L</span><span>L</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">22</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">43.5%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_KI866522.1&start=118376&end=118502">NZ_KI866522.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: MPTA5024_RS40525_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>-</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span>H</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span>L</span><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>N</span><span>L</span><span class="dha">S</span><span>L</span><span>L</span><span>G</span><span class="cys">C</span><span class="dhb">T</span><span>P</span><span>H</span><span>E</span><span>N</span><span class="dha">S</span><span class="dha">S</span><span>L</span><span class="dha">S</span><span>V</span><span>L</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">21</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">43.5%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_QOIL01000022.1&start=27807&end=27933">NZ_QOIL01000022.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_QOIL01000022.1&start=27615&end=27741">NZ_QOIL01000022.1</a></span>
<span class="comparippson-description">Lanthipeptides: DQ384_RS31185_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: DQ384_RS31190_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>-</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span>H</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span>L</span><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>N</span><span>L</span><span class="dha">S</span><span>L</span><span>L</span><span>G</span><span class="cys">C</span><span class="dhb">T</span><span>P</span><span>H</span><span>E</span><span>N</span><span class="dha">S</span><span class="dha">S</span><span>L</span><span class="dha">S</span><span>I</span><span>L</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">21</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">42.9%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_BMVR01000001.1&start=54863&end=54992">NZ_BMVR01000001.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: IE274_RS00225_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span></div>
<div class="comparippson-coordinate">15</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span> </span><span>V</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span>L</span><span class="cys near-match-cycle">C</span><span>+</span><span> </span><span> </span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span>L</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">4</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>Q</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span>A</span><span class="dha">S</span><span>Q</span><span>A</span><span class="dha">S</span><span>F</span><span>L</span></div>
<div class="comparippson-coordinate">18</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">42.1%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP032402.1&start=3694709&end=3694826">NZ_CP032402.1</a><span class="comparippson-extra-indicator">and 1 other</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP032402.1&start=3694709&end=3694826">NZ_CP032402.1</a></span>
<span class="comparippson-description">Lanthipeptides: D3U04_RS17070_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: D3U04_RS17070_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="near-match-cycle">+</span><span> </span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span>L</span><span class="cys near-match-cycle">C</span><span> </span><span> </span><span>H</span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span class="near-match-cycle">+</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>A</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span>D</span><span>K</span><span>H</span><span class="dha">S</span><span class="dha">S</span><span>Q</span><span class="dha">S</span><span class="dhb">T</span><span>V</span><span>L</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">18</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">41.7%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBWU010000019.1&start=75847&end=75973">NZ_JABBWU010000019.1</a><span class="comparippson-extra-indicator">and 5 others</span>:<div class="collapser collapser-target-comparippson-extra-names collapser-level-none"> </div><div class="collapser-content">
<div class="comparippson-extra-names"><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBWU010000019.1&start=75454&end=75580">NZ_JABBWU010000019.1</a></span>
<span class="comparippson-description">Lanthipeptides: HHX30_RS29850_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBWV010000002.1&start=113708&end=113840">NZ_JABBWV010000002.1</a></span>
<span class="comparippson-description">Lanthipeptides: HHX31_RS05735_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBWX010000013.1&start=197931&end=198063">NZ_JABBWX010000013.1</a></span>
<span class="comparippson-description">Lanthipeptides: HHX35_RS30145_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBXA010000003.1&start=91677&end=91809">NZ_JABBXA010000003.1</a></span>
<span class="comparippson-description">Lanthipeptides: HHX41_RS08060_lanthipeptide</span><span><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JABBXA010000003.1&start=91477&end=91609">NZ_JABBXA010000003.1</a></span>
<span class="comparippson-description">Lanthipeptides: HHX41_RS08055_lanthipeptide</span>
</div>
</div></span>
<span class="comparippson-description">Lanthipeptides: HHX30_RS29860_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>-</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span> </span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span> </span><span>H</span><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span>L</span><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">4</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>N</span><span>L</span><span class="dha">S</span><span>L</span><span>L</span><span>G</span><span class="cys">C</span><span class="dhb">T</span><span>P</span><span>H</span><span>E</span><span>N</span><span class="dha">S</span><span class="dha">S</span><span>L</span><span class="dha">S</span><span>L</span><span>I</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">23</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">36.8%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP074380.1&start=5975549&end=5975663">NZ_CP074380.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: KGS77_RS26180_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span></div>
<div class="comparippson-coordinate">14</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span> </span><span>+</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span>L</span><span class="cys near-match-cycle">C</span><span>+</span><span> </span><span>H</span><span> </span><span class="dha near-match-cycle">S</span><span> </span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>V</span><span>I</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span>N</span><span class="dha">S</span><span>H</span><span>A</span><span class="dha">S</span><span>V</span></div>
<div class="comparippson-coordinate">13</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">36.8%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_JAEVFI010000019.1&start=42528&end=42651">NZ_JAEVFI010000019.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: JJV70_RS15420_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span> </span><span>V</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span>+</span><span class="cys near-match-cycle">C</span><span> </span><span> </span><span>H</span><span> </span><span> </span><span> </span><span> </span><span class="near-match-cycle">+</span><span> </span><span>+</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span>N</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>I</span><span class="cys">C</span><span>D</span><span>K</span><span>H</span><span class="dha">S</span><span>G</span><span>A</span><span class="dha">S</span><span class="dhb">T</span><span>I</span><span>L</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">18</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">33.3%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP042266.1&start=278193&end=278322">NZ_CP042266.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: FQU76_RS00910_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">13</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span>+</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span> </span><span class="cys near-match-cycle">C</span><span> </span><span>H</span><span>H</span><span> </span><span>+</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">3</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dha">S</span><span>L</span><span class="dha">S</span><span>L</span><span>L</span><span class="dha">S</span><span class="cys">C</span><span>G</span><span>H</span><span>H</span><span class="dha">S</span><span>N</span></div>
<div class="comparippson-coordinate">15</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">33.3%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_CP045702.1&start=7230790&end=7230922">NZ_CP045702.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: F0344_RS33200_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span></div>
<div class="comparippson-coordinate">10</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span> </span><span>V</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span>+</span><span class="cys near-match-cycle">C</span><span> </span><span>H</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>Q</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>V</span><span class="cys">C</span><span>E</span><span>H</span></div>
<div class="comparippson-coordinate">11</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">33.3%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_LWLE01000019.1&start=13267&end=13399">NZ_LWLE01000019.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: A6A29_RS44250_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span></div>
<div class="comparippson-coordinate">10</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span> </span><span>V</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span>+</span><span class="cys near-match-cycle">C</span><span> </span><span>H</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>Q</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>I</span><span class="cys">C</span><span>E</span><span>H</span></div>
<div class="comparippson-coordinate">11</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">24.1%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_FNDJ01000037.1&start=8761&end=8989">NZ_FNDJ01000037.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: allorf_008762_008989_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">13</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span> </span><span> </span><span> </span><span>L</span><span> </span><span>L</span><span class="cys near-match-cycle">C</span><span> </span><span>H</span><span> </span><span>D</span><span class="dha near-match-cycle">S</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">8</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>R</span><span>G</span><span>R</span><span>L</span><span>F</span><span>L</span><span class="cys">C</span><span>G</span><span>H</span><span class="dhb">T</span><span>D</span><span class="dha">S</span></div>
<div class="comparippson-coordinate">20</div>
</div>
</div>
<div class="comparippson-container">
<span class="comparippson-similarity">20.0%</span>
<span class="comparippson-link"><a href="https://antismash-db.secondarymetabolites.org/area.html?record=NZ_FPIZ01000007.1&start=21385&end=21547">NZ_FPIZ01000007.1</a>:</span>
<span class="comparippson-description">Lanthipeptides: BT118_RS34215_lanthipeptide</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">2</div>
<div class="comparippson-sequence"><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dhb near-match-cycle">T</span><span>V</span><span class="dha near-match-cycle">S</span><span> </span><span> </span><span>+</span><span class="cys near-match-cycle">C</span><span class="near-match-cycle">+</span><span>+</span><span>+</span><span> </span><span class="dha near-match-cycle">S</span><span> </span><span>+</span><span> </span><span>+</span><span>+</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">6</div>
<div class="comparippson-sequence"><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>F</span><span>A</span><span>V</span><span class="cys">C</span><span class="dhb">T</span><span>N</span><span>N</span><span>A</span><span class="dha">S</span><span>Q</span><span>V</span><span>L</span><span class="dha">S</span><span>I</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">23</div>
</div>
</div>
</div>
</div>
</div>
<div class="comparippson-details">
<h4>MIBiG 3.1 matches for HS041_24985</h4>
<div class="comparippson-block">
<div class="comparippson-container">
<span class="comparippson-similarity">45.5%</span>
<span class="comparippson-link"><a href="https://mibig.secondarymetabolites.org/go/BGC0000553">BGC0000553 (SSGG_RS09925)</a>:</span>
<span class="comparippson-description">Lanthipeptide: SRO15-2212</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">1</div>
<div class="comparippson-sequence"><span class="dha">S</span><span class="dhb">T</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>L</span><span class="cys">C</span><span class="dha">S</span><span>H</span><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span></div>
<div class="comparippson-coordinate">18</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span class="dha near-match-cycle">S</span><span> </span><span>V</span><span class="dha near-match-cycle">S</span><span>L</span><span>L</span><span>+</span><span class="cys near-match-cycle">C</span><span> </span><span>+</span><span> </span><span> </span><span class="dha near-match-cycle">S</span><span class="near-match-cycle">+</span><span>L</span><span class="dha near-match-cycle">S</span><span> </span><span>V</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">3</div>
<div class="comparippson-sequence"><span class="dha">S</span><span>Q</span><span>V</span><span class="dha">S</span><span>L</span><span>L</span><span>V</span><span class="cys">C</span><span>E</span><span>Y</span><span>-</span><span>-</span><span class="dha">S</span><span class="dha">S</span><span>L</span><span class="dha">S</span><span>V</span><span>V</span></div>
<div class="comparippson-coordinate">18</div>
</div>
</div><div class="comparippson-container">
<span class="comparippson-similarity">18.2%</span>
<span class="comparippson-link"><a href="https://mibig.secondarymetabolites.org/go/BGC0000582">BGC0000582 (ADR72962.1)</a>:</span>
<span class="comparippson-description">Linaridin: cypemycin</span>
<div class="comparippson-alignment">
<div class="comparippson-coordinate">11</div>
<div class="comparippson-sequence"><span>H</span><span>D</span><span class="dha">S</span><span class="dhb">T</span><span>L</span><span class="dha">S</span><span>A</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">19</div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-midline"><span> </span><span> </span><span class="dha near-match-cycle">S</span><span class="dhb near-match-cycle">T</span><span>+</span><span class="near-match-cycle"> </span><span> </span><span>V</span><span class="cys near-match-cycle">C</span></div>
<div class="comparippson-coordinate"></div>
<div class="comparippson-coordinate">14</div>
<div class="comparippson-sequence"><span>Q</span><span>G</span><span class="dha">S</span><span class="dhb">T</span><span>I</span><span class="cys">C</span><span>L</span><span>V</span><span class="cys">C</span></div>
<div class="comparippson-coordinate">22</div>
</div>
</div>
</div>
</div>
</div>


<div class="ripp-legend">
<h4> Legend: </h4>
<div>
<span class="dha">Dha</span>: Didehydroalanine<br>
<span class="dhb">Dhb</span>: Didehydrobutyrine<br>
</div>
</div>

</div>
</div>
</div>





<div class="body-details-section r3c3-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-128"></div> <span class="help-tooltip" id="tfbs-body-help-128">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.16 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r3c3-751536-AbrC3"></div>

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.16 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r3c3-767390-AbrC3"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 20.82 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r3c3-760435-LexA"></div>

</div>
</div>
</div>

</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r3c3-lanthipeptides" data-name="r3c3-lanthipeptides">
Lanthipeptides
</div>





<div class="sidepanel-details-header r3c3-tfbs-finder" data-name="r3c3-tfbs-finder">
TFBS Finder
</div>


</div>








<div class="sidepanel-details-section r3c3-lanthipeptides">
<div class="more-details">
<div class="heading">
<span>Core peptide predictions</span>
<div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-126"></div> <span class="help-tooltip" id="lanthi-side-help-126">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
</div>

<dl class ="prediction-text">
<dt><span class="serif">HS041_24985</span> - Class III</dt>
<dd>Cleavage pHMM score: 7.50</dd>
<dd>RODEO score: 0</dd>
<dd>Monoisotopic mass: 1845.9 Da</dd>
<dd>Molecular weight: 1847.1 Da</dd>
<dd>Number of bridges: 2</dd>
<dd>
<dl>
<dt>Alternative weights</dt>
<dd class="alt-weight-desc">(assuming N unmodified Ser/Thr residues)</dd>

<dd>
<span class="alt-weight"> 1865.1 Da</span>
<span class="alt-weight-n"> N = 1</span>
</dd>

<dd>
<span class="alt-weight"> 1883.1 Da</span>
<span class="alt-weight-n"> N = 2</span>
</dd>

<dd>
<span class="alt-weight"> 1901.2 Da</span>
<span class="alt-weight-n"> N = 3</span>
</dd>

<dd>
<span class="alt-weight"> 1919.2 Da</span>
<span class="alt-weight-n"> N = 4</span>
</dd>

<dd>
<span class="alt-weight"> 1937.2 Da</span>
<span class="alt-weight-n"> N = 5</span>
</dd>

</dl>
</dd>

</dl>

</div>

</div>





<div class="sidepanel-details-section r3c3-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-127"></div> <span class="help-tooltip" id="tfbs-side-help-127">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>


</div>

</div>

</div>

<div class="page" id='r3c4' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000003.1 - Region 4 - NAPAA <div class="help-container"> <div class="help-icon" data-id="region_svg-help-264"></div> <span class="help-tooltip" id="region_svg-help-264">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000003.1.region004.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c4-svg" data-filename="JABTEZ010000003.1_r3c4.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 916,608 - 950,510 nt. (total: 33,903 nt)

<a class="cluster-rules-header" id="r3c4-rules-header" href="#r3c4">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c4-rules">NAPAA: (AMP-binding and TIGR02353)</div>

<div class="region-svg-container">
<div id='r3c4-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-265"></div> <span class="help-tooltip" id="focus-panel-help-265">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c4">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c4-gene-table-details" data-name="r3c4-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c4-nrps_pks" data-name="r3c4-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r3c4-MIBiG-cluster-compare" data-name="r3c4-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c4-knownclusterblast" data-name="r3c4-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c4-subclusterblast" data-name="r3c4-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r3c4-tfbs-finder" data-name="r3c4-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r3c4-nrps-pks-bubbles" data-name="r3c4-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r3c4-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-139"></div> <span class="help-tooltip" id="gt-help-tooltip-help-139">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c4">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c4-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c4-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c4">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c4-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-130"></div> <span class="help-tooltip" id="nrps-domain-help-130">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c4-details-svg" data-filename="JABTEZ010000003.1_r3c4_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r3c4-details-svg"></div>
</div>
</div>



<div class="body-details-section r3c4-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-131"></div> <span class="help-tooltip" id="MIBiG-help-131">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c4-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c4-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c4"></div>
<div id="comparison-MIBiG-r3c4-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NAPAA</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002174: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002174/index.html#r1c1" target="_new">BGC0002174</a></td>





<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>


<td>0.38</td>
<td>NRP</td>

<td>ε-Poly-L-lysine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007: 584-20653">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.55, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>Terpene</td>

<td>atolypene A, atolypene B</td>


<td>Amycolatopsis tolypomycina</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002535: 1708-15164">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002535/index.html#r1c1" target="_new">BGC0002535</a></td>





<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.30</td>
<td>NRP</td>

<td>γ-poly-L-2,4-diaminobutyric acid</td>


<td>Streptomyces celluloflavus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000844: 2-13901">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000844/index.html#r1c1" target="_new">BGC0000844</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>


<td>0.29</td>
<td>Other</td>

<td>clavulanic acid</td>


<td>Streptomyces clavuligerus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001463: 0-6707">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001463/index.html#r1c1" target="_new">BGC0001463</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>Other</td>

<td>2,3,4,5-tetrabromopyrrole</td>


<td>Pseudoalteromonas piscicida</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>RiPP</td>

<td>SCO-2138</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>RiPP</td>

<td>SLI-2138</td>


<td>Streptomyces lividans TK24</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001464: 0-6705">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001464/index.html#r1c1" target="_new">BGC0001464</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>tetrabromopyrrole</td>


<td>Pseudoalteromonas sp. PS5</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001895: 425-18341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001895/index.html#r1c1" target="_new">BGC0001895</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Polyketide</td>

<td>huanglongmycin A, huanglongmycin B, huanglongmycin C</td>


<td>Streptomyces sp. CB09001</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c4-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002174: 0-3894">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002174/index.html#r1c1" target="_new">BGC0002174</a></td>


<td class="cc-heat-cell" title="0.71 (id:0.50, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.71), rgb(0, 0, 0, 0.71) 71%, white 71%)"></td>
<td>0.71</td>
<td>NRP</td>
<td>ε-Poly-L-lysine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002007: 584-20653">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002007/index.html#r1c1" target="_new">BGC0002007</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.55, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>Terpene</td>
<td>atolypene A, atolypene B</td>
<td>Amycolatopsis tolypomycina</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002535: 1708-15164">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002535/index.html#r1c1" target="_new">BGC0002535</a></td>


<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>
<td>0.64</td>
<td>NRP</td>
<td>γ-poly-L-2,4-diaminobutyric acid</td>
<td>Streptomyces celluloflavus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000844: 2-13901">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000844/index.html#r1c1" target="_new">BGC0000844</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.45, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Other</td>
<td>clavulanic acid</td>
<td>Streptomyces clavuligerus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001463: 0-6707">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001463/index.html#r1c1" target="_new">BGC0001463</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>2,3,4,5-tetrabromopyrrole</td>
<td>Pseudoalteromonas piscicida</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>RiPP</td>
<td>SCO-2138</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>RiPP</td>
<td>SLI-2138</td>
<td>Streptomyces lividans TK24</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001464: 0-6705">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001464/index.html#r1c1" target="_new">BGC0001464</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>tetrabromopyrrole</td>
<td>Pseudoalteromonas sp. PS5</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001895: 425-18341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001895/index.html#r1c1" target="_new">BGC0001895</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.74)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Polyketide</td>
<td>huanglongmycin A, huanglongmycin B, huanglongmycin C</td>
<td>Streptomyces sp. CB09001</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c4-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-132"></div> <span class="help-tooltip" id="cb-known-help-132">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r3c4-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r3c4_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r3c4_1.svg">
ε-Poly-L-lysine
</option>

<option value = "svg/knownclusterblast_r3c4_2.svg">
monensin
</option>

<option value = "svg/knownclusterblast_r3c4_3.svg">
desertomycin B/desertomycin A/desertomycin G
</option>

</select>
<button id="knownclusterblast-r3c4-download">Download graphic</button>
</div>
<div id="knownclusterblast-r3c4-svg"></div>

</div>
</div>

<div class="body-details-section r3c4-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-133"></div> <span class="help-tooltip" id="cb-sub-help-133">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r3c4-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-137"></div> <span class="help-tooltip" id="tfbs-body-help-137">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AbrC3
<div class="collapser collapser-target-AbrC3 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Antibiotic production activator<br>
<span>Score</span>: 18.16 out of 18.16
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r3c4-950234-AbrC3"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 17.52 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r3c4-919620-ZuR"></div>

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 19.92 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r3c4-949770-LexA"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r3c4-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-138"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-138">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r3c4-domain-bubble-svg" data-filename="JABTEZ010000003.1_r3c4_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r3c4-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC4">
Candidate 4 (916607 - 950510): single NAPAA
</option>


</select>

<div class="details-svg" id="r3c4-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r3c4-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r3c4-nrps_pks_products" data-name="r3c4-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r3c4-nrps_pks_monomers" data-name="r3c4-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r3c4-tfbs-finder" data-name="r3c4-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r3c4-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-134"></div> <span class="help-tooltip" id="nrps-structures-help-134">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 4, location 916607 - 950510:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r3c4-4" class="smiles-canvas" data-smiles="NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r3c4-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r3c4-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-135"></div> <span class="help-tooltip" id="nrps-monomers-help-135">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">HS041_25760</span></strong>: X
<div class="collapser collapser-target-HS041_25760 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (32..430)</strong>: X
<div class="collapser collapser-target-HS041_25760 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-HS041_25760-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAESIGTVVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Lys <span class="serif">DAEDIGTVVK</span> (71% 8Å match)</dd>
<dd>Gln or Tyr <span class="serif">DAESVGTVVK</span> (65% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r3c4-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-136"></div> <span class="help-tooltip" id="tfbs-side-help-136">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AbrC3" target="_blank">AbrC3</a></td>
<td>Antibiotic production activator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r3c5' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000003.1 - Region 5 - lanthipeptide-class-iv,terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-266"></div> <span class="help-tooltip" id="region_svg-help-266">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000003.1.region005.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r3c5-svg" data-filename="JABTEZ010000003.1_r3c5.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 972,257 - 1,007,606 nt. (total: 35,350 nt)

<a class="cluster-rules-header" id="r3c5-rules-header" href="#r3c5">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r3c5-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)<br>lanthipeptide-class-iv: (cds(LANC_like and Pkinase) and not micKC)</div>

<div class="region-svg-container">
<div id='r3c5-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-267"></div> <span class="help-tooltip" id="focus-panel-help-267">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r3c5">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r3c5-gene-table-details" data-name="r3c5-gene-table-details">
Gene overview
</div>



<div class="body-details-header r3c5-MIBiG-cluster-compare" data-name="r3c5-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r3c5-knownclusterblast" data-name="r3c5-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r3c5-subclusterblast" data-name="r3c5-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r3c5-lanthipeptides" data-name="r3c5-lanthipeptides">
Lanthipeptides
</div>





<div class="body-details-header r3c5-tfbs-finder" data-name="r3c5-tfbs-finder">
TFBS Finder
</div>


</div>


<div class="body-details-section r3c5-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-147"></div> <span class="help-tooltip" id="gt-help-tooltip-help-147">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r3c5">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r3c5-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r3c5-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r3c5">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r3c5-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-140"></div> <span class="help-tooltip" id="MIBiG-help-140">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r3c5-selector">Analysis type:</label>
<select id="comparison-MIBiG-r3c5-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r3c5"></div>
<div id="comparison-MIBiG-r3c5-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>

<th><span class="cc-heat-product">lanthipeptide-class-iv</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.48, order:0.57, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.48, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.47</td>
<td>RiPP</td>

<td>SapB</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519: 0-6400">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.50, order:0.51, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.50, order:0.51, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.44</td>
<td>RiPP</td>

<td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>


<td>Actinomadura namibiensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501: 0-7208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501</a></td>





<td class="cc-heat-cell" title="0.50 (id:0.48, order:0.51, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.50), rgb(0, 0, 0, 0.50) 50%, white 50%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.48, order:0.51, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.44</td>
<td>RiPP</td>

<td>catenulipeptin</td>


<td>Catenulispora acidiphila DSM 44928</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513: 0-6203">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.48, order:0.57, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.48, order:0.57, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.43</td>
<td>RiPP</td>

<td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>


<td>Saccharopolyspora erythraea NRRL 2338</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496: 0-7345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.47, order:0.51, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.47, order:0.51, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.43</td>
<td>RiPP</td>

<td>AmfS</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000553: 66-7440">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000553/index.html#r1c1" target="_new">BGC0000553</a></td>





<td class="cc-heat-cell" title="0.48 (id:0.46, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.48), rgb(0, 0, 0, 0.48) 48%, white 48%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.42</td>
<td>RiPP</td>

<td>SRO15-2212</td>


<td>Streptomyces filamentosus NRRL 15998</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002260: 942-16838">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002260/index.html#r1c1" target="_new">BGC0002260</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.42, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>






<td class="cc-heat-cell" title="0.46 (id:0.42, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>


<td>0.39</td>
<td>Terpene</td>

<td>trichobrasilenol, xylarenic acid B, brasilane A, brasilane F, brasilane E, brasilane D</td>


<td>Annulohypoxylon truncatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000554: 0-6546">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000554/index.html#r1c1" target="_new">BGC0000554</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.43, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.36</td>
<td>RiPP</td>

<td>SRO15-3108</td>


<td>Streptomyces filamentosus NRRL 15998</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001407: 0-10802">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001407/index.html#r1c1" target="_new">BGC0001407</a></td>





<td class="cc-heat-cell" title="0.47 (id:0.45, order:N/A, components:0.50)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.47), rgb(0, 0, 0, 0.47) 47%, white 47%)"></td>






<td class="cc-heat-cell" title="0.52 (id:0.45, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.36</td>
<td>RiPP</td>

<td>bicereucin</td>


<td>Bacillus cereus SJ1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001226: 0-6018">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001226/index.html#r1c1" target="_new">BGC0001226</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>RiPP</td>

<td>streptocollin</td>


<td>Streptomyces collinus Tu 365</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r3c5-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001226: 0-6018">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001226/index.html#r1c1" target="_new">BGC0001226</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>RiPP</td>
<td>streptocollin</td>
<td>Streptomyces collinus Tu 365</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000551: 0-6728">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000551/index.html#r1c1" target="_new">BGC0000551</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.48, order:0.57, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>RiPP</td>
<td>SapB</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000519: 0-6400">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000519/index.html#r1c1" target="_new">BGC0000519</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.50, order:0.51, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>RiPP</td>
<td>labyrinthopeptin A2, labyrinthopeptin A1, labyrinthopeptin A3</td>
<td>Actinomadura namibiensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000501: 0-7208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000501/index.html#r1c1" target="_new">BGC0000501</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.48, order:0.51, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>RiPP</td>
<td>catenulipeptin</td>
<td>Catenulispora acidiphila DSM 44928</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002700: 0-5832">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002700/index.html#r1c1" target="_new">BGC0002700</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>RiPP</td>
<td>bacinapeptin</td>
<td>Bacillus nakamurai</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002111: 0-6681">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002111/index.html#r1c1" target="_new">BGC0002111</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>RiPP</td>
<td>andalusicin A, andalusicin B</td>
<td>Bacillus thuringiensis serovar andalousiensis NRRL B-23139</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000496: 0-7345">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000496/index.html#r1c1" target="_new">BGC0000496</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.47, order:0.51, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>RiPP</td>
<td>AmfS</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000553: 66-7440">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000553/index.html#r1c1" target="_new">BGC0000553</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>RiPP</td>
<td>SRO15-2212</td>
<td>Streptomyces filamentosus NRRL 15998</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001911: 591-10299">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001911/index.html#r1c1" target="_new">BGC0001911</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Terpene</td>
<td>raimonol</td>
<td>Streptomyces anulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000513: 0-6203">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000513/index.html#r1c1" target="_new">BGC0000513</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.48, order:0.57, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>RiPP</td>
<td>Ery-9, Ery-6, Ery-8, Ery-7, Ery-5, Ery-4, Ery-3</td>
<td>Saccharopolyspora erythraea NRRL 2338</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r3c5-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-141"></div> <span class="help-tooltip" id="cb-known-help-141">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r3c5-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r3c5_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r3c5_1.svg">
labyrinthopeptin A2/labyrinthopeptin A1/labyrinthopeptin A3
</option>

</select>
<button id="knownclusterblast-r3c5-download">Download graphic</button>
</div>
<div id="knownclusterblast-r3c5-svg"></div>

</div>
</div>

<div class="body-details-section r3c5-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-142"></div> <span class="help-tooltip" id="cb-sub-help-142">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r3c5-lanthipeptides">
<div class="details">
<div class="heading">
<span>Lanthipeptide predictions</span>
<div class="help-container"> <div class="help-icon" data-id="lanthi-body-help-143"></div> <span class="help-tooltip" id="lanthi-body-help-143">Shows the possible core peptides for each biosynthetic enzyme.<br>Includes CompaRiPPson results for any available databases, with a detailed explanation <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/comparippson' target='_blank'>here</a>.</span></div>
</div>
<div class="details-text ripp-body">

Lanthipeptides - No core peptides found.

</div>
</div>
</div>





<div class="body-details-section r3c5-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-146"></div> <span class="help-tooltip" id="tfbs-body-help-146">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 17.61 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r3c5-973783-ZuR"></div>

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 22.37 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r3c5-997786-NuR"></div>

</div>
</div>
</div>

</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r3c5-lanthipeptides" data-name="r3c5-lanthipeptides">
Lanthipeptides
</div>





<div class="sidepanel-details-header r3c5-tfbs-finder" data-name="r3c5-tfbs-finder">
TFBS Finder
</div>


</div>








<div class="sidepanel-details-section r3c5-lanthipeptides">
<div class="more-details">
<div class="heading">
<span>Core peptide predictions</span>
<div class="help-container"> <div class="help-icon" data-id="lanthi-side-help-144"></div> <span class="help-tooltip" id="lanthi-side-help-144">Lists the possible core peptides in the region. Each core peptide lists the number of lanthionine bridges, possible molecular weights, and the scores for cleavage site prediction and RODEO.</span></div>
</div>

<div class="prediction-text">No core peptides found.</div>

</div>

</div>





<div class="sidepanel-details-section r3c5-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-145"></div> <span class="help-tooltip" id="tfbs-side-help-145">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>


</div>

</div>

</div>





<div class="page" id='r4c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000004.1 - Region 1 - NRPS,NRPS-like <div class="help-container"> <div class="help-icon" data-id="region_svg-help-268"></div> <span class="help-tooltip" id="region_svg-help-268">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000004.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c1-svg" data-filename="JABTEZ010000004.1_r4c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 86,408 - 134,411 nt. (total: 48,004 nt)

<a class="cluster-rules-header" id="r4c1-rules-header" href="#r4c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>NRPS-like: cds((PP-binding or NAD_binding_4) and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r4c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-269"></div> <span class="help-tooltip" id="focus-panel-help-269">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c1-gene-table-details" data-name="r4c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c1-nrps_pks" data-name="r4c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c1-MIBiG-cluster-compare" data-name="r4c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c1-knownclusterblast" data-name="r4c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c1-subclusterblast" data-name="r4c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c1-tfbs-finder" data-name="r4c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r4c1-nrps-pks-bubbles" data-name="r4c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r4c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-157"></div> <span class="help-tooltip" id="gt-help-tooltip-help-157">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-148"></div> <span class="help-tooltip" id="nrps-domain-help-148">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-details-svg" data-filename="JABTEZ010000004.1_r4c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-149"></div> <span class="help-tooltip" id="MIBiG-help-149">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c1"></div>
<div id="comparison-MIBiG-r4c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">NRPS-like</span></th>

<th><span class="cc-heat-product">NRPS-like</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002051: 162-12936">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002051/index.html#r1c1" target="_new">BGC0002051</a></td>





<td class="cc-heat-cell" title="0.76 (id:0.53, order:0.92, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>






<td class="cc-heat-cell" title="0.76 (id:0.53, order:0.92, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>






<td class="cc-heat-cell" title="0.76 (id:0.53, order:0.92, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>


<td>1.37</td>
<td>NRP</td>

<td>antipain</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.44, order:1.00, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.44, order:1.00, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.44, order:1.00, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>1.05</td>
<td>NRP</td>

<td>taxlllaid A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>1.04</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.46, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.46, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.46, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>1.02</td>
<td>NRP</td>

<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>


<td>Pseudomonas sp. QS1027</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>1.01</td>
<td>NRP</td>

<td>vicibactin</td>


<td>Rhizobium etli CFN 42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>1.01</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465: 0-10043">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>1.00</td>
<td>NRP</td>

<td>xenortide A, xenortide B, xenortide C, xenortide D</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.98</td>
<td>NRP</td>

<td>sevadicin</td>


<td>Paenibacillus larvae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001434: 0-14218">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001434/index.html#r1c1" target="_new">BGC0001434</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:0.83, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.43, order:0.83, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.43, order:0.83, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.96</td>
<td>NRP</td>

<td>nematophin</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.96</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002051: 162-12936">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002051/index.html#r1c1" target="_new">BGC0002051</a></td>


<td class="cc-heat-cell" title="0.76 (id:0.53, order:0.92, components:0.89)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
<td>0.76</td>
<td>NRP</td>
<td>antipain</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.44, order:1.00, components:0.71)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>taxlllaid A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.46, order:1.00, components:0.66)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>
<td>Pseudomonas sp. QS1027</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>vicibactin</td>
<td>Rhizobium etli CFN 42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465: 0-10043">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>xenortide A, xenortide B, xenortide C, xenortide D</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>sevadicin</td>
<td>Paenibacillus larvae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001434: 0-14218">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001434/index.html#r1c1" target="_new">BGC0001434</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:0.83, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>nematophin</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-150"></div> <span class="help-tooltip" id="cb-known-help-150">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r4c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r4c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r4c1_1.svg">
antipain
</option>

<option value = "svg/knownclusterblast_r4c1_2.svg">
deimino-antipain
</option>

<option value = "svg/knownclusterblast_r4c1_3.svg">
clifednamide D/clifednamide E/clifednamide F/clifednamide G/clifednamide H/clifednamide I/clifednamide J
</option>

</select>
<button id="knownclusterblast-r4c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r4c1-svg"></div>

</div>
</div>

<div class="body-details-section r4c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-151"></div> <span class="help-tooltip" id="cb-sub-help-151">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-155"></div> <span class="help-tooltip" id="tfbs-body-help-155">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (6):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
HypR
<div class="collapser collapser-target-HypR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: L-hydroxyproline utilization repressor<br>
<span>Score</span>: 16.96 out of 22.40
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r4c1-90881-HypR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 18.44 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r4c1-118339-ZuR"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 19.24 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r4c1-121251-NrtR"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 19.77 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r4c1-122909-NrtR"></div>

<div class="tfbs-hit-detail">
ArgR
<div class="collapser collapser-target-ArgR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Regulator of arginine biosynthesis genes<br>
<span>Score</span>: 17.27 out of 20.01
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r4c1-129477-ArgR"></div>

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 22.24 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r4c1-133936-NrtR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r4c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-156"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-156">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c1-domain-bubble-svg" data-filename="JABTEZ010000004.1_r4c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r4c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (86407 - 134411): neighbouring NRPS-NRPS-like
</option>



<option value="CC2">
Candidate 2 (86407 - 129521): single NRPS
</option>



<option value="CC3">
Candidate 3 (90687 - 132571): single NRPS-like
</option>



<option value="CC4">
Candidate 4 (92593 - 134411): single NRPS-like
</option>


</select>

<div class="details-svg" id="r4c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r4c1-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c1-nrps_pks_products" data-name="r4c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r4c1-nrps_pks_monomers" data-name="r4c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r4c1-tfbs-finder" data-name="r4c1-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r4c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-152"></div> <span class="help-tooltip" id="nrps-structures-help-152">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 86407 - 134411:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c1-1" class="smiles-canvas" data-smiles="NC(CCCNC(N)=N)C(=O)NC([*])C(=O)NC(C(C)CC)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Arg) + (X) + (Ile)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=arg&amp;nrps3=ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=arg*&amp;nrps2=x&amp;nrps3=ile*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCCNC(N)=N)C(=O)NC([*])C(=O)NC(C(C)CC)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 2, location 86407 - 129521:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c1-2" class="smiles-canvas" data-smiles="NC(CCCNC(N)=N)C(=O)NC([*])C(=O)NC(C(C)CC)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Arg) + (X) + (Ile)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=arg&amp;nrps3=ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=arg*&amp;nrps2=x&amp;nrps3=ile*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c1-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCCNC(N)=N)C(=O)NC([*])C(=O)NC(C(C)CC)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 3, location 90687 - 132571:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c1-3" class="smiles-canvas" data-smiles="NC(CCCNC(N)=N)C(=O)NC([*])C(=O)NC(C(C)CC)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Arg) + (X) + (Ile)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=arg&amp;nrps3=ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=arg*&amp;nrps2=x&amp;nrps3=ile*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c1-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCCNC(N)=N)C(=O)NC([*])C(=O)NC(C(C)CC)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 4, location 92593 - 134411:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r4c1-4" class="smiles-canvas" data-smiles="NC(CCCNC(N)=N)C(=O)NC([*])C(=O)NC(C(C)CC)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Arg) + (X) + (Ile)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=arg&amp;nrps3=ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=arg*&amp;nrps2=x&amp;nrps3=ile*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r4c1-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CCCNC(N)=N)C(=O)NC([*])C(=O)NC(C(C)CC)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r4c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-153"></div> <span class="help-tooltip" id="nrps-monomers-help-153">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">HS041_27080</span></strong>: Arg
<div class="collapser collapser-target-HS041_27080 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=arg" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=arg*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (461..851)</strong>: Arg
<div class="collapser collapser-target-HS041_27080 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Arg
<div class="collapser collapser-target-HS041_27080-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVENVGAITK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DVENVGAITK</span> (68% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_27090</span></strong>: X
<div class="collapser collapser-target-HS041_27090 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (39..423)</strong>: X
<div class="collapser collapser-target-HS041_27090 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-HS041_27090-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>Aromatic side chain (Phe, Trp, Ph-Gly, Tyr, bOH-Tyr)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DPMVVGVVAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>ndmHex <span class="serif">DAMVLGVVAK</span> (44% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_27095</span></strong>: Ile
<div class="collapser collapser-target-HS041_27095 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ile*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (14..403)</strong>: Ile
<div class="collapser collapser-target-HS041_27095 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ile
<div class="collapser collapser-target-HS041_27095-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>val (Val)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAFFLGVTFK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ile <span class="serif">DAFFLGVTFK</span> (71% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r4c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-154"></div> <span class="help-tooltip" id="tfbs-side-help-154">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/HypR" target="_blank">HypR</a></td>
<td>L-hydroxyproline utilization repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ArgR" target="_blank">ArgR</a></td>
<td>Regulator of arginine biosynthesis genes</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r4c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000004.1 - Region 2 - PKS-like,butyrolactone <div class="help-container"> <div class="help-icon" data-id="region_svg-help-270"></div> <span class="help-tooltip" id="region_svg-help-270">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000004.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r4c2-svg" data-filename="JABTEZ010000004.1_r4c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 244,100 - 285,164 nt. (total: 41,065 nt)

<a class="cluster-rules-header" id="r4c2-rules-header" href="#r4c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r4c2-rules">PKS-like: (t2pks2 or ksIII)<br>butyrolactone: AfsA</div>

<div class="region-svg-container">
<div id='r4c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-271"></div> <span class="help-tooltip" id="focus-panel-help-271">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r4c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r4c2-gene-table-details" data-name="r4c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r4c2-nrps_pks" data-name="r4c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r4c2-MIBiG-cluster-compare" data-name="r4c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r4c2-knownclusterblast" data-name="r4c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r4c2-subclusterblast" data-name="r4c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r4c2-tfbs-finder" data-name="r4c2-tfbs-finder">
TFBS Finder
</div>


</div>


<div class="body-details-section r4c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-165"></div> <span class="help-tooltip" id="gt-help-tooltip-help-165">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r4c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r4c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r4c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r4c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r4c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-158"></div> <span class="help-tooltip" id="nrps-domain-help-158">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r4c2-details-svg" data-filename="JABTEZ010000004.1_r4c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox" checked>  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r4c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r4c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-159"></div> <span class="help-tooltip" id="MIBiG-help-159">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r4c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r4c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r4c2"></div>
<div id="comparison-MIBiG-r4c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">PKS-like</span></th>

<th><span class="cc-heat-product">butyrolactone</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000919: 746-6062">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000919/index.html#r1c1" target="_new">BGC0000919</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>






<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.48</td>
<td>Other</td>

<td>phaseolotoxin</td>


<td>Pseudomonas syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444: 3281-14045">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444</a></td>





<td class="cc-heat-cell" title="0.63 (id:0.49, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>






<td class="cc-heat-cell" title="0.39 (id:0.49, order:N/A, components:0.31)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.41</td>
<td>Other (shikimate derived)</td>

<td>caboxamycin</td>


<td>Streptomyces sp. NTK 937</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001458: 100-11195">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001458/index.html#r1c1" target="_new">BGC0001458</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>






<td class="cc-heat-cell" title="0.42 (id:0.43, order:N/A, components:0.42)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.42), rgb(0, 0, 0, 0.42) 42%, white 42%)"></td>


<td>0.34</td>
<td>Other (Aminocoumarin)</td>

<td>salinipostin G</td>


<td>Salinispora tropica CNB-440</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000217: 2-8089">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000217/index.html#r1c1" target="_new">BGC0000217</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>






<td class="cc-heat-cell" title="0.41 (id:0.46, order:N/A, components:0.37)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.41), rgb(0, 0, 0, 0.41) 41%, white 41%)"></td>


<td>0.33</td>
<td>Polyketide</td>

<td>daunorubicin</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000895: 2-11180">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000895/index.html#r1c1" target="_new">BGC0000895</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.34 (id:0.44, order:N/A, components:0.27)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.34), rgb(0, 0, 0, 0.34) 34%, white 34%)"></td>


<td>0.27</td>
<td>Other</td>

<td>citrulline</td>


<td>Lactobacillus plantarum subsp. plantarum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000218: 27-8720">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000218/index.html#r1c1" target="_new">BGC0000218</a></td>





<td class="cc-heat-cell" title="0.46 (id:0.45, order:0.33, components:0.65)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.46), rgb(0, 0, 0, 0.46) 46%, white 46%)"></td>






<td class="cc-heat-cell" title="0.39 (id:0.46, order:N/A, components:0.32)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.39), rgb(0, 0, 0, 0.39) 39%, white 39%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>doxorubicin</td>


<td>Streptomyces peucetius</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000905: 1-16359">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000905/index.html#r1c1" target="_new">BGC0000905</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.27</td>
<td>Other</td>

<td>heme D1</td>


<td>Heliobacillus mobilis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000906: 424-16949">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000906/index.html#r1c1" target="_new">BGC0000906</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.44, order:N/A, components:0.56)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.33 (id:0.44, order:N/A, components:0.26)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.33), rgb(0, 0, 0, 0.33) 33%, white 33%)"></td>


<td>0.27</td>
<td>Other</td>

<td>heme D1</td>


<td>Heliophilum fasciatum</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001747: 0-11156">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001747/index.html#r1c1" target="_new">BGC0001747</a></td>





<td class="cc-heat-cell" title="0.49 (id:0.42, order:N/A, components:0.58)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.49), rgb(0, 0, 0, 0.49) 49%, white 49%)"></td>






<td class="cc-heat-cell" title="0.32 (id:0.42, order:N/A, components:0.25)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.32), rgb(0, 0, 0, 0.32) 32%, white 32%)"></td>


<td>0.26</td>
<td>Other</td>

<td>pseudaminic acid</td>


<td>Tannerella forsythia</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000812: 502-25834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000812/index.html#r1c1" target="_new">BGC0000812</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.26</td>
<td>Alkaloid</td>

<td>huperzine A</td>


<td>Shiraia sp. slf14</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r4c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001444: 3281-14045">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001444/index.html#r1c1" target="_new">BGC0001444</a></td>


<td class="cc-heat-cell" title="0.63 (id:0.49, order:N/A, components:0.80)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.63), rgb(0, 0, 0, 0.63) 63%, white 63%)"></td>
<td>0.63</td>
<td>Other (shikimate derived)</td>
<td>caboxamycin</td>
<td>Streptomyces sp. NTK 937</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000812: 502-25834">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000812/index.html#r1c1" target="_new">BGC0000812</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.48, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Alkaloid</td>
<td>huperzine A</td>
<td>Shiraia sp. slf14</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001986: 0-3657">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001986/index.html#r1c1" target="_new">BGC0001986</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other</td>
<td>purincyclamide</td>
<td>Streptomyces chrestomyceticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928: 108-8402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other</td>
<td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>
<td>Streptomyces sp. MA37</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000919: 746-6062">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000919/index.html#r1c1" target="_new">BGC0000919</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.44, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Other</td>
<td>phaseolotoxin</td>
<td>Pseudomonas syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000665: 0-21259">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000665/index.html#r1c1" target="_new">BGC0000665</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.45, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Terpene, Alkaloid</td>
<td>xiamycin</td>
<td>Streptomyces sp. HKI0576</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002127: 147-3013">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002127/index.html#r1c1" target="_new">BGC0002127</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>flaviolin, 1,3,6,8-tetrahydroxynaphthalene</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001310: 0-2705">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001310/index.html#r1c1" target="_new">BGC0001310</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.45, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Polyketide</td>
<td>naringenin</td>
<td>Streptomyces clavuligerus ATCC 27064</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001458: 100-11195">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001458/index.html#r1c1" target="_new">BGC0001458</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.43, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other (Aminocoumarin)</td>
<td>salinipostin G</td>
<td>Salinispora tropica CNB-440</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000217: 2-8089">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000217/index.html#r1c1" target="_new">BGC0000217</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.46, order:N/A, components:0.60)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>Polyketide</td>
<td>daunorubicin</td>
<td>Streptomyces sp.</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r4c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-160"></div> <span class="help-tooltip" id="cb-known-help-160">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r4c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r4c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r4c2_1.svg">
ECO-0501
</option>

<option value = "svg/knownclusterblast_r4c2_2.svg">
dudomycin A
</option>

<option value = "svg/knownclusterblast_r4c2_3.svg">
incednine
</option>

</select>
<button id="knownclusterblast-r4c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r4c2-svg"></div>

</div>
</div>

<div class="body-details-section r4c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-161"></div> <span class="help-tooltip" id="cb-sub-help-161">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r4c2-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-164"></div> <span class="help-tooltip" id="tfbs-body-help-164">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 17.57 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r4c2-252292-ZuR"></div>

</div>
</div>
</div>

</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r4c2-nrps_pks_monomers" data-name="r4c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r4c2-tfbs-finder" data-name="r4c2-tfbs-finder">
TFBS Finder
</div>


</div>










<div class="sidepanel-details-section r4c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-162"></div> <span class="help-tooltip" id="nrps-monomers-help-162">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

</dl>
</div>
</div>



<div class="sidepanel-details-section r4c2-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-163"></div> <span class="help-tooltip" id="tfbs-side-help-163">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>


</div>

</div>

</div>





<div class="page" id='r5c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000005.1 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-272"></div> <span class="help-tooltip" id="region_svg-help-272">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000005.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c1-svg" data-filename="JABTEZ010000005.1_r5c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 377,573 - 426,460 nt. (total: 48,888 nt)

<a class="cluster-rules-header" id="r5c1-rules-header" href="#r5c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r5c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-273"></div> <span class="help-tooltip" id="focus-panel-help-273">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c1-gene-table-details" data-name="r5c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c1-nrps_pks" data-name="r5c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c1-MIBiG-cluster-compare" data-name="r5c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c1-knownclusterblast" data-name="r5c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c1-subclusterblast" data-name="r5c1-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c1-tfbs-finder" data-name="r5c1-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r5c1-nrps-pks-bubbles" data-name="r5c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r5c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-175"></div> <span class="help-tooltip" id="gt-help-tooltip-help-175">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-166"></div> <span class="help-tooltip" id="nrps-domain-help-166">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-details-svg" data-filename="JABTEZ010000005.1_r5c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-167"></div> <span class="help-tooltip" id="MIBiG-help-167">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c1"></div>
<div id="comparison-MIBiG-r5c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.45, order:1.00, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>NRP</td>

<td>taxlllaid A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.44, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.37</td>
<td>NRP</td>

<td>sevadicin</td>


<td>Paenibacillus larvae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.45, order:1.00, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>NRP, Polyketide</td>

<td>thalassospiramide A</td>


<td>Tistrella bauzanensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.47, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.36</td>
<td>NRP</td>

<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>


<td>Pseudomonas sp. QS1027</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185: 0-11721">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>bacillibactin</td>


<td>Bacillus velezensis FZB42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001925: 0-20488">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001925/index.html#r1c1" target="_new">BGC0001925</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>Alkaloid</td>

<td>altemicidin, SB-203207, SB-203208</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417: 0-12076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.34</td>
<td>NRP</td>

<td>rhodochelin</td>


<td>Rhodococcus jostii RHA1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465: 0-10043">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>xenortide A, xenortide B, xenortide C, xenortide D</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401: 0-12882">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>paenibactin</td>


<td>Paenibacillus elgii B69</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.33</td>
<td>NRP</td>

<td>vicibactin</td>


<td>Rhizobium etli CFN 42</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.45, order:1.00, components:0.76)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>taxlllaid A</td>
<td>Xenorhabdus bovienii SS-2004</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.44, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>sevadicin</td>
<td>Paenibacillus larvae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.45, order:1.00, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>NRP, Polyketide</td>
<td>thalassospiramide A</td>
<td>Tistrella bauzanensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.47, order:1.00, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>NRP</td>
<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>
<td>Pseudomonas sp. QS1027</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185: 0-11721">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>bacillibactin</td>
<td>Bacillus velezensis FZB42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001925: 0-20488">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001925/index.html#r1c1" target="_new">BGC0001925</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>Alkaloid</td>
<td>altemicidin, SB-203207, SB-203208</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000417: 0-12076">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000417/index.html#r1c1" target="_new">BGC0000417</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>rhodochelin</td>
<td>Rhodococcus jostii RHA1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465: 0-10043">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.44, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>xenortide A, xenortide B, xenortide C, xenortide D</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000401: 0-12882">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000401/index.html#r1c1" target="_new">BGC0000401</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.46, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>paenibactin</td>
<td>Paenibacillus elgii B69</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>vicibactin</td>
<td>Rhizobium etli CFN 42</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-168"></div> <span class="help-tooltip" id="cb-known-help-168">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r5c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r5c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r5c1_1.svg">
crochelin A
</option>

<option value = "svg/knownclusterblast_r5c1_2.svg">
theonellamide
</option>

<option value = "svg/knownclusterblast_r5c1_3.svg">
plantaribactin
</option>

<option value = "svg/knownclusterblast_r5c1_4.svg">
azotobactin D
</option>

<option value = "svg/knownclusterblast_r5c1_5.svg">
bleomycin
</option>

<option value = "svg/knownclusterblast_r5c1_6.svg">
bleomycin A2/bleomycin B2
</option>

<option value = "svg/knownclusterblast_r5c1_7.svg">
kedarcidin
</option>

</select>
<button id="knownclusterblast-r5c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r5c1-svg"></div>

</div>
</div>

<div class="body-details-section r5c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-169"></div> <span class="help-tooltip" id="cb-sub-help-169">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-173"></div> <span class="help-tooltip" id="tfbs-body-help-173">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 23.68 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r5c1-385539-CelR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 16.49 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r5c1-380589-ZuR"></div>

<div class="tfbs-hit-detail">
NuR
<div class="collapser collapser-target-NuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Nickel-responsive regulator<br>
<span>Score</span>: 19.95 out of 30.98
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r5c1-382473-NuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 16.54 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r5c1-389379-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r5c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-174"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-174">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c1-domain-bubble-svg" data-filename="JABTEZ010000005.1_r5c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (377572 - 426460): single NRPS
</option>


</select>

<div class="details-svg" id="r5c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c1-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c1-nrps_pks_products" data-name="r5c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c1-nrps_pks_monomers" data-name="r5c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r5c1-tfbs-finder" data-name="r5c1-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r5c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-170"></div> <span class="help-tooltip" id="nrps-structures-help-170">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 377572 - 426460:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c1-1" class="smiles-canvas" data-smiles="NC(CC1=CNc2c1cccc2)C(=O)NC(CO)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Trp) + (Ser)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=trp&amp;nrps2=ser" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=trp*&amp;nrps2=ser*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CC1=CNc2c1cccc2)C(=O)NC(CO)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-171"></div> <span class="help-tooltip" id="nrps-monomers-help-171">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">HS041_32680</span></strong>: Trp
<div class="collapser collapser-target-HS041_32680 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=trp" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=trp*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (457..861)</strong>: Trp
<div class="collapser collapser-target-HS041_32680 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Trp
<div class="collapser collapser-target-HS041_32680-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>asp,asn (Asp, Asn)</dd>
<dt>Single AA prediction:</dt>
<dd>asn (Asn)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLTKVGEVGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Trp <span class="serif">DLTKVGEVGK</span> (88% 8Å match)</dd>
<dd>Asn <span class="serif">DLTKVGEVGK</span> (76% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_32685</span></strong>: Ser
<div class="collapser collapser-target-HS041_32685 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ser*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (536..941)</strong>: Ser
<div class="collapser collapser-target-HS041_32685 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ser
<div class="collapser collapser-target-HS041_32685-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>ser (Ser)</dd>
<dt>Single AA prediction:</dt>
<dd>ser (Ser)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVWHISLIDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ser <span class="serif">DVWHISLIDK</span> (97% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r5c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-172"></div> <span class="help-tooltip" id="tfbs-side-help-172">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NuR" target="_blank">NuR</a></td>
<td>Nickel-responsive regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CelR" target="_blank">CelR</a></td>
<td>Cellobiose uptake repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r5c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000005.1 - Region 2 - NRPS,T1PKS,terpene <div class="help-container"> <div class="help-icon" data-id="region_svg-help-274"></div> <span class="help-tooltip" id="region_svg-help-274">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000005.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c2-svg" data-filename="JABTEZ010000005.1_r5c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 590,054 - 681,562 nt. (total: 91,509 nt)

<a class="cluster-rules-header" id="r5c2-rules-header" href="#r5c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c2-rules">terpene: (Terpene_synth or Terpene_synth_C or phytoene_synt or Lycopene_cycl or terpene_cyclase or NapT7 or fung_ggpps or fung_ggpps2 or trichodiene_synth or TRI5)<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))<br>NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r5c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-275"></div> <span class="help-tooltip" id="focus-panel-help-275">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c2-gene-table-details" data-name="r5c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c2-nrps_pks" data-name="r5c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c2-MIBiG-cluster-compare" data-name="r5c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c2-knownclusterblast" data-name="r5c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c2-subclusterblast" data-name="r5c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c2-tfbs-finder" data-name="r5c2-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r5c2-nrps-pks-bubbles" data-name="r5c2-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r5c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-185"></div> <span class="help-tooltip" id="gt-help-tooltip-help-185">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-176"></div> <span class="help-tooltip" id="nrps-domain-help-176">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c2-details-svg" data-filename="JABTEZ010000005.1_r5c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-177"></div> <span class="help-tooltip" id="MIBiG-help-177">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c2"></div>
<div id="comparison-MIBiG-r5c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">terpene</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>

<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.73 (id:0.46, order:1.00, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>






<td class="cc-heat-cell" title="0.73 (id:0.46, order:1.00, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>


<td>0.83</td>
<td>NRP, Polyketide</td>

<td>thalassospiramide A</td>


<td>Tistrella bauzanensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661: 0-5412">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.52, order:0.83, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>






<td class="cc-heat-cell" title="0.76 (id:0.52, order:0.83, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.82</td>
<td>Terpene</td>

<td>geosmin</td>


<td>Nostoc punctiforme PCC 73102</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001181: 0-2181">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001181/index.html#r1c1" target="_new">BGC0001181</a></td>





<td class="cc-heat-cell" title="0.72 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>






<td class="cc-heat-cell" title="0.72 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.79</td>
<td>Terpene</td>

<td>geosmin</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.73 (id:0.45, order:1.00, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>






<td class="cc-heat-cell" title="0.70 (id:0.45, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.79</td>
<td>NRP</td>

<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>


<td>Pseudomonas sp. QS1027</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001133: 0-25888">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001133/index.html#r1c1" target="_new">BGC0001133</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.72 (id:0.44, order:1.00, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.44, order:0.75, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.74</td>
<td>NRP</td>

<td>taxlllaid A</td>


<td>Xenorhabdus bovienii SS-2004</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002309: 0-2466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002309/index.html#r1c1" target="_new">BGC0002309</a></td>





<td class="cc-heat-cell" title="0.70 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>






<td class="cc-heat-cell" title="0.70 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>



<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>


<td>0.74</td>
<td>Terpene</td>

<td>cattleyene</td>


<td>Streptomyces cattleya NRRL 8057 = DSM 46488</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.70</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002017: 0-12514">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002017/index.html#r1c1" target="_new">BGC0002017</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.70</td>
<td>NRP</td>

<td>potensibactin</td>


<td>Nocardiopsis potens DSM 45234</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:1.00, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.70 (id:0.43, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>


<td>0.70</td>
<td>NRP</td>

<td>sevadicin</td>


<td>Paenibacillus larvae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="no hits" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0%), rgb(0, 0, 0, 0%) 0%, white 0%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.68</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000661: 0-5412">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000661/index.html#r1c1" target="_new">BGC0000661</a></td>


<td class="cc-heat-cell" title="0.76 (id:0.52, order:0.83, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.76), rgb(0, 0, 0, 0.76) 76%, white 76%)"></td>
<td>0.76</td>
<td>Terpene</td>
<td>geosmin</td>
<td>Nostoc punctiforme PCC 73102</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001050: 0-19730">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001050/index.html#r1c1" target="_new">BGC0001050</a></td>


<td class="cc-heat-cell" title="0.73 (id:0.46, order:1.00, components:0.84)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.73), rgb(0, 0, 0, 0.73) 73%, white 73%)"></td>
<td>0.73</td>
<td>NRP, Polyketide</td>
<td>thalassospiramide A</td>
<td>Tistrella bauzanensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001181: 0-2181">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001181/index.html#r1c1" target="_new">BGC0001181</a></td>


<td class="cc-heat-cell" title="0.72 (id:0.51, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.72), rgb(0, 0, 0, 0.72) 72%, white 72%)"></td>
<td>0.72</td>
<td>Terpene</td>
<td>geosmin</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002309: 0-2466">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002309/index.html#r1c1" target="_new">BGC0002309</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>Terpene</td>
<td>cattleyene</td>
<td>Streptomyces cattleya NRRL 8057 = DSM 46488</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002071: 0-27266">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002071/index.html#r1c1" target="_new">BGC0002071</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.45, order:0.75, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>virginiafactin A, virginiafactin B, virginiafactin C, virginiafactin D</td>
<td>Pseudomonas sp. QS1027</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000426: 0-11658">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000426/index.html#r1c1" target="_new">BGC0000426</a></td>


<td class="cc-heat-cell" title="0.70 (id:0.43, order:1.00, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.70), rgb(0, 0, 0, 0.70) 70%, white 70%)"></td>
<td>0.70</td>
<td>NRP</td>
<td>sevadicin</td>
<td>Paenibacillus larvae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002688: 824-19983">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002688/index.html#r1c1" target="_new">BGC0002688</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>NRP</td>
<td>benarthin, dibenarthin</td>
<td>Planobispora rosea</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001185: 0-11721">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001185/index.html#r1c1" target="_new">BGC0001185</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>bacillibactin</td>
<td>Bacillus velezensis FZB42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001561: 0-17578">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001561/index.html#r1c1" target="_new">BGC0001561</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.47, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>curacomycin</td>
<td>Streptomyces curacoi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-178"></div> <span class="help-tooltip" id="cb-known-help-178">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r5c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r5c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r5c2_1.svg">
geosmin
</option>

<option value = "svg/knownclusterblast_r5c2_2.svg">
crochelin A
</option>

<option value = "svg/knownclusterblast_r5c2_3.svg">
bleomycin
</option>

<option value = "svg/knownclusterblast_r5c2_4.svg">
geosmin
</option>

<option value = "svg/knownclusterblast_r5c2_5.svg">
catenulisporolides
</option>

<option value = "svg/knownclusterblast_r5c2_6.svg">
funisamine
</option>

<option value = "svg/knownclusterblast_r5c2_7.svg">
2-methylisoborneol
</option>

<option value = "svg/knownclusterblast_r5c2_8.svg">
glycopeptidolipid
</option>

<option value = "svg/knownclusterblast_r5c2_9.svg">
viomycin
</option>

<option value = "svg/knownclusterblast_r5c2_10.svg">
capreomycin IA/capreomycin IB/capreomycin IIA/capreomycin IIB
</option>

</select>
<button id="knownclusterblast-r5c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r5c2-svg"></div>

</div>
</div>

<div class="body-details-section r5c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-179"></div> <span class="help-tooltip" id="cb-sub-help-179">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c2-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-183"></div> <span class="help-tooltip" id="tfbs-body-help-183">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
AfsQ1
<div class="collapser collapser-target-AfsQ1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Two-component system AfsQ1-Q2, activator of antibiotic production<br>
<span>Score</span>: 18.42 out of 19.52
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r5c2-594417-AfsQ1"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (1):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
HypR
<div class="collapser collapser-target-HypR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: L-hydroxyproline utilization repressor<br>
<span>Score</span>: 18.59 out of 22.40
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r5c2-597193-HypR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r5c2-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-184"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-184">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c2-domain-bubble-svg" data-filename="JABTEZ010000005.1_r5c2_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c2-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC2">
Candidate 2 (590053 - 681562): neighbouring terpene-T1PKS-NRPS
</option>



<option value="CC4">
Candidate 4 (595870 - 641309): single T1PKS
</option>



<option value="CC5">
Candidate 5 (602518 - 681562): single NRPS
</option>


</select>

<div class="details-svg" id="r5c2-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c2-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c2-nrps_pks_products" data-name="r5c2-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c2-nrps_pks_monomers" data-name="r5c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r5c2-tfbs-finder" data-name="r5c2-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r5c2-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-180"></div> <span class="help-tooltip" id="nrps-structures-help-180">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 2, location 590053 - 681562:
<div class="collapser collapser-target-CC2 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c2-2" class="smiles-canvas" data-smiles="C=CNC(CCCNC(N)=N)C(=O)NC(CCCN)C(=O)NC([*])C(=O)NC([*])C(=O)NC(Cc1ccccc1)C(=O)NC(CO)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CC(=O)N)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC(CC(=O)N)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(ccmal) + (Arg - Orn) + (X) + (X) + (Phe - Ser) + (Thr - X - Asn) + (X) + (Thr - Asn)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal&amp;nrps2=arg,orn&amp;nrps5=phe,ser&amp;nrps6=thr,asn&amp;nrps8=thr,asn" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal*&amp;nrps2=arg*,orn*&amp;nrps3=x&amp;nrps4=x&amp;nrps5=phe*,ser*&amp;nrps6=thr*,x,asn*&amp;nrps7=x&amp;nrps8=thr*,asn*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c2-2-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C=CNC(CCCNC(N)=N)C(=O)NC(CCCN)C(=O)NC([*])C(=O)NC([*])C(=O)NC(Cc1ccccc1)C(=O)NC(CO)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CC(=O)N)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC(CC(=O)N)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 4, location 595870 - 641309:
<div class="collapser collapser-target-CC4 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c2-4" class="smiles-canvas" data-smiles="C=CNC(CCCNC(N)=N)C(=O)NC(CCCN)C(=O)NC([*])C(=O)NC([*])C(=O)NC(Cc1ccccc1)C(=O)NC(CO)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CC(=O)N)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(ccmal) + (Arg - Orn) + (X) + (X) + (Phe - Ser) + (Thr - X - Asn)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal&amp;nrps2=arg,orn&amp;nrps5=phe,ser&amp;nrps6=thr,asn" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal*&amp;nrps2=arg*,orn*&amp;nrps3=x&amp;nrps4=x&amp;nrps5=phe*,ser*&amp;nrps6=thr*,x,asn*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c2-4-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C=CNC(CCCNC(N)=N)C(=O)NC(CCCN)C(=O)NC([*])C(=O)NC([*])C(=O)NC(Cc1ccccc1)C(=O)NC(CO)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CC(=O)N)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 5, location 602518 - 681562:
<div class="collapser collapser-target-CC5 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c2-5" class="smiles-canvas" data-smiles="C=CNC(CCCNC(N)=N)C(=O)NC(CCCN)C(=O)NC([*])C(=O)NC([*])C(=O)NC(Cc1ccccc1)C(=O)NC(CO)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CC(=O)N)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC(CC(=O)N)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(ccmal) + (Arg - Orn) + (X) + (X) + (Phe - Ser) + (Thr - X - Asn) + (X) + (Thr - Asn)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal&amp;nrps2=arg,orn&amp;nrps5=phe,ser&amp;nrps6=thr,asn&amp;nrps8=thr,asn" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=ccmal*&amp;nrps2=arg*,orn*&amp;nrps3=x&amp;nrps4=x&amp;nrps5=phe*,ser*&amp;nrps6=thr*,x,asn*&amp;nrps7=x&amp;nrps8=thr*,asn*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c2-5-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="C=CNC(CCCNC(N)=N)C(=O)NC(CCCN)C(=O)NC([*])C(=O)NC([*])C(=O)NC(Cc1ccccc1)C(=O)NC(CO)C(=O)NC(C(O)C)C(=O)NC([*])C(=O)NC(CC(=O)N)C(=O)NC([*])C(=O)NC(C(O)C)C(=O)NC(CC(=O)N)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-181"></div> <span class="help-tooltip" id="nrps-monomers-help-181">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">HS041_33640</span></strong>: mal
<div class="collapser collapser-target-HS041_33640 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (532..817)</strong>: mal
<div class="collapser collapser-target-HS041_33640 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-HS041_33640-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 91.7%</dd>
<dd>inactive: 87.5%</dd>
<dd>Methylmalonyl-CoA: 58.3%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-HS041_33640-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 154.2</dt>

<dd></dd><dt>inactive: 96.1</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 63.1</dt>

<dd></dd><dt>Methylmalonyl-CoA: 52.1</dt>

<dd></dd><dt>Propionyl-CoA: 47.8</dt>

<dd></dd><dt>Isobutyryl-CoA: 34.7</dt>

<dd></dd><dt>Benzoyl-CoA: 29.3</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 24.1</dt>

<dd></dd><dt>Acetyl-CoA: 21.8</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 21.3</dt>

<dd></dd><dt>CHC-CoA: 20.1</dt>

<dd></dd><dt>fatty_acid: 18.8</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 8.6</dt>

<dd></dd><dt>trans-1,2-CPDA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

<strong><span class="serif">HS041_33655</span></strong>: Arg - Orn
<div class="collapser collapser-target-HS041_33655 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=arg,orn" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=arg*,orn*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (17..398)</strong>: Arg
<div class="collapser collapser-target-HS041_33655 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Arg
<div class="collapser collapser-target-HS041_33655-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVEDTGAITK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Arg <span class="serif">DAEDIGAITK</span> (65% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1055..1453)</strong>: Orn
<div class="collapser collapser-target-HS041_33655 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Orn
<div class="collapser collapser-target-HS041_33655-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Long positively charged side chain (Orn, Lys, Arg)</dd>
<dt>Small clusters prediction:</dt>
<dd>Orn and hydroxy-Orn specific (Orn, OH-Orn)</dd>
<dt>Single AA prediction:</dt>
<dd>orn (Orn)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVGEAGVVGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Orn <span class="serif">DVGEIGSVDK</span> (59% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_33660</span></strong>: X
<div class="collapser collapser-target-HS041_33660 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>AMP-binding (460..859)</strong>: X
<div class="collapser collapser-target-HS041_33660 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr
<div class="collapser collapser-target-HS041_33660-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLEHVGVMVK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>bLys <span class="serif">DTEDVGTMVK</span> (47% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_33665</span></strong>: X
<div class="collapser collapser-target-HS041_33665 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>AMP-binding (463..861)</strong>: X
<div class="collapser collapser-target-HS041_33665 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-HS041_33665-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">VDWLVSLWDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>bAla <span class="serif">VDWVVSLADK</span> (71% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_33675</span></strong>: Phe - Ser
<div class="collapser collapser-target-HS041_33675 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe,ser" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=phe*,ser*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (500..889)</strong>: Phe
<div class="collapser collapser-target-HS041_33675 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Phe
<div class="collapser collapser-target-HS041_33675-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aromatic (Phe, Tyr, diOH-Bz, Ph-Gly, bOH-Tyr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>phe (Phe)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DAWQFATIDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gln <span class="serif">DAWQFGLIDK</span> (74% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1538..1941)</strong>: Ser
<div class="collapser collapser-target-HS041_33675 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Ser
<div class="collapser collapser-target-HS041_33675-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>ser (Ser)</dd>
<dt>Single AA prediction:</dt>
<dd>ser (Ser)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVWHFSLVDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Ser <span class="serif">DVWHFSLVDK</span> (94% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_33680</span></strong>: Thr - X - Asn
<div class="collapser collapser-target-HS041_33680 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr,asn" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*,asn*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (532..961)</strong>: Thr
<div class="collapser collapser-target-HS041_33680 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Thr
<div class="collapser collapser-target-HS041_33680-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>thr (Thr)</dd>
<dt>Single AA prediction:</dt>
<dd>thr (Thr)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DFWNVGMVHK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Thr <span class="serif">DFWNVGMVHK</span> (94% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1621..2019)</strong>: X
<div class="collapser collapser-target-HS041_33680 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-HS041_33680-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DMPKLIEVAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Asn <span class="serif">DMTKVGEVGK</span> (56% 8Å match)</dd>
<dd>D-Ala <span class="serif">DMPQLGMVWK</span> (38% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (2685..3103)</strong>: Asn
<div class="collapser collapser-target-HS041_33680 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Asn
<div class="collapser collapser-target-HS041_33680-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>asp,asn (Asp, Asn)</dd>
<dt>Single AA prediction:</dt>
<dd>asn (Asn)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DMTKVGEVAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Asn <span class="serif">DMTKVGEVGK</span> (65% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_33685</span></strong>: X
<div class="collapser collapser-target-HS041_33685 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>AMP-binding (482..895)</strong>: X
<div class="collapser collapser-target-HS041_33685 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad
<div class="collapser collapser-target-HS041_33685-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DIPDVGEVGK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Asn <span class="serif">DLTKVGEVGK</span> (65% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_33690</span></strong>: Thr - Asn
<div class="collapser collapser-target-HS041_33690 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr,asn" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=thr*,asn*" target="_new">relaxed</a>
</dd>
<br>





<dd><strong>AMP-binding (555..957)</strong>: Thr
<div class="collapser collapser-target-HS041_33690 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Thr
<div class="collapser collapser-target-HS041_33690-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain or phenyl group with -OH (Ser, Thr, Dhpg, Hpg)</dd>
<dt>Small clusters prediction:</dt>
<dd>thr (Thr)</dd>
<dt>Single AA prediction:</dt>
<dd>thr (Thr)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DFWNVGMVHK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Thr <span class="serif">DFWNVGMVHK</span> (94% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>100% (strong)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (1619..2008)</strong>: Asn
<div class="collapser collapser-target-HS041_33690 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Asn
<div class="collapser collapser-target-HS041_33690-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>asp,asn (Asp, Asn)</dd>
<dt>Single AA prediction:</dt>
<dd>asn (Asn)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLTKIGEVAK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Asn <span class="serif">DLTKIGEVGK</span> (68% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r5c2-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-182"></div> <span class="help-tooltip" id="tfbs-side-help-182">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsQ1" target="_blank">AfsQ1</a></td>
<td>Two-component system AfsQ1-Q2, activator of antibiotic production</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/HypR" target="_blank">HypR</a></td>
<td>L-hydroxyproline utilization repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>

<div class="page" id='r5c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000005.1 - Region 3 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-276"></div> <span class="help-tooltip" id="region_svg-help-276">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000005.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r5c3-svg" data-filename="JABTEZ010000005.1_r5c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 706,575 - 757,169 nt. (total: 50,595 nt)

<a class="cluster-rules-header" id="r5c3-rules-header" href="#r5c3">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r5c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r5c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-277"></div> <span class="help-tooltip" id="focus-panel-help-277">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r5c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r5c3-gene-table-details" data-name="r5c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r5c3-nrps_pks" data-name="r5c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r5c3-MIBiG-cluster-compare" data-name="r5c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r5c3-knownclusterblast" data-name="r5c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r5c3-subclusterblast" data-name="r5c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r5c3-tfbs-finder" data-name="r5c3-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r5c3-nrps-pks-bubbles" data-name="r5c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r5c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-195"></div> <span class="help-tooltip" id="gt-help-tooltip-help-195">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r5c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r5c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r5c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r5c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r5c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-186"></div> <span class="help-tooltip" id="nrps-domain-help-186">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c3-details-svg" data-filename="JABTEZ010000005.1_r5c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r5c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r5c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-187"></div> <span class="help-tooltip" id="MIBiG-help-187">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r5c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r5c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r5c3"></div>
<div id="comparison-MIBiG-r5c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.67</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Escherichia coli str. K-12 substr. MG1655</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002474: 103-7254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002474/index.html#r1c1" target="_new">BGC0002474</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.43, order:N/A, components:0.92)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.61</td>
<td>NRP</td>

<td>agrobactin</td>


<td>Agrobacterium tumefaciens</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002494: 273-12179">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002494/index.html#r1c1" target="_new">BGC0002494</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.44, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>






<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.56</td>
<td>NRP</td>

<td>vibriobactin</td>


<td>Vibrio cholerae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002091: 1-17973">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002091/index.html#r1c1" target="_new">BGC0002091</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.54</td>
<td>NRP</td>

<td>holomycin</td>


<td>Yersinia ruckeri ATCC 29473</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002412: 0-12427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002412/index.html#r1c1" target="_new">BGC0002412</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>NRP</td>

<td>holomycin</td>


<td>Photobacterium galatheae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>






<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>


<td>0.53</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001940: 1-19375">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001940/index.html#r1c1" target="_new">BGC0001940</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.49, order:N/A, components:0.70)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.61 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.52</td>
<td>Polyketide</td>

<td>pyxidicycline A, pyxidicycline B</td>


<td>Pyxidicoccus fallax</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001560: 0-28792">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001560/index.html#r1c1" target="_new">BGC0001560</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>






<td class="cc-heat-cell" title="0.59 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.51</td>
<td>NRP, Polyketide</td>

<td>cryptomaldamide</td>


<td>Moorea producens JHB</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001928: 108-8402">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001928/index.html#r1c1" target="_new">BGC0001928</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>Other</td>

<td>(2R,3S,4S)-5-fluoro-2,3,4-trihydroxypentanoic acid</td>


<td>Streptomyces sp. MA37</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000465: 0-10043">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000465/index.html#r1c1" target="_new">BGC0000465</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.43, order:0.75, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>






<td class="cc-heat-cell" title="0.58 (id:0.43, order:0.75, components:0.61)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.50</td>
<td>NRP</td>

<td>xenortide A, xenortide B, xenortide C, xenortide D</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r5c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002476: 0-10330">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002476/index.html#r1c1" target="_new">BGC0002476</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Escherichia coli str. K-12 substr. MG1655</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002474: 103-7254">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002474/index.html#r1c1" target="_new">BGC0002474</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>agrobactin</td>
<td>Agrobacterium tumefaciens</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002494: 273-12179">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002494/index.html#r1c1" target="_new">BGC0002494</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.44, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>NRP</td>
<td>vibriobactin</td>
<td>Vibrio cholerae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001940: 1-19375">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001940/index.html#r1c1" target="_new">BGC0001940</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>pyxidicycline A, pyxidicycline B</td>
<td>Pyxidicoccus fallax</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002091: 1-17973">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002091/index.html#r1c1" target="_new">BGC0002091</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>holomycin</td>
<td>Yersinia ruckeri ATCC 29473</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002412: 0-12427">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002412/index.html#r1c1" target="_new">BGC0002412</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.45, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>holomycin</td>
<td>Photobacterium galatheae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001895: 425-18341">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001895/index.html#r1c1" target="_new">BGC0001895</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.42, order:N/A, components:0.86)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Polyketide</td>
<td>huanglongmycin A, huanglongmycin B, huanglongmycin C</td>
<td>Streptomyces sp. CB09001</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.41, order:N/A, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001735: 0-39550">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001735/index.html#r1c1" target="_new">BGC0001735</a></td>


<td class="cc-heat-cell" title="0.60 (id:0.43, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.60), rgb(0, 0, 0, 0.60) 60%, white 60%)"></td>
<td>0.60</td>
<td>Other</td>
<td>pentostatine, vidarabine</td>
<td>Streptomyces antibioticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002340: 194-28112">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002340/index.html#r1c1" target="_new">BGC0002340</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.48, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP, Other</td>
<td>pyrroloformamide A, pyrroloformamide B, pyrroloformamide D, pyrroloformamide C</td>
<td>Streptomyces sp. CB02980</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r5c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-188"></div> <span class="help-tooltip" id="cb-known-help-188">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r5c3-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r5c3_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r5c3_1.svg">
vazabitide A
</option>

<option value = "svg/knownclusterblast_r5c3_2.svg">
kedarcidin
</option>

<option value = "svg/knownclusterblast_r5c3_3.svg">
C-1027
</option>

<option value = "svg/knownclusterblast_r5c3_4.svg">
hedamycin
</option>

<option value = "svg/knownclusterblast_r5c3_5.svg">
trioxacarcin A
</option>

<option value = "svg/knownclusterblast_r5c3_6.svg">
LL-D49194α1 (LLD)
</option>

<option value = "svg/knownclusterblast_r5c3_7.svg">
sporolide A/sporolide B
</option>

<option value = "svg/knownclusterblast_r5c3_8.svg">
C-1027
</option>

<option value = "svg/knownclusterblast_r5c3_9.svg">
lidamycin
</option>

</select>
<button id="knownclusterblast-r5c3-download">Download graphic</button>
</div>
<div id="knownclusterblast-r5c3-svg"></div>

</div>
</div>

<div class="body-details-section r5c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-189"></div> <span class="help-tooltip" id="cb-sub-help-189">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r5c3-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-193"></div> <span class="help-tooltip" id="tfbs-body-help-193">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 23.65 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r5c3-710907-ZuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 21.67 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r5c3-712381-ZuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 21.91 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r5c3-726527-ZuR"></div>

<div class="tfbs-hit-detail">
AfsR
<div class="collapser collapser-target-AfsR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Pleiotropic regulatory for antibiotic production<br>
<span>Score</span>: 18.46 out of 23.05
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r5c3-750763-AfsR"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
LexA
<div class="collapser collapser-target-LexA collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Repressor of DNA damage response<br>
<span>Score</span>: 19.73 out of 29.65
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r5c3-720925-LexA"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 20.39 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r5c3-726575-ZuR"></div>

<div class="tfbs-hit-detail">
ZuR
<div class="collapser collapser-target-ZuR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Zinc-responsive repressor<br>
<span>Score</span>: 16.48 out of 28.19
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r5c3-737172-ZuR"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r5c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-194"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-194">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r5c3-domain-bubble-svg" data-filename="JABTEZ010000005.1_r5c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r5c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC6">
Candidate 6 (706574 - 757169): neighbouring NRPS-T1PKS
</option>



<option value="CC7">
Candidate 7 (706574 - 750708): single NRPS
</option>



<option value="CC8">
Candidate 8 (710722 - 757169): single T1PKS
</option>


</select>

<div class="details-svg" id="r5c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r5c3-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r5c3-nrps_pks_products" data-name="r5c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r5c3-nrps_pks_monomers" data-name="r5c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r5c3-tfbs-finder" data-name="r5c3-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r5c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-190"></div> <span class="help-tooltip" id="nrps-structures-help-190">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 6, location 706574 - 757169:
<div class="collapser collapser-target-CC6 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c3-6" class="smiles-canvas" data-smiles="NC(CS)C(=O)CC(O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Cys) + (ohmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys&amp;nrps2=ohmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*&amp;nrps2=ohmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c3-6-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CS)C(=O)CC(O)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 7, location 706574 - 750708:
<div class="collapser collapser-target-CC7 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c3-7" class="smiles-canvas" data-smiles="NC(CS)C(=O)CC(O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Cys) + (ohmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys&amp;nrps2=ohmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*&amp;nrps2=ohmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c3-7-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CS)C(=O)CC(O)C(=O)O"></canvas>
</div>
</div>


<hr>




<div>
<div class="prediction-text">
For candidate cluster 8, location 710722 - 757169:
<div class="collapser collapser-target-CC8 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r5c3-8" class="smiles-canvas" data-smiles="NC(CS)C(=O)CC(O)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Cys) + (ohmal)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys&amp;nrps2=ohmal" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*&amp;nrps2=ohmal*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r5c3-8-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(CS)C(=O)CC(O)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r5c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-191"></div> <span class="help-tooltip" id="nrps-monomers-help-191">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">HS041_34005</span></strong>: Cys
<div class="collapser collapser-target-HS041_34005 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=cys*" target="_new">relaxed</a>
</dd>
<br>







<dd><strong>AMP-binding (577..979)</strong>: Cys
<div class="collapser collapser-target-HS041_34005 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Cys
<div class="collapser collapser-target-HS041_34005-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Polar, uncharged (aliphatic with -SH) (Cys)</dd>
<dt>Small clusters prediction:</dt>
<dd>Polar, uncharged (aliphatic with -SH) (Cys)</dd>
<dt>Single AA prediction:</dt>
<dd>Polar, uncharged (aliphatic with -SH) (Cys)</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DLYNFSLIWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Cys <span class="serif">DLYNLSLIWK</span> (85% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>90% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_34010</span></strong>: mal
<div class="collapser collapser-target-HS041_34010 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (529..805)</strong>: mal
<div class="collapser collapser-target-HS041_34010 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-HS041_34010-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-HS041_34010-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 85.5</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 65.2</dt>

<dd></dd><dt>Methylmalonyl-CoA: 59.1</dt>

<dd></dd><dt>Propionyl-CoA: 37.3</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 29.5</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 29.3</dt>

<dd></dd><dt>inactive: 29.1</dt>

<dd></dd><dt>trans-1,2-CPDA: 18.5</dt>

<dd></dd><dt>CHC-CoA: 16.4</dt>

<dd></dd><dt>Isobutyryl-CoA: 15.6</dt>

<dd></dd><dt>fatty_acid: 11.0</dt>

<dd></dd><dt>Acetyl-CoA: 7.0</dt>

<dd></dd><dt>Benzoyl-CoA: 0.0</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>










</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r5c3-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-192"></div> <span class="help-tooltip" id="tfbs-side-help-192">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/ZuR" target="_blank">ZuR</a></td>
<td>Zinc-responsive repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/LexA" target="_blank">LexA</a></td>
<td>Repressor of DNA damage response</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/AfsR" target="_blank">AfsR</a></td>
<td>Pleiotropic regulatory for antibiotic production</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>





<div class="page" id='r6c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000006.1 - Region 1 - hydrogen-cyanide <div class="help-container"> <div class="help-icon" data-id="region_svg-help-278"></div> <span class="help-tooltip" id="region_svg-help-278">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000006.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c1-svg" data-filename="JABTEZ010000006.1_r6c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 67,500 - 80,656 nt. (total: 13,157 nt)

<a class="cluster-rules-header" id="r6c1-rules-header" href="#r6c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c1-rules">hydrogen-cyanide: (Fer2_4 and cds(Pyr_redox_2 and Fer2_BFD) and DAO)</div>

<div class="region-svg-container">
<div id='r6c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-279"></div> <span class="help-tooltip" id="focus-panel-help-279">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c1-gene-table-details" data-name="r6c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c1-MIBiG-cluster-compare" data-name="r6c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c1-knownclusterblast" data-name="r6c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c1-subclusterblast" data-name="r6c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r6c1-tfbs-finder" data-name="r6c1-tfbs-finder">
TFBS Finder
</div>


</div>


<div class="body-details-section r6c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-201"></div> <span class="help-tooltip" id="gt-help-tooltip-help-201">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-196"></div> <span class="help-tooltip" id="MIBiG-help-196">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c1"></div>
<div id="comparison-MIBiG-r6c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">hydrogen-cyanide</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002285: 918-14445">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002285/index.html#r1c1" target="_new">BGC0002285</a></td>





<td class="cc-heat-cell" title="0.36 (id:0.58, order:0.44, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>


<td>0.10</td>
<td>RiPP</td>

<td>aborycin</td>


<td>Streptomyces sp. ZS0098</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001013: 0-116855">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001013/index.html#r1c1" target="_new">BGC0001013</a></td>





<td class="cc-heat-cell" title="0.31 (id:0.47, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>


<td>0.08</td>
<td>NRP, Polyketide</td>

<td>meridamycin</td>


<td>Streptomyces sp.</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000721: 1-42884">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000721/index.html#r1c1" target="_new">BGC0000721</a></td>





<td class="cc-heat-cell" title="0.30 (id:0.42, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>


<td>0.08</td>
<td>Saccharide</td>

<td>tobramycin</td>


<td>Streptoalloteichus tenebrarius</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.43, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>RiPP</td>

<td>SCO-2138</td>


<td>Streptomyces coelicolor A3(2)</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>





<td class="cc-heat-cell" title="0.25 (id:0.43, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>


<td>0.06</td>
<td>RiPP</td>

<td>SLI-2138</td>


<td>Streptomyces lividans TK24</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001011: 195-116831">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001011/index.html#r1c1" target="_new">BGC0001011</a></td>





<td class="cc-heat-cell" title="0.21 (id:0.47, order:N/A, components:0.09)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>


<td>0.04</td>
<td>NRP, Polyketide</td>

<td>meridamycin</td>


<td>Streptomyces sp. NRRL 30748</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000135: 0-104853">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000135/index.html#r1c1" target="_new">BGC0000135</a></td>





<td class="cc-heat-cell" title="0.18 (id:0.42, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>


<td>0.04</td>
<td>Polyketide</td>

<td>reveromycin A</td>


<td>Streptomyces sp. SN-593</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002285: 918-14445">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002285/index.html#r1c1" target="_new">BGC0002285</a></td>


<td class="cc-heat-cell" title="0.36 (id:0.58, order:0.44, components:0.19)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.36), rgb(0, 0, 0, 0.36) 36%, white 36%)"></td>
<td>0.36</td>
<td>RiPP</td>
<td>aborycin</td>
<td>Streptomyces sp. ZS0098</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001013: 0-116855">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001013/index.html#r1c1" target="_new">BGC0001013</a></td>


<td class="cc-heat-cell" title="0.31 (id:0.47, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.31), rgb(0, 0, 0, 0.31) 31%, white 31%)"></td>
<td>0.31</td>
<td>NRP, Polyketide</td>
<td>meridamycin</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000721: 1-42884">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000721/index.html#r1c1" target="_new">BGC0000721</a></td>


<td class="cc-heat-cell" title="0.30 (id:0.42, order:N/A, components:0.21)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.30), rgb(0, 0, 0, 0.30) 30%, white 30%)"></td>
<td>0.30</td>
<td>Saccharide</td>
<td>tobramycin</td>
<td>Streptoalloteichus tenebrarius</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000595: 0-21771">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000595/index.html#r1c1" target="_new">BGC0000595</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.43, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>RiPP</td>
<td>SCO-2138</td>
<td>Streptomyces coelicolor A3(2)</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000596: 0-21820">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000596/index.html#r1c1" target="_new">BGC0000596</a></td>


<td class="cc-heat-cell" title="0.25 (id:0.43, order:N/A, components:0.14)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.25), rgb(0, 0, 0, 0.25) 25%, white 25%)"></td>
<td>0.25</td>
<td>RiPP</td>
<td>SLI-2138</td>
<td>Streptomyces lividans TK24</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001011: 195-116831">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001011/index.html#r1c1" target="_new">BGC0001011</a></td>


<td class="cc-heat-cell" title="0.21 (id:0.47, order:N/A, components:0.09)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.21), rgb(0, 0, 0, 0.21) 21%, white 21%)"></td>
<td>0.21</td>
<td>NRP, Polyketide</td>
<td>meridamycin</td>
<td>Streptomyces sp. NRRL 30748</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000135: 0-104853">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000135/index.html#r1c1" target="_new">BGC0000135</a></td>


<td class="cc-heat-cell" title="0.18 (id:0.42, order:N/A, components:0.08)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.18), rgb(0, 0, 0, 0.18) 18%, white 18%)"></td>
<td>0.18</td>
<td>Polyketide</td>
<td>reveromycin A</td>
<td>Streptomyces sp. SN-593</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-197"></div> <span class="help-tooltip" id="cb-known-help-197">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r6c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r6c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r6c1_1.svg">
aborycin
</option>

</select>
<button id="knownclusterblast-r6c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r6c1-svg"></div>

</div>
</div>

<div class="body-details-section r6c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-198"></div> <span class="help-tooltip" id="cb-sub-help-198">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r6c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-200"></div> <span class="help-tooltip" id="tfbs-body-help-200">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
HypR
<div class="collapser collapser-target-HypR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: L-hydroxyproline utilization repressor<br>
<span>Score</span>: 18.59 out of 22.40
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r6c1-68198-HypR"></div>

<div class="tfbs-hit-detail">
HypR
<div class="collapser collapser-target-HypR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: L-hydroxyproline utilization repressor<br>
<span>Score</span>: 16.96 out of 22.40
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r6c1-72457-HypR"></div>

</div>
</div>
</div>

</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r6c1-tfbs-finder" data-name="r6c1-tfbs-finder">
TFBS Finder
</div>


</div>








<div class="sidepanel-details-section r6c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-199"></div> <span class="help-tooltip" id="tfbs-side-help-199">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/HypR" target="_blank">HypR</a></td>
<td>L-hydroxyproline utilization repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>


</div>

</div>

</div>

<div class="page" id='r6c2' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000006.1 - Region 2 - T3PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-280"></div> <span class="help-tooltip" id="region_svg-help-280">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000006.1.region002.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c2-svg" data-filename="JABTEZ010000006.1_r6c2.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 119,067 - 160,176 nt. (total: 41,110 nt)

<a class="cluster-rules-header" id="r6c2-rules-header" href="#r6c2">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r6c2-rules">T3PKS: (Chal_sti_synt_C or Chal_sti_synt_N)</div>

<div class="region-svg-container">
<div id='r6c2-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-281"></div> <span class="help-tooltip" id="focus-panel-help-281">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c2">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c2-gene-table-details" data-name="r6c2-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c2-nrps_pks" data-name="r6c2-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c2-MIBiG-cluster-compare" data-name="r6c2-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c2-knownclusterblast" data-name="r6c2-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c2-subclusterblast" data-name="r6c2-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c2-tfbs-finder" data-name="r6c2-tfbs-finder">
TFBS Finder
</div>


</div>


<div class="body-details-section r6c2-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-209"></div> <span class="help-tooltip" id="gt-help-tooltip-help-209">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c2">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c2-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c2-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c2">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c2-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-202"></div> <span class="help-tooltip" id="nrps-domain-help-202">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c2-details-svg" data-filename="JABTEZ010000006.1_r6c2_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c2-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c2-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-203"></div> <span class="help-tooltip" id="MIBiG-help-203">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c2-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c2-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c2"></div>
<div id="comparison-MIBiG-r6c2-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">T3PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282: 0-2613">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282</a></td>





<td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>


<td>0.58</td>
<td>Polyketide</td>

<td>alkylresorcinol</td>


<td>Streptomyces griseus subsp. griseus NBRC 13350</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001586: 0-19073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001586/index.html#r1c1" target="_new">BGC0001586</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.29</td>
<td>Saccharide</td>

<td>EPS273</td>


<td>Pseudomonas stutzeri</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002362: 1-24294">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002362/index.html#r1c1" target="_new">BGC0002362</a></td>





<td class="cc-heat-cell" title="0.62 (id:0.51, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>


<td>0.28</td>
<td>Polyketide</td>

<td>loseolamycin A1, loseolamycin A2</td>


<td>Micromonospora endolithica</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001077: 0-6550">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001077/index.html#r1c1" target="_new">BGC0001077</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.27</td>
<td>Polyketide</td>

<td>alkyl-O-dihydrogeranyl-methoxyhydroquinones</td>


<td>Actinoplanes missouriensis 431</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000917: 216-2477">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000917/index.html#r1c1" target="_new">BGC0000917</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>Other</td>

<td>molybdenum cofactor</td>


<td>Rhodobacter capsulatus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831: 1-6152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.50, order:0.43, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>Polyketide</td>

<td>alkylpyrone-407, alkylpyrone-393</td>


<td>Myxococcus xanthus DK 1622</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000723: 1114-36970">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000723/index.html#r1c1" target="_new">BGC0000723</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.53, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Other (Cyclitol), Saccharide</td>

<td>validamycin A</td>


<td>Streptomyces hygroscopicus subsp. limoneus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000722: 1394-44423">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000722/index.html#r1c1" target="_new">BGC0000722</a></td>





<td class="cc-heat-cell" title="0.54 (id:0.53, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>


<td>0.21</td>
<td>Other (Cyclitol), Saccharide</td>

<td>validamycin A</td>


<td>Streptomyces hygroscopicus subsp. jinggangensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000922: 0-13136">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000922/index.html#r1c1" target="_new">BGC0000922</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Other</td>

<td>pseudomonas quinolone signal (PQS)</td>


<td>Pseudomonas aeruginosa PAO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>





<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>


<td>0.20</td>
<td>RiPP</td>

<td>SWA-2138</td>


<td>Streptomyces sp. e14</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c2-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000282: 0-2613">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000282/index.html#r1c1" target="_new">BGC0000282</a></td>


<td class="cc-heat-cell" title="0.83 (id:0.57, order:1.00, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.83), rgb(0, 0, 0, 0.83) 83%, white 83%)"></td>
<td>0.83</td>
<td>Polyketide</td>
<td>alkylresorcinol</td>
<td>Streptomyces griseus subsp. griseus NBRC 13350</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001586: 0-19073">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001586/index.html#r1c1" target="_new">BGC0001586</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.46, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Saccharide</td>
<td>EPS273</td>
<td>Pseudomonas stutzeri</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002362: 1-24294">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002362/index.html#r1c1" target="_new">BGC0002362</a></td>


<td class="cc-heat-cell" title="0.62 (id:0.51, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.62), rgb(0, 0, 0, 0.62) 62%, white 62%)"></td>
<td>0.62</td>
<td>Polyketide</td>
<td>loseolamycin A1, loseolamycin A2</td>
<td>Micromonospora endolithica</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001077: 0-6550">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001077/index.html#r1c1" target="_new">BGC0001077</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.49, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Polyketide</td>
<td>alkyl-O-dihydrogeranyl-methoxyhydroquinones</td>
<td>Actinoplanes missouriensis 431</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000917: 216-2477">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000917/index.html#r1c1" target="_new">BGC0000917</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.47, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Other</td>
<td>molybdenum cofactor</td>
<td>Rhodobacter capsulatus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001831: 1-6152">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001831/index.html#r1c1" target="_new">BGC0001831</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.50, order:0.43, components:0.88)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>Polyketide</td>
<td>alkylpyrone-407, alkylpyrone-393</td>
<td>Myxococcus xanthus DK 1622</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000723: 1114-36970">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000723/index.html#r1c1" target="_new">BGC0000723</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.53, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Other (Cyclitol), Saccharide</td>
<td>validamycin A</td>
<td>Streptomyces hygroscopicus subsp. limoneus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000722: 1394-44423">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000722/index.html#r1c1" target="_new">BGC0000722</a></td>


<td class="cc-heat-cell" title="0.54 (id:0.53, order:N/A, components:0.55)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.54), rgb(0, 0, 0, 0.54) 54%, white 54%)"></td>
<td>0.54</td>
<td>Other (Cyclitol), Saccharide</td>
<td>validamycin A</td>
<td>Streptomyces hygroscopicus subsp. jinggangensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000922: 0-13136">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000922/index.html#r1c1" target="_new">BGC0000922</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>pseudomonas quinolone signal (PQS)</td>
<td>Pseudomonas aeruginosa PAO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000597: 0-10846">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000597/index.html#r1c1" target="_new">BGC0000597</a></td>


<td class="cc-heat-cell" title="0.52 (id:0.43, order:N/A, components:0.62)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.52), rgb(0, 0, 0, 0.52) 52%, white 52%)"></td>
<td>0.52</td>
<td>RiPP</td>
<td>SWA-2138</td>
<td>Streptomyces sp. e14</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c2-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-204"></div> <span class="help-tooltip" id="cb-known-help-204">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r6c2-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r6c2_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r6c2_1.svg">
alkylresorcinol
</option>

<option value = "svg/knownclusterblast_r6c2_2.svg">
lagunapyrone A/lagunapyrone B/lagunapyrone C
</option>

<option value = "svg/knownclusterblast_r6c2_3.svg">
alkylpyrone-407/alkylpyrone-393
</option>

<option value = "svg/knownclusterblast_r6c2_4.svg">
alkylpyrone-407/alkylpyrone-393
</option>

</select>
<button id="knownclusterblast-r6c2-download">Download graphic</button>
</div>
<div id="knownclusterblast-r6c2-svg"></div>

</div>
</div>

<div class="body-details-section r6c2-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-205"></div> <span class="help-tooltip" id="cb-sub-help-205">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c2-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-208"></div> <span class="help-tooltip" id="tfbs-body-help-208">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 28.15 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r6c2-147162-CelR"></div>

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 29.02 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r6c2-147163-CelR"></div>

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 28.65 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r6c2-147257-CelR"></div>

<div class="tfbs-hit-detail">
CelR
<div class="collapser collapser-target-CelR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Cellobiose uptake repressor<br>
<span>Score</span>: 28.15 out of 30.11
</div>
</div>
<div class="tfbs-confidence">
Confidence: strong
</div>
<div class="tfbs-svg-container" id="tfbs-r6c2-147258-CelR"></div>

</div>


</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c2-nrps_pks_monomers" data-name="r6c2-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r6c2-tfbs-finder" data-name="r6c2-tfbs-finder">
TFBS Finder
</div>


</div>










<div class="sidepanel-details-section r6c2-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-206"></div> <span class="help-tooltip" id="nrps-monomers-help-206">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">HS041_35185</span></strong>: X
<div class="collapser collapser-target-HS041_35185 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>



<dd><strong>AMP-binding (24..481)</strong>: X
<div class="collapser collapser-target-HS041_35185 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly, Ala, Val, Leu, Ile, Abu, Ival
<div class="collapser collapser-target-HS041_35185-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">IALFMAALLK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Trp <span class="serif">GALFFAAGVK</span> (29% 8Å match)</dd>
<dd>Leu <span class="serif">DALFMGAVAK</span> (21% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>60% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>


</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r6c2-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-207"></div> <span class="help-tooltip" id="tfbs-side-help-207">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CelR" target="_blank">CelR</a></td>
<td>Cellobiose uptake repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>


</div>

</div>

</div>

<div class="page" id='r6c3' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000006.1 - Region 3 - NRPS,T1PKS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-282"></div> <span class="help-tooltip" id="region_svg-help-282">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000006.1.region003.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r6c3-svg" data-filename="JABTEZ010000006.1_r6c3.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 417,280 - 449,361 nt. (total: 32,082 nt)

<a class="cluster-rules-header" id="r6c3-rules-header" href="#r6c3">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r6c3-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))<br>T1PKS: cds(PKS_AT and (PKS_KS or ene_KS or mod_KS or hyb_KS or itr_KS or tra_KS))</div>

<div class="region-svg-container">
<div id='r6c3-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-283"></div> <span class="help-tooltip" id="focus-panel-help-283">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r6c3">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r6c3-gene-table-details" data-name="r6c3-gene-table-details">
Gene overview
</div>



<div class="body-details-header r6c3-nrps_pks" data-name="r6c3-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r6c3-MIBiG-cluster-compare" data-name="r6c3-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r6c3-knownclusterblast" data-name="r6c3-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r6c3-subclusterblast" data-name="r6c3-subclusterblast">
SubClusterBlast
</div>





<div class="body-details-header r6c3-tfbs-finder" data-name="r6c3-tfbs-finder">
TFBS Finder
</div>



<div class="body-details-header r6c3-nrps-pks-bubbles" data-name="r6c3-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r6c3-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-219"></div> <span class="help-tooltip" id="gt-help-tooltip-help-219">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r6c3">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r6c3-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r6c3-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r6c3">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r6c3-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-210"></div> <span class="help-tooltip" id="nrps-domain-help-210">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c3-details-svg" data-filename="JABTEZ010000006.1_r6c3_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r6c3-details-svg"></div>
</div>
</div>



<div class="body-details-section r6c3-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-211"></div> <span class="help-tooltip" id="MIBiG-help-211">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r6c3-selector">Analysis type:</label>
<select id="comparison-MIBiG-r6c3-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r6c3"></div>
<div id="comparison-MIBiG-r6c3-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>

<th><span class="cc-heat-product">T1PKS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000016: 226-5727">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000016/index.html#r1c1" target="_new">BGC0000016</a></td>





<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>






<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>


<td>0.73</td>
<td>Other</td>

<td>amphotericin B</td>


<td>Streptomyces nodosus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.70</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.69</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 10E9</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>





<td class="cc-heat-cell" title="0.68 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>






<td class="cc-heat-cell" title="0.68 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>


<td>0.69</td>
<td>NRP, Polyketide</td>

<td>cylindrospermopsin</td>


<td>Aphanizomenon sp. 22D11</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002017: 0-12514">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002017/index.html#r1c1" target="_new">BGC0002017</a></td>





<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>






<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>


<td>0.69</td>
<td>NRP</td>

<td>potensibactin</td>


<td>Nocardiopsis potens DSM 45234</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>





<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>






<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>


<td>0.65</td>
<td>NRP</td>

<td>vicibactin</td>


<td>Rhizobium etli CFN 42</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.63</td>
<td>NRP</td>

<td>enterobactin</td>


<td>Pseudomonas sp. J465</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.63</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.63</td>
<td>NRP</td>

<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>


<td>Metarhizium robertsii ARSEF 23</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001925: 0-20488">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001925/index.html#r1c1" target="_new">BGC0001925</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>






<td class="cc-heat-cell" title="0.64 (id:0.45, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.64), rgb(0, 0, 0, 0.64) 64%, white 64%)"></td>


<td>0.62</td>
<td>Alkaloid</td>

<td>altemicidin, SB-203207, SB-203208</td>


<td>Streptomyces sp.</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r6c3-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000016: 226-5727">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000016/index.html#r1c1" target="_new">BGC0000016</a></td>


<td class="cc-heat-cell" title="0.69 (id:0.48, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.69), rgb(0, 0, 0, 0.69) 69%, white 69%)"></td>
<td>0.69</td>
<td>Other</td>
<td>amphotericin B</td>
<td>Streptomyces nodosus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.46, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000979: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000979/index.html#r1c1" target="_new">BGC0000979</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 10E9</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000980: 0-6921">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000980/index.html#r1c1" target="_new">BGC0000980</a></td>


<td class="cc-heat-cell" title="0.68 (id:0.49, order:N/A, components:0.93)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.68), rgb(0, 0, 0, 0.68) 68%, white 68%)"></td>
<td>0.68</td>
<td>NRP, Polyketide</td>
<td>cylindrospermopsin</td>
<td>Aphanizomenon sp. 22D11</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002017: 0-12514">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002017/index.html#r1c1" target="_new">BGC0002017</a></td>


<td class="cc-heat-cell" title="0.67 (id:0.45, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.67), rgb(0, 0, 0, 0.67) 67%, white 67%)"></td>
<td>0.67</td>
<td>NRP</td>
<td>potensibactin</td>
<td>Nocardiopsis potens DSM 45234</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000457: 0-11274">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000457/index.html#r1c1" target="_new">BGC0000457</a></td>


<td class="cc-heat-cell" title="0.66 (id:0.43, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.66), rgb(0, 0, 0, 0.66) 66%, white 66%)"></td>
<td>0.66</td>
<td>NRP</td>
<td>vicibactin</td>
<td>Rhizobium etli CFN 42</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001925: 0-20488">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001925/index.html#r1c1" target="_new">BGC0001925</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.45, order:N/A, components:0.95)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Alkaloid</td>
<td>altemicidin, SB-203207, SB-203208</td>
<td>Streptomyces sp.</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000343: 1-6208">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000343/index.html#r1c1" target="_new">BGC0000343</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>enterobactin</td>
<td>Pseudomonas sp. J465</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002710: 0-33332">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002710/index.html#r1c1" target="_new">BGC0002710</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>metachelin C, metachelin A, metachelin A-CE, metachelin B, dimerumic acid 11-mannoside, dimerumic acid</td>
<td>Metarhizium robertsii ARSEF 23</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r6c3-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-212"></div> <span class="help-tooltip" id="cb-known-help-212">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r6c3-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r6c3_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r6c3_1.svg">
microtermolide A
</option>

<option value = "svg/knownclusterblast_r6c3_2.svg">
crochelin A
</option>

<option value = "svg/knownclusterblast_r6c3_3.svg">
glidobactin/glidobactin B/glidobactin C
</option>

<option value = "svg/knownclusterblast_r6c3_4.svg">
fabrubactin A/fabrubactin B
</option>

<option value = "svg/knownclusterblast_r6c3_5.svg">
didemnin B/nordidemnin B/dehydrodidemnin B (aplidine)/didemnin X/didemnin Y
</option>

<option value = "svg/knownclusterblast_r6c3_6.svg">
griseoviridin/fijimycin A
</option>

<option value = "svg/knownclusterblast_r6c3_7.svg">
lipopeptide 8D1-1/lipopeptide 8D1-2
</option>

<option value = "svg/knownclusterblast_r6c3_8.svg">
petrichorin A/petrichorin B
</option>

<option value = "svg/knownclusterblast_r6c3_9.svg">
nargenicin
</option>

<option value = "svg/knownclusterblast_r6c3_10.svg">
nargenicin A1
</option>

</select>
<button id="knownclusterblast-r6c3-download">Download graphic</button>
</div>
<div id="knownclusterblast-r6c3-svg"></div>

</div>
</div>

<div class="body-details-section r6c3-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-213"></div> <span class="help-tooltip" id="cb-sub-help-213">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>





<div class="body-details-section r6c3-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-217"></div> <span class="help-tooltip" id="tfbs-body-help-217">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<span class="tfbs-no-hits">No hits above weak threshold.</span>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Weak hits (2):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
BldD
<div class="collapser collapser-target-BldD collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Development and antibiotic global regulator<br>
<span>Score</span>: 18.83 out of 23.43
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r6c3-418280-BldD"></div>

<div class="tfbs-hit-detail">
DmdR1
<div class="collapser collapser-target-DmdR1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Iron(II)-dependent repressor<br>
<span>Score</span>: 21.70 out of 36.53
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r6c3-419484-DmdR1"></div>

</div>
</div>
</div>

</div>
</div>



<div class="body-details-section r6c3-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-218"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-218">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r6c3-domain-bubble-svg" data-filename="JABTEZ010000006.1_r6c3_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r6c3-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC3">
Candidate 3 (417279 - 449361): interleaved NRPS-T1PKS
</option>


</select>

<div class="details-svg" id="r6c3-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r6c3-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r6c3-nrps_pks_products" data-name="r6c3-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r6c3-nrps_pks_monomers" data-name="r6c3-nrps_pks_monomers">
NRPS/PKS substrates
</div>



<div class="sidepanel-details-header r6c3-tfbs-finder" data-name="r6c3-tfbs-finder">
TFBS Finder
</div>




</div>










<div class="sidepanel-details-section r6c3-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-214"></div> <span class="help-tooltip" id="nrps-structures-help-214">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 3, location 417279 - 449361:
<div class="collapser collapser-target-CC3 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r6c3-3" class="smiles-canvas" data-smiles="NCC(=O)NC([*])C(=O)CC(=O)NC([*])C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Gly - X) + (mal) + (X)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*,x&amp;nrps3=x" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r6c3-3-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NCC(=O)NC([*])C(=O)CC(=O)NC([*])C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r6c3-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-215"></div> <span class="help-tooltip" id="nrps-monomers-help-215">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">HS041_36620</span></strong>: X
<div class="collapser collapser-target-HS041_36620 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>AMP-binding (494..886)</strong>: X
<div class="collapser collapser-target-HS041_36620 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Glu, Gln
<div class="collapser collapser-target-HS041_36620-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophilic (Arg, Asp, Glu, Asn, Lys, Gln, Orn, Aad)</dd>
<dt>Large clusters prediction:</dt>
<dd>Aliphatic chain with H-bond donor (Asp, Asn, Glu, Gln, Aad)</dd>
<dt>Small clusters prediction:</dt>
<dd>glu,gln (Glu, Gln)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DPWDFGQIDK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gln <span class="serif">DAWQFGLIDK</span> (68% 8Å match)</dd>
<dd>ohdmTyr <span class="serif">DPWGLGLIDK</span> (65% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>70% (weak)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>






</div>
<br>

<strong><span class="serif">HS041_36625</span></strong>: mal
<div class="collapser collapser-target-HS041_36625 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>





<dd><strong>PKS_AT (521..798)</strong>: mal
<div class="collapser collapser-target-HS041_36625 collapser-level-candidate"> </div><div class="collapser-content">

ATSignature:  Malonyl-CoA
<div class="collapser collapser-target-HS041_36625-ATSignature collapser-level-none"> </div><div class="collapser-content">
<dl>
<dt>Top 3 matches:</dt>
<dd>Malonyl-CoA: 95.8%</dd>
<dd>inactive: 79.2%</dd>
<dd>2-Rhyd-Malonyl-CoA: 54.2%</dd>
</dl>

</div><br>

minowa:  Malonyl-CoA
<div class="collapser collapser-target-HS041_36625-minowa collapser-level-none"> </div><div class="collapser-content">

<dl><dt>Prediction, score:</dt>
<dd>
<dl>
<dd></dd><dt>Malonyl-CoA: 174.5</dt>

<dd></dd><dt>inactive: 103.4</dt>

<dd></dd><dt>Methoxymalonyl-CoA: 61.6</dt>

<dd></dd><dt>Methylmalonyl-CoA: 54.7</dt>

<dd></dd><dt>Propionyl-CoA: 37.4</dt>

<dd></dd><dt>CHC-CoA: 30.7</dt>

<dd></dd><dt>Benzoyl-CoA: 22.6</dt>

<dd></dd><dt>Isobutyryl-CoA: 20.3</dt>

<dd></dd><dt>Ethylmalonyl-CoA: 19.0</dt>

<dd></dd><dt>trans-1,2-CPDA: 16.1</dt>

<dd></dd><dt>2-Methylbutyryl-CoA: 14.8</dt>

<dd></dd><dt>Acetyl-CoA: 12.5</dt>

<dd></dd><dt>fatty_acid: 9.4</dt>

<dd></dd><dt>3-Methylbutyryl-CoA: 0.0</dt>

</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

<strong><span class="serif">HS041_36630</span></strong>: Gly - X
<div class="collapser collapser-target-HS041_36630 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=gly*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (7..265)</strong>: Gly
<div class="collapser collapser-target-HS041_36630 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Gly
<div class="collapser collapser-target-HS041_36630-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>N/A</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>Tiny, hydrophilic, transition to aliphatic (Gly, Ala)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DILQMGGVYK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Gly <span class="serif">DILQMGMVWK</span> (68% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>







<dd><strong>AMP-binding (921..1375)</strong>: X
<div class="collapser collapser-target-HS041_36630 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  (unknown)
<div class="collapser collapser-target-HS041_36630-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>Apolar, aliphatic (Gly, Ala, Val, Leu, Ile, Abu, Ival)</dd>
<dt>Small clusters prediction:</dt>
<dd>N/A</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DTAVVAEVWK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>His <span class="serif">DSALVAEVWK</span> (76% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>



<div class="sidepanel-details-section r6c3-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-216"></div> <span class="help-tooltip" id="tfbs-side-help-216">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/BldD" target="_blank">BldD</a></td>
<td>Development and antibiotic global regulator</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DmdR1" target="_blank">DmdR1</a></td>
<td>Iron(II)-dependent repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>




</div>

</div>

</div>





<div class="page" id='r8c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000008.1 - Region 1 - NI-siderophore <div class="help-container"> <div class="help-icon" data-id="region_svg-help-284"></div> <span class="help-tooltip" id="region_svg-help-284">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000008.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r8c1-svg" data-filename="JABTEZ010000008.1_r8c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 10,930 - 42,204 nt. (total: 31,275 nt)

<a class="cluster-rules-header" id="r8c1-rules-header" href="#r8c1">Show pHMM detection rules used</a>


</div>

<div class="cluster-rules" id="r8c1-rules">NI-siderophore: IucA_IucC</div>

<div class="region-svg-container">
<div id='r8c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-285"></div> <span class="help-tooltip" id="focus-panel-help-285">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r8c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r8c1-gene-table-details" data-name="r8c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r8c1-MIBiG-cluster-compare" data-name="r8c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r8c1-knownclusterblast" data-name="r8c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r8c1-subclusterblast" data-name="r8c1-subclusterblast">
SubClusterBlast
</div>



<div class="body-details-header r8c1-tfbs-finder" data-name="r8c1-tfbs-finder">
TFBS Finder
</div>


</div>


<div class="body-details-section r8c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-225"></div> <span class="help-tooltip" id="gt-help-tooltip-help-225">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r8c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r8c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r8c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r8c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r8c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-220"></div> <span class="help-tooltip" id="MIBiG-help-220">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r8c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r8c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r8c1"></div>
<div id="comparison-MIBiG-r8c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NI-siderophore</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002692: 1491-11626">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002692/index.html#r1c1" target="_new">BGC0002692</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.32</td>
<td>Other</td>

<td>speibonoxamine, desoxy-desferrioxamine D1, desferrioxamine D1, desferrioxamine B, didesoxy-desferrioxamine D1, desoxy-desferrioxamine B, didesoxy-desferrioxamine B</td>


<td>Kribbella speibonae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002620: 0-7797">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002620/index.html#r1c1" target="_new">BGC0002620</a></td>





<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>


<td>0.28</td>
<td>Other</td>

<td>fulvivirgamide A2, fulvivirgamide B2, fulvivirgamide B3, fulvivirgamide B4</td>


<td>Fulvivirga marina</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000944: 0-6093">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000944/index.html#r1c1" target="_new">BGC0000944</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Other (Non-NRP siderophore)</td>

<td>staphyloferrin A</td>


<td>Staphylococcus aureus subsp. aureus NCTC 8325</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713: 164-10986">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.25</td>
<td>Other</td>

<td>ochrobactin</td>


<td>Ochrobactrum anthropi</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002633: 0-11574">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002633/index.html#r1c1" target="_new">BGC0002633</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>Other</td>

<td>schizokinen</td>


<td>Leptolyngbya sp. PCC 7376</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002464: 0-10571">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002464/index.html#r1c1" target="_new">BGC0002464</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>alcaligin</td>


<td>Bordetella pertussis Tohama I</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000946: 282-7531">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000946/index.html#r1c1" target="_new">BGC0000946</a></td>





<td class="cc-heat-cell" title="0.56 (id:0.45, order:0.51, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>


<td>0.23</td>
<td>Other</td>

<td>vibrioferrin</td>


<td>Vibrio parahaemolyticus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000939: 1834-15938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000939/index.html#r1c1" target="_new">BGC0000939</a></td>





<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>


<td>0.22</td>
<td>Other</td>

<td>aerobactin</td>


<td>Grimontia hollisae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000387: 251-12509">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000387/index.html#r1c1" target="_new">BGC0000387</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>NRP</td>

<td>mangotoxin</td>


<td>Pseudomonas syringae pv. syringae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002683: 0-14011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002683/index.html#r1c1" target="_new">BGC0002683</a></td>





<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>


<td>0.20</td>
<td>Other</td>

<td>schizokinen</td>


<td>Nostoc sp. PCC 7120 = FACHB-418</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r8c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002692: 1491-11626">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002692/index.html#r1c1" target="_new">BGC0002692</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>speibonoxamine, desoxy-desferrioxamine D1, desferrioxamine D1, desferrioxamine B, didesoxy-desferrioxamine D1, desoxy-desferrioxamine B, didesoxy-desferrioxamine B</td>
<td>Kribbella speibonae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002620: 0-7797">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002620/index.html#r1c1" target="_new">BGC0002620</a></td>


<td class="cc-heat-cell" title="0.61 (id:0.42, order:N/A, components:0.90)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.61), rgb(0, 0, 0, 0.61) 61%, white 61%)"></td>
<td>0.61</td>
<td>Other</td>
<td>fulvivirgamide A2, fulvivirgamide B2, fulvivirgamide B3, fulvivirgamide B4</td>
<td>Fulvivirga marina</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000944: 0-6093">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000944/index.html#r1c1" target="_new">BGC0000944</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.42, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Other (Non-NRP siderophore)</td>
<td>staphyloferrin A</td>
<td>Staphylococcus aureus subsp. aureus NCTC 8325</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001713: 164-10986">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001713/index.html#r1c1" target="_new">BGC0001713</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.41, order:N/A, components:0.83)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>Other</td>
<td>ochrobactin</td>
<td>Ochrobactrum anthropi</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002633: 0-11574">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002633/index.html#r1c1" target="_new">BGC0002633</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.42, order:N/A, components:0.81)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>Other</td>
<td>schizokinen</td>
<td>Leptolyngbya sp. PCC 7376</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002464: 0-10571">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002464/index.html#r1c1" target="_new">BGC0002464</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>alcaligin</td>
<td>Bordetella pertussis Tohama I</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000946: 282-7531">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000946/index.html#r1c1" target="_new">BGC0000946</a></td>


<td class="cc-heat-cell" title="0.56 (id:0.45, order:0.51, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.56), rgb(0, 0, 0, 0.56) 56%, white 56%)"></td>
<td>0.56</td>
<td>Other</td>
<td>vibrioferrin</td>
<td>Vibrio parahaemolyticus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000939: 1834-15938">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000939/index.html#r1c1" target="_new">BGC0000939</a></td>


<td class="cc-heat-cell" title="0.55 (id:0.41, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.55), rgb(0, 0, 0, 0.55) 55%, white 55%)"></td>
<td>0.55</td>
<td>Other</td>
<td>aerobactin</td>
<td>Grimontia hollisae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000387: 251-12509">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000387/index.html#r1c1" target="_new">BGC0000387</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>NRP</td>
<td>mangotoxin</td>
<td>Pseudomonas syringae pv. syringae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002683: 0-14011">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002683/index.html#r1c1" target="_new">BGC0002683</a></td>


<td class="cc-heat-cell" title="0.53 (id:0.42, order:N/A, components:0.67)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.53), rgb(0, 0, 0, 0.53) 53%, white 53%)"></td>
<td>0.53</td>
<td>Other</td>
<td>schizokinen</td>
<td>Nostoc sp. PCC 7120 = FACHB-418</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r8c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-221"></div> <span class="help-tooltip" id="cb-known-help-221">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<select id="knownclusterblast-r8c1-select" class="clusterblast-selector">
<option value="svg/knownclusterblast_r8c1_all.svg">
All hits
</option>

<option value = "svg/knownclusterblast_r8c1_1.svg">
peucechelin
</option>

</select>
<button id="knownclusterblast-r8c1-download">Download graphic</button>
</div>
<div id="knownclusterblast-r8c1-svg"></div>

</div>
</div>

<div class="body-details-section r8c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-222"></div> <span class="help-tooltip" id="cb-sub-help-222">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>



<div class="body-details-section r8c1-tfbs-finder">
<div class="details tfbs-details-container">
<div class="heading">
<span>Transcription Factor Binding Sites</span> <div class="help-container"> <div class="help-icon" data-id="tfbs-body-help-224"></div> <span class="help-tooltip" id="tfbs-body-help-224">Detailed information for Transcription Factor Binding Site hits, including surrounding genes where applicable. <br>The strand of the match is shown by an arrow underneath the match.<br>Detailed documentation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/tfbs' target='_blank'>here</a>.</span></div>
</div>

<div class="tfbs-section-header">Binding sites found:</div>
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
DmdR1
<div class="collapser collapser-target-DmdR1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Iron(II)-dependent repressor<br>
<span>Score</span>: 30.62 out of 36.53
</div>
</div>
<div class="tfbs-confidence">
Confidence: medium
</div>
<div class="tfbs-svg-container" id="tfbs-r8c1-23737-DmdR1"></div>

</div>


<div class="tfbs-weak-hits"><span class="tfbs-section-header">Additional weak hits (3):</span>
<div class="collapser collapser-target-weak collapser-level-cds"> </div><div class="collapser-content">
<div class="tfbs-hit-list">

<div class="tfbs-hit-detail">
NrtR
<div class="collapser collapser-target-NrtR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: NAD synthesis repressor<br>
<span>Score</span>: 19.23 out of 35.63
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r8c1-17261-NrtR"></div>

<div class="tfbs-hit-detail">
CatR
<div class="collapser collapser-target-CatR collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: H2O2-responsive repressor<br>
<span>Score</span>: 17.99 out of 25.31
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r8c1-22462-CatR"></div>

<div class="tfbs-hit-detail">
DmdR1
<div class="collapser collapser-target-DmdR1 collapser-level-cds"> </div><div class="collapser-content">
<span>Description</span>: Iron(II)-dependent repressor<br>
<span>Score</span>: 21.56 out of 36.53
</div>
</div>
<div class="tfbs-confidence">
Confidence: weak
</div>
<div class="tfbs-svg-container" id="tfbs-r8c1-23732-DmdR1"></div>

</div>
</div>
</div>

</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">








<div class="sidepanel-details-header r8c1-tfbs-finder" data-name="r8c1-tfbs-finder">
TFBS Finder
</div>


</div>








<div class="sidepanel-details-section r8c1-tfbs-finder">
<div class="tfbs-sidepanel">
<div class="heading">
<span>TFBS definitions</span>
<div class="help-container"> <div class="help-icon" data-id="tfbs-side-help-223"></div> <span class="help-tooltip" id="tfbs-side-help-223">Shows descriptions for Transcription Factor Binding Site models</span></div>
</div>
<div class = 'prediction-text'>
<table class="tfbs-sidepanel-table">
<thead>
<tr>
<th>Regulator</th>
<th>Description</th>
</tr>
</thead>
<tbody>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/NrtR" target="_blank">NrtR</a></td>
<td>NAD synthesis repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/DmdR1" target="_blank">DmdR1</a></td>
<td>Iron(II)-dependent repressor</td>
</tr>

<tr>
<td><a href="https://logomotif.bioinformatics.nl/regulator/CatR" target="_blank">CatR</a></td>
<td>H2O2-responsive repressor</td>
</tr>

</tbody>
</table>
</div>
</div>
</div>


</div>

</div>

</div>





<div class="page" id='r9c1' style="display: none;">
<div class="region-grid">
<div class = "content">
<div class ="description-container">
<div class="heading"> JABTEZ010000009.1 - Region 1 - NRPS <div class="help-container"> <div class="help-icon" data-id="region_svg-help-286"></div> <span class="help-tooltip" id="region_svg-help-286">Shows the layout of the region, marking coding sequences and areas of interest. Clicking a gene will select it and show any relevant details. Clicking an area feature (e.g. a candidate cluster) will select all coding sequences within that area. Double clicking an area feature will zoom to that area. Multiple genes and area features can be selected by clicking them while holding the Ctrl key.<br>More detailed help is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/understanding_output/#the-antismash-5-region-concept' target='_blank'>here</a>.</span></div></div>
<div class="region-download">
<a href = JABTEZ010000009.1.region001.gbk>Download region GenBank file</a>
</div>
<div class="region-download">
<div class="link-like download-svg" data-tag="r9c1-svg" data-filename="JABTEZ010000009.1_r9c1.svg">Download region SVG</div>
</div>
<div class = 'description-text'>
Location: 1 - 2,803 nt. (total: 2,803 nt)

<a class="cluster-rules-header" id="r9c1-rules-header" href="#r9c1">Show pHMM detection rules used</a>


<div class="contig-edge-warning">Region on contig edge.</div>

</div>

<div class="cluster-rules" id="r9c1-rules">NRPS: cds(Condensation and (AMP-binding or A-OX))</div>

<div class="region-svg-container">
<div id='r9c1-svg'>
</div>
</div>
</div>
<div class="legend">
<h4>Legend:</h4>
<div class="legend-items">
<div class="button-like legend-selector" data-id="legend-type-biosynthetic">
<div class="legend-field legend-type-biosynthetic"></div>
<div class="legend-label">core biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-biosynthetic-additional">
<div class="legend-field legend-type-biosynthetic-additional"></div>
<div class="legend-label">additional biosynthetic genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-transport">
<div class="legend-field legend-type-transport"></div>
<div class="legend-label">transport-related genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-regulatory">
<div class="legend-field legend-type-regulatory"></div>
<div class="legend-label">regulatory genes</div>
</div>
<div class="button-like legend-selector" data-id="legend-type-other">
<div class="legend-field legend-type-other"></div>
<div class="legend-label">other genes</div>
</div>
<div style="margin-right: 2em;"></div>
<div class="button-like legend-selector" data-id="legend-resistance">
<div class="legend-symbol">
<svg viewbox="0 0 8 8">
<rect x=0 y=2 height=4 width=8 class="svgene-resistance"></rect>
</svg>
</div>
<div class="legend-label">resistance</div>
</div>

<div class="button-like legend-selector" data-id="legend-tta-codon">
<div class="legend-symbol">
<svg viewbox="0 0 6 6">
<polyline class="svgene-tta-codon" points="3,0 0,6 6,6 3,0">
</svg>
</div>
<div class="legend-label">TTA codons</div>
</div>


<div class="legend-selector-static">
<div class="legend-symbol">
<svg viewbox="0 -1 8 8">
<g class="svgene-binding-site">
<line x1="4" y1="8" x2="4" y1="4"></line>
<circle cx="4" cy="2" r="2"></line>
</svg>
</div>
<div class="legend-label">binding site</div>
</div>


</div>
</div>
<div class="zoom-buttons">
<div class="button-like zoom-button zoom-reset"><img src="images/expand-arrows-alt-solid.svg" alt="reset zoom">reset view</div>
<div class="button-like zoom-button zoom-in"><img src="images/search-solid.svg" alt="zoom">zoom to selection</div>
</div>
</div>

<div class="focus-panel">
<div class="heading">
<span>Gene details</span>
<div class="help-container"> <div class="help-icon" data-id="focus-panel-help-287"></div> <span class="help-tooltip" id="focus-panel-help-287">Shows details of the most recently selected gene, including names, products, location, and other annotations.</span></div>
</div>
<div class="focus-panel-content focus-panel-content-r9c1">
<div style="text-align: center; margin-top: 30%;">
Select a gene to view the details available for it
</div>
</div>
</div>


<div class="body-details">
<div class="body-details-headers">


<div class="body-details-header r9c1-gene-table-details" data-name="r9c1-gene-table-details">
Gene overview
</div>



<div class="body-details-header r9c1-nrps_pks" data-name="r9c1-nrps_pks">
NRPS/PKS domains
</div>



<div class="body-details-header r9c1-MIBiG-cluster-compare" data-name="r9c1-MIBiG-cluster-compare">
MIBiG comparison
</div>



<div class="body-details-header r9c1-knownclusterblast" data-name="r9c1-knownclusterblast">
KnownClusterBlast
</div>

<div class="body-details-header r9c1-subclusterblast" data-name="r9c1-subclusterblast">
SubClusterBlast
</div>







<div class="body-details-header r9c1-nrps-pks-bubbles" data-name="r9c1-nrps-pks-bubbles">
NRPS/PKS modules
</div>


</div>


<div class="body-details-section r9c1-gene-table-details">
<div class="details">
<div class="heading">
<span>Gene/CDS overview</span>
<div class="help-container"> <div class="help-icon" data-id="gt-help-tooltip-help-233"></div> <span class="help-tooltip" id="gt-help-tooltip-help-233">A brief tabular summary of genes/CDS features within the region.<br>Filtering the table will also search biosynthetic profiles and gene function data. If enabled, the overview will then zoom to show the area covered by the filtered selection.<br>Genes selected in the region drawing above will be marked in the table with an indicator to the left of the gene name.</span></div>
</div>
<div class="gt-container" id="gt-container-r9c1">
<div class="gt-header">
<div class="gt-filter">
Filter: <input type="search" id="gt-r9c1-search" class="search-field gt-search">
</div>
<div class="gt-zoom-toggle switch-container"><span class="switch-desc">Automatically zoom to filtered/selected features</span> <label class="switch">  <input class="gt-zoom-toggle" type="checkbox" id="gt-r9c1-toggle" checked>  <span class="slider"></span> </label></div>
</div>
<div class="gt-scroll-container">
<table class="gt" id="gt-r9c1">
<thead>
<tr>
<th>Identifier</th>
<th>Product</th>
<th colspan="2">Length</th>
<th>Function</th>
<th colspan="2">Sequence</th>
<th>NCBI Blast</th>
<th class="gt-column-match" style="display:none">Filter details</th>
</tr>
<tr>
<th></th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th>NT</th>
<th>AA</th>
<th></th>
<th class="gt-column-match" style="display:none"></th>
</tr>
</thead>
<tbody>
</tbody>
</table>
</div>
<div class="gt-no-hits" style="display:none">No genes match the given filter</div>
</div>
</div>
</div>



<div class="body-details-section r9c1-nrps_pks">
<div class="details">
<div class="heading">
<span>Detailed domain annotation</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-domain-help-226"></div> <span class="help-tooltip" id="nrps-domain-help-226">Shows <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a>- and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a>-related domains for each feature that contains them. Click on each domain for more information about the domain's location, consensus monomer prediction, and other details.<br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/' target='_blank'>here</a>.</span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r9c1-details-svg" data-filename="JABTEZ010000009.1_r9c1_nrps_pks.svg">
</div>
</div>
</div>
<div class="nrps-pks-domain-buttons">
<div class="domains-selected-only switch-container"><span class="switch-desc">Selected features only</span> <label class="switch">  <input class="domains-selected-only" type="checkbox">  <span class="slider"></span> </label></div>
<div class="show-module-domains switch-container"><span class="switch-desc">Show module domains</span> <label class="switch">  <input class="show-module-domains" type="checkbox">  <span class="slider"></span> </label></div>
</div>
<div class="details-svg" id="r9c1-details-svg"></div>
</div>
</div>



<div class="body-details-section r9c1-MIBiG-cluster-compare">
<div class="comparison-MIBiG">
<div class="heading">
<span>Similar gene clusters</span>
<div class="help-container"> <div class="help-icon" data-id="MIBiG-help-227"></div> <span class="help-tooltip" id="MIBiG-help-227">Shows areas that are similar to the current region to a reference database.<br>Mouseover a score cell in the table to get a breakdown of how the score was calculated.The MIBiG database.<br><br>Click on an accession to open that entry in the MIBiG database.</span></div>
</div>

<label for="comparison-MIBiG-r9c1-selector">Analysis type:</label>
<select id="comparison-MIBiG-r9c1-selector" class="comparison-selector" data-tag="MIBiG">
<option value="ProtoToRegion_RiQ">Protocluster to Region</option>

<option value="RegionToRegion_RiQ">Region to Region</option>
</select>
<div id="comparison-MIBiG-r9c1"></div>
<div id="comparison-MIBiG-r9c1-ProtoToRegion_RiQ" class="comparison-container comparison-container-active">

<table class="cc-heat-table">
<tr>
<th>Reference</th>


<th><span class="cc-heat-product">NRPS</span></th>


<th>Similarity score</th>
<th>Type</th>

<th>Compound(s)</th>


<th>Organism</th>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>Other</td>

<td>ferrichrome</td>


<td>Aspergillus oryzae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>





<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>


<td>0.31</td>
<td>NRP</td>

<td>dimethylcoprogen</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>





<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>


<td>0.26</td>
<td>NRP</td>

<td>indigoidine</td>


<td>Streptomyces chromofuscus</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>gamexpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>xenotetrapeptide</td>


<td>Xenorhabdus nematophila ATCC 19061</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.25</td>
<td>NRP</td>

<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>


<td>Photorhabdus laumondii subsp. laumondii TTO1</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>





<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>bicornutin A1, bicornutin A2</td>


<td>Xenorhabdus budapestensis</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.24</td>
<td>NRP</td>

<td>peramine</td>


<td>Epichloe festucae</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>AM-toxin</td>


<td>Alternaria alternata</td>

</tr>

<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>





<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>


<td>0.23</td>
<td>NRP</td>

<td>peramine, intermediate 1, intermediate 2</td>


<td>Metarhizium rileyi</td>

</tr>

</table>

</div>

<div id="comparison-MIBiG-r9c1-RegionToRegion_RiQ" class="comparison-container ">

<table class="cc-heat-table">
<tr>
<th>Reference</th>
<th><span class="cc-heat-product">Aggregated</span></th>
<th>Similarity score</th>
<th>Type</th>
<th>Compound(s)</th>
<th>Organism</th>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000900: 2044-8911">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000900/index.html#r1c1" target="_new">BGC0000900</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>Other</td>
<td>ferrichrome</td>
<td>Aspergillus oryzae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001249: 663-5994">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001249/index.html#r1c1" target="_new">BGC0001249</a></td>


<td class="cc-heat-cell" title="0.65 (id:0.42, order:N/A, components:1.00)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.65), rgb(0, 0, 0, 0.65) 65%, white 65%)"></td>
<td>0.65</td>
<td>NRP</td>
<td>dimethylcoprogen</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0000375: 0-9457">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0000375/index.html#r1c1" target="_new">BGC0000375</a></td>


<td class="cc-heat-cell" title="0.59 (id:0.45, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.59), rgb(0, 0, 0, 0.59) 59%, white 59%)"></td>
<td>0.59</td>
<td>NRP</td>
<td>indigoidine</td>
<td>Streptomyces chromofuscus</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001128: 35-15686">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001128/index.html#r1c1" target="_new">BGC0001128</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>gamexpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001132: 0-12417">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001132/index.html#r1c1" target="_new">BGC0001132</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>xenotetrapeptide</td>
<td>Xenorhabdus nematophila ATCC 19061</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002286: 0-16374">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002286/index.html#r1c1" target="_new">BGC0002286</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.46, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>ririwpeptide A, ririwpeptide B, ririwpeptide C</td>
<td>Photorhabdus laumondii subsp. laumondii TTO1</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001135: 0-16848">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001135/index.html#r1c1" target="_new">BGC0001135</a></td>


<td class="cc-heat-cell" title="0.58 (id:0.45, order:N/A, components:0.73)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.58), rgb(0, 0, 0, 0.58) 58%, white 58%)"></td>
<td>0.58</td>
<td>NRP</td>
<td>bicornutin A1, bicornutin A2</td>
<td>Xenorhabdus budapestensis</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002164: 0-8322">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002164/index.html#r1c1" target="_new">BGC0002164</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>peramine</td>
<td>Epichloe festucae</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0001261: 987-14070">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0001261/index.html#r1c1" target="_new">BGC0001261</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.41, order:N/A, components:0.78)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>AM-toxin</td>
<td>Alternaria alternata</td>
</tr>


<tr class="cc-heat-row heat-row-MIBiG" data-accession="BGC0002166: 0-8397">
<td><a href="https://mibig.secondarymetabolites.org/repository/BGC0002166/index.html#r1c1" target="_new">BGC0002166</a></td>


<td class="cc-heat-cell" title="0.57 (id:0.43, order:N/A, components:0.75)" style="background-image: linear-gradient(to top, rgb(0, 0, 0, 0.57), rgb(0, 0, 0, 0.57) 57%, white 57%)"></td>
<td>0.57</td>
<td>NRP</td>
<td>peramine, intermediate 1, intermediate 2</td>
<td>Metarhizium rileyi</td>
</tr>

</table>

</div>

</div>
</div>



<div class="body-details-section r9c1-knownclusterblast">
<div class = "knownclusterblast">
<div class="heading">
<span>Similar known gene clusters from MIBiG 3.1</span>
<div class="help-container"> <div class="help-icon" data-id="cb-known-help-228"></div> <span class="help-tooltip" id="cb-known-help-228">Shows clusters from the MiBIG database that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.<br>Click on an accession to open that entry in the MiBIG database.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>

<div class="body-details-section r9c1-subclusterblast">
<div class = "subclusterblast">
<div class="heading">
<span>Similar subclusters</span>
<div class="help-container"> <div class="help-icon" data-id="cb-sub-help-229"></div> <span class="help-tooltip" id="cb-sub-help-229">Shows sub-cluster units that are similar to the current region. Genes marked with the same colour are interrelated. White genes have no relationship.<br>Click on reference genes to show details of similarities to genes within the current region.</span></div>
</div>

<div>
<p>No matches found.</p>
</div>

</div>
</div>







<div class="body-details-section r9c1-nrps-pks-bubbles">
<div class="details">
<div class="heading">
<span>Module view</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-pks-bubble-tooltip-help-232"></div> <span class="help-tooltip" id="nrps-pks-bubble-tooltip-help-232">Shows module structures for each candidate cluster in <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#nrps' target='_blank'>NRPS</a> and <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/glossary/#t1pks' target='_blank'>PKS</a> regions. <br>Genes are shown in predicted order, and are only present when containing at least one complete module. <br>A domain glossary is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_domains/' target='_blank'>here</a>, and an explanation of the visualisation is available <a class='external-link' href='https://docs.antismash.secondarymetabolites.org/modules/nrps_pks_modules/#bubblemodule-only-drawing' target='_blank'>here</a>. </span></div>
<div class="download-container">
<div class="download-icon download-svg" data-tag="r9c1-domain-bubble-svg" data-filename="JABTEZ010000009.1_r9c1_domain_bubbles.svg">
</div>
</div>
</div>
<select id="r9c1-domain-bubble-select" class="nrps-bubble-selector">


<option value="CC1">
Candidate 1 (0 - 2803): single NRPS
</option>


</select>

<div class="details-svg" id="r9c1-domain-bubble-svg-container" style="overflow:auto;">
</div>
<div class="bubble-legend-title">Legend</div>

<div class="bubble-legend-container" id="r9c1-bubble-legend"></div>
</div>
</div>


</div>

<div class="sidepanel">
<div class="sidepanel-details-headers">










<div class="sidepanel-details-header r9c1-nrps_pks_products" data-name="r9c1-nrps_pks_products">
NRPS/PKS products
</div>

<div class="sidepanel-details-header r9c1-nrps_pks_monomers" data-name="r9c1-nrps_pks_monomers">
NRPS/PKS substrates
</div>






</div>










<div class="sidepanel-details-section r9c1-nrps_pks_products">
<div class="structure">

<div class="heading">
<span>Predicted core structure(s)</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-structures-help-230"></div> <span class="help-tooltip" id="nrps-structures-help-230">Shows estimated product structure and polymer for each candidate cluster in the region. To show the product, click on the expander or the candidate cluster feature drawn in the overview. </span></div>
</div>


<div>
<div class="prediction-text">
For candidate cluster 1, location 0 - 2803:
<div class="collapser collapser-target-CC1 collapser-level-candidate"> </div><div class="collapser-content">

<canvas id="smiles-canvas-r9c1-1" class="smiles-canvas" data-smiles="NC(C(C)C)C(=O)O"></canvas>
<div class="as-structure-warning">
Rough prediction of core scaffold based on assumed PKS/NRPS colinearity; tailoring reactions not taken into account
</div>


<div>
<dl>
<dt><b>Polymer prediction:</b></dt><dd>(Val)</dd>

</dl>
<br>Direct lookup in NORINE database:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*" target="_new">relaxed</a><br>
</div>

</div>
</div>
</div>

<div id="smiles-canvas-r9c1-1-modal" class="modal-container">
<div class="modal-content">
<canvas class="smiles-canvas-modal" data-smiles="NC(C(C)C)C(=O)O"></canvas>
</div>
</div>




<div class="prediction-text">
<br>
<a class="external-link" href="http://bioinfo.lifl.fr/norine/form2.jsp" target="_new">Link to NORINE database query form</a>
</div>

</div>
</div>

<div class="sidepanel-details-section r9c1-nrps_pks_monomers">
<div class="more-details">
<div class="heading">
<span>NRPS/PKS substrate predictions</span>
<div class="help-container"> <div class="help-icon" data-id="nrps-monomers-help-231"></div> <span class="help-tooltip" id="nrps-monomers-help-231">Shows the predicted substrates for each adenylation domain and acyltransferase within genes. Each gene prediction can be expanded to view detailed predictions of each domain. Each prediction can be expanded to view the predictions by tool  (and, for some tools, further expanded for extra details). </span></div>
</div>
<dl class="prediction-text">

<strong><span class="serif">HS041_37945</span></strong>: Val
<div class="collapser collapser-target-HS041_37945 collapser-level-cds"> </div><div class="collapser-content">
<dt></dt>

<dd> Search NORINE for peptide:
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val" target="_new">strict</a>
or
<a class="external-link" href="http://bioinfo.lifl.fr/norine/fingerPrintSearch.jsp?nrps1=val*" target="_new">relaxed</a>
</dd>
<br>



<dd><strong>AMP-binding (254..681)</strong>: Val
<div class="collapser collapser-target-HS041_37945 collapser-level-candidate"> </div><div class="collapser-content">

nrpys:  Val
<div class="collapser collapser-target-HS041_37945-nrpys collapser-level-none"> </div><div class="collapser-content">

<dl><dt>SVM prediction details:</dt>
<dd>    <dl>   <dt>Predicted physicochemical class:</dt>
<dd>hydrophobic-aliphatic (Ala, Gly, Val, Leu, Ile, Abu, Ival, Ser, Thr, Hpg, Dhpg, Cys, Pro, Hpr)</dd>
<dt>Large clusters prediction:</dt>
<dd>N/A</dd>
<dt>Small clusters prediction:</dt>
<dd>Aliphatic, branched hydrophobic (Val, Leu, Ile, Abu, Ival)</dd>
<dt>Single AA prediction:</dt>
<dd>N/A</dd>
</dl>
</dd>
</dl>
<dl><dt>Stachelhaus prediction details:</dt>
<dd>
<dl>
<dt>Stachelhaus sequence:</dt>
<dd><span class="serif">DVLWIGGIFK</span></dd>
<dt>Nearest Stachelhaus code(s):</dt>
<dd>Val <span class="serif">DALWIGGTFK</span> (68% 8Å match)</dd>
<dt>Stachelhaus code match:</dt>
<dd>80% (moderate)</dd>
</dl>
</dd>
</dl>

</div><br>

</div>
</dd>




</div>
<br>

</dl>
</div>
</div>






</div>

</div>

</div>


<footer class="footer">
<div class="container">
<div>
<img src="images/bacteria_antismash_logo.svg" style="height:90px;width:unset;">
</div>
<div class="cite-me">
If you have found antiSMASH useful, please <a href="https://antismash.secondarymetabolites.org/#!/about">cite us</a>.
</div>
<div>
<img src="images/bacteria_antismash_icon.svg" style="height:100px;width:unset;">
</div>
</div>
</footer>

<script src="js/jquery.js"></script>
<script src="js/antismash.js"></script>
<script src="regions.js"></script>
<script>
$(document).ready(function() {
viewer["start"](all_regions, details_data, resultsData, recordData);
})
</script>

<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width=0 height=0>
<defs>
<filter id="inset-shadow">
<feOffset dx="-2" dy="-2"></feOffset>
<feGaussianBlur result="offset-blur" stdDeviation="2"></feGaussianBlur>
<feComposite operator="out" in="SourceGraphic" in2="offset-blur" result="inverse"></feComposite>
<feFlood flood-color="black" result="color" flood-opacity="1"></feFlood>
<feComposite operator="in" in="color" in2="inverse" result="shadow"></feComposite>
<feComponentTransfer in="shadow" result="shadow">
<feFuncA type="linear" slope=".95"></feFuncA>
</feComponentTransfer>
<feComposite operator="over" in="shadow" in2="SourceGraphic"></feComposite>
</filter>
</defs>
</svg>
</body>
</html>